---
title: "Spinal premotor interneurons controlling antagonistic muscles are spatially intermingled"
author: ""
date: ""
output:
  html_document: default
  pdf_document: default
editor_options: 
  markdown: 
    wrap: 72
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning=FALSE)
```

```{r}
R <- 1 # number of replica for bootstrap, R=5000 was used to generate the results in Ronzano et al. 
bin <- 5 # number of contour for density plots

```

```{r}
## Main Directory ##
maindir <- "~"

library(openxlsx)
library(reshape2)
library(ggplot2)
library(RColorBrewer)
library(ggpubr)
library(knitr)
library(FSA)
library(car)
require(multcomp)
library(multcompView)
library(ggbeeswarm)
library(cowplot)
library(grid)
library(mratios)
library(plotrix)
library(ggpval)
library(forcats)
library(plotly)
library(tidyr)
library(dplyr)
library(plotlyGeoAssets)
library(tools)
library(stringr)
library(png)
library(devtools)
library(ggcorrplot)
library(scales)
library(eulerr)
library(ggvenn)
library(ggplotify)
library(gridExtra)
library(boot)
source("~/Desktop/last_version_segregation/Analysis/functions_elife_2022.R")

```
Remi Ronzano^\*1^, Sophie Skarlatou\*^2^, Bianca K. Barriga^\*6,7^, B. Anne Bannatyne^\*3^, Gardave S. Bhumbra^\*4^, Joshua D. Foster^4^, Jeffrey D. Moore^8^, Camille Lancelin^1^, Amanda Pocratsky^1^, Mustafa Görkem Özyurt^1^, Calvin C. Smith^1^, Andrew J. Todd^3^, David J. Maxwell^3^, Andrew J. Murray^5^, Samuel L. Pfaff^6§^, Robert M. Brownstone^1§^, Niccolò Zampieri^2§^, Marco Beato^4§^

^1^ Department of Neuromuscular Diseases, UCL Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK

^2^ Max Delbrück Center for Molecular Medicine (MDC), Roybert-Rössle-Str. 10, 13092 Berlin, Germany

^3^ Institute of Neuroscience and Psychology, College of Medical, Veterinary and Life Sciences, University of Glasgow, West Medical Building, Glasgow G12 8QQ, UK

^4^ Department of Neuroscience Physiology and Pharmacology (NPP), Gower Street, University College London, WC1E 6BT, UK

^5^ Sainsbury Wellcome Centre for Neural Circuits and Behaviour, University College London, London W1T 4JG, UK

^6^ The Salk Institute for Biological Studies, Gene Expression Laboratory, 10010 North Torrey Pines Road, La Jolla CA 92037, USA

^7^ Biological Sciences Graduate Program, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92037, USA.

^8^ Howard Hughes Medical Institute and Department of Molecular and Cellular Biology, Center for Brain Science, Harvard University, Cambridge, MA 02138, USA

\*equal contribution

^§^co-senior authors

[Corresponding authors]{.ul}:

Niccolò Zampieri, Max Delbrück Center for Molecular Medicine (MDC), Robert-Rössle-Str. 10, 13092 Berlin, Germany: [niccolo.zampieri\@mdc-berlin.de](mailto:niccolo.zampieri@mdc-berlin.de)

Samuel J. Pfaff, The Salk Institute for Biological Studies, Gene Expression Laboratory, 10010 North Torrey Pines Road, La Jolla CA 92037, USA: [pfaff\@salk.edu](javascript:void(window.open('/imp/dynamic.php?page=compose&to=pfaff%40salk.edu&popup=1','','width=820,height=610,status=1,scrollbars=yes,resizable=yes')))

Robert M. Brownstone, Department of Neuromuscular Diseases, UCL Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK: [r.brownstone\@ucl.ac.uk](mailto:r.brownstone@ucl.ac.uk)

Marco Beato, Department of Neuroscience Physiology and Pharmacology (NPP), Gower Street, University College London, WC1E 6BT, UK: [m.beato\@ucl.ac.uk](mailto:m.beato@ucl.ac.uk)

**Key words:** spinal cord, premotor interneurons, flexor muscles, extensor muscles, rabies, viral tracing

# Abstract

Elaborate behaviours are produced by tightly controlled flexor-extensor motor neuron activation patterns. Motor neurons are regulated by a network of interneurons within the spinal cord, but the computational processes involved in motor control are not fully understood. The neuroanatomical arrangement of motor and premotor neurons into topographic patterns related to their controlled muscles is thought to facilitate how information is processed by spinal circuits. Rabies retrograde monosynaptic tracing has been used to label premotor interneurons innervating specific motor neuron pools, with previous studies reporting topographic mediolateral positional biases in flexor and extensor premotor interneurons. To more precisely define how premotor interneurons contacting specific motor pools are organized we used multiple complementary viral-tracing approaches to minimize systematic biases associated with each method. Contrary to expectations, we found that premotor interneurons contacting motor pools controlling flexion and extension of the ankle are highly intermingled rather than segregated into specific domains like motor neurons. Thus, premotor spinal neurons controlling different muscles process motor instructions in the absence of clear spatial patterns among the flexor-extensor circuit components.


```{r, echo=FALSE}
## color used ##
R <- 10 # number of replica for bootstrap, R=5000 was used to generate the results in Ronzano et al. 

## files for spinal cord shape ##
setwd(str_c(maindir, "", "/data"))
spinal <- read.delim("halfcordt.txt", header = FALSE)
spinal$V1 <- spinal$V1 * 1700
spinal$V2 <- spinal$V2 * 900
spinal_contra <- spinal[order(nrow(spinal):1),] #transform the data to draw the contralateral side
spinal_contra$V1[sapply(spinal_contra$V1 , is.numeric)] <- spinal_contra$V1 [sapply(spinal_contra$V1, is.numeric)] * -1
spinal <- rbind(spinal, spinal_contra)

spinal_rc <- read.delim("halfcordld.txt", header = FALSE)
spinal_rc$V1 <- spinal_rc$V1 * 1700
spinal_rc <- spinal_rc %>% mutate(V2 = ifelse(V2 > 0, V2 * 1507.5404, V2 * 2421.9371)) #transform the data to fit to the scale from -2500 to 3000
spinal_rc_contra <- spinal_rc %>% mutate(V1 = V1*-1) #transform the data to draw the contralateral side

spinal_rc_y <- read.delim("halfcordf.txt", header = FALSE)
spinal_rc_y$V1 <- spinal_rc_y$V1 * 900
spinal_rc_y <- spinal_rc_y %>% mutate(V2 = ifelse(V2 > 0, V2 * 4491.354, V2 * 4534.224))
spinal_rc_y_contra <- spinal_rc_y %>% mutate(V1 = V1*-1) #transform the data to draw the contralateral side

## muscle injected ##
lg <- "lg"
ta <- "ta"
mg <- "mg"
pl <- "pl"
LG <- "LG"
TA <- "TA"
MG <- "MG"
PL <- "PL"
N_A <- ""
gs <- "gs"
GS <- "GS"

## color used ##

color_range_ext <- colorRampPalette(c("#006eb8", "#000a68"))
color_range_ext2 <- colorRampPalette(c("#000a68", "#006eb8"))
color_range_flex <- colorRampPalette(c("#eda911", "#8b4700"))
color_range_flex2 <- colorRampPalette(c("#8b4700", "#eda911"))

color_lg <- "#006eb8"
color_gs <- "#006eb8"
color_ta <- "#eda911"
color_mg <- "#000a68"
color_pl <- "#8b4700"
transpa <- "transparent"
color_glyt2_off <- "#db6ca9"
color_glyt2_on <- "#00a875"

```

```{r, echo=FALSE, warning=FALSE}

## Data organisation ##
IN <- "IN"
MN <- "MN"
Size <- c(0.5, 2) #Size of dots for INs, MNs, can be set whenever plotting a new distribution
ymin_d <- 0 #limit x min when ploting only dorsal part
ymin_vd <- -850 #limit x min when ploting from ventral to dorsal part
ymax_950 <- 950 #limit x max when ploting to more than 850
ymax_850 <- 850 #limit x max when ploting exactly to 850
data_all <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
# The rest of the chunk built the dataframe with all the data. The rest of the code work with this dataframe. 
## -- LG high G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/lg_high_chat"))
titer <- "high"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- TA high G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/ta_high_chat"))
titer <- "high"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- LG low G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/lg_low_chat"))
titer <- "low"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- TA low G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/ta_low_chat"))
titer <- "low"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- MG G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/mg_chat"))
titer <- "high"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- PL G_Rab_ChAT_RGT -- ##
setwd(str_c(maindir, "", "/data/pl_chat"))
titer <- "high"
method <- "G_Rab_ChAT_RGT"
data_all <- Data_building(data_all, titer, method) 

## -- GS G_Rab_Olig2_RGT -- ##
setwd(str_c(maindir, "", "/data/gs_olig2"))
titer <- "high"
method <- "G_Rab_Olig2_RGT"
data_all <- Data_building(data_all, titer, method)

## -- TA G_Rab_Olig2_RGT -- ##
setwd(str_c(maindir, "", "/data/ta_olig2"))
titer <- "high"
method <- "G_Rab_Olig2_RGT"
data_all <- Data_building(data_all, titer, method)

## -- GS AAV-G_WT -- ##
setwd(str_c(maindir, "", "/data/gs_aav_wt"))
titer <- "high"
method <- "AAV-G_WT"
data_all <- Data_building(data_all, titer, method)

## -- TA AAV-G_WT -- ##
setwd(str_c(maindir, "", "/data/ta_aav_wt"))
titer <- "high"
method <- "AAV-G_WT"
data_all <- Data_building(data_all, titer, method)


## -- GS AAV-G_ChAT -- ##
setwd(str_c(maindir, "", "/data/gs_aav_chat"))
titer <- "high"
method <- "AAV-G-FLEX_ChAT-cre"
data_all <- Data_building(data_all, titer, method)

## -- TA AAV-G_ChAT -- ##
setwd(str_c(maindir, "", "/data/ta_aav_chat"))
titer <- "high"
method <- "AAV-G-FLEX_ChAT-cre"
data_all <- Data_building(data_all, titer, method)

## -- GS PRV -- ##
setwd(str_c(maindir, "", "/data/gs_prv"))
titer <- "high"
method <- "PRV_WT"
data_all <- Data_building(data_all, titer, method)

## -- TA PRV -- ##
setwd(str_c(maindir, "", "/data/ta_prv"))
titer <- "high"
method <- "PRV_WT"
data_all <- Data_building(data_all, titer, method)


```

# Introduction

Precise regulation in the timing and pattern of activation of muscle groups across a joint is at the basis of motor control. In limbed vertebrates, the activity of flexor and extensor muscles is directed by dedicated pools of motor neurons that receive inputs from different subtypes of excitatory and inhibitory interneurons. Several of these classes of interneurons have been described in electrophysiological, anatomical, and genetic studies (Hultborn et al., 1971; Jankowska, 2001; Goulding, 2009), however the incomplete knowledge of the composition of spinal circuitry that control the activity of flexor and extensor motor neurons limits progress towards a full understanding of motor circuits.

Viral trans-synaptic tracing techniques have been used for several decades to map motor circuits (Ugolini, 2020). Given that rabies virus (RabV) jumps across synapses in the retrograde direction to infect presynaptic neurons, its use via intramuscular injections led to the identification of polysynaptic pathways (Rathelot and Strick, 2009). But the identification of monosynaptic connectivity relied on the timing of transsynaptic jumps, thus leading to a degree of uncertainty about the number of synapses between a labelled neuron and the motor neurons innervating the injected muscle.

The introduction of RabV monosynaptic tracing provided a high-throughput method for mapping presynaptic connectivity of selected neuronal populations (Callaway and Luo, 2015). Monosynaptic restriction is achieved by using a mutant virus lacking the gene encoding the rabies glycoprotein (G; ΔG-RabV), which is necessary for transsynaptic transfer, combined with selective complementation of G expression in neurons of choice (Wickersham et al., 2007). Various methodologies have been used to restrict G expression to the target neuronal population, with these G-expressing neurons becoming "starter cells" from which infecting ΔG-RabV can jump only one synapse and selectively label presynaptic neurons (Wall et al., 2010; Callaway and Luo, 2015).

Shortly after its introduction, monosynaptic rabies tracing was applied to the study of premotor interneurons in the spinal cord. To obtain selective complementation of G and subsequent rabies monosynaptic transfer from a single motor neuron pool, an elegant approach based on intramuscular co-injection of an AAV expressing G (AAV-G) and ΔG-RabV, both of which can infect motor neurons retrogradely, was described (Stepien et al., 2010). Thus, starter cells are generated in one fell swoop by taking advantage of the stringent anatomical specificity of motor neuron to muscle connectivity (Figure 1A). When this method was applied to study the distribution of premotor interneurons controlling the activity of extensor and flexor muscles in the hindlimb, a prominent spatial segregation along the medio-lateral axis of the dorsal ipsilateral spinal cord was observed, with extensor premotor interneurons found in more medial positions than flexors (Tripodi et al., 2011). The authors proposed that this organisation led to "private" disynaptic pathways from proprioceptive afferents to appropriate motor neurons, and that this might offer some circuit organisational advantages.

More recently, in order to address concerns that this method could also lead to rabies infection and transsynaptic transfer from the sensory route (Figure 1A; Zampieri et al., 2014), G expression was further restricted to motor neurons by combining the use of a mouse line expressing Cre recombinase under the control of choline acetyltransferase (ChAT) and intramuscular injection of an AAV driving expression of G in a conditional manner (AAV-flex-G; Figure 1B). Under these conditions, segregation of flexor and extensor premotor interneurons was also shown at forelimb level (Wang et al., 2017). Finally, a further modification to the original AAV complementation strategy was introduced: stereotactic injection of AAV-flex-G in the spinal cord of *ChAT::Cre* mice was used to target G expression to cholinergic neurons, with restriction of starter cells to a motor pool achieved by ΔG-RabV muscle injection. These experiments also showed medio-lateral segregation in the distribution of flexor and extensor premotor interneurons (Figure 1C; Takeoka and Arber, 2019). Thus, these different experiments that all used AAV for complementing G expression in motor neurons demonstrated similar segregation of extensor and flexor premotor interneurons.

Given the importance that neuronal position may play in circuit organization and function, we sought to identify premotor interneurons for further investigation. However, we elected to achieve G complementation by using a mouse genetic approach that takes advantage of a conditional mouse line that drives G expression under control of Cre recombinase (*RΦGT* mice; Figure 1D, 1E; Takatoh et al., 2013). This method has been previously used to trace premotor circuitry of the vibrissal and orofacial motor systems, in combination with *ChAT::Cre* mice (Takatoh et al., 2013; Stanek et al., 2014), as well as forelimb muscles in combination with *Olig2::Cre* mice (Skarlatou et al., 2020). We reasoned that using this approach, G should be available at high levels in all motor neurons, thereby leading to efficient monosynaptic transfer from all the cells infected by ΔG-RabV, which was supplied via intramuscular injection. In contrast with previous studies, we did not observe any difference in the distribution of flexor and extensor premotor interneurons. Thus, we decided to replicate the experiments using AAV complementation strategies: AAV-G injection in wild-type mice (Tripodi et al., 2011) and AAV-flex-G in *ChAT::Cre* mice (Wang et al., 2017). Surprisingly, we did not observe segregation in the spatial organization of flexor and extensor premotor interneurons. Finally, we resorted to a different viral tracing method and used timed infection with pseudorabies virus (PRV; Strack and Loewy, 1990; Jovanovic et al., 2010), which also resulted in overlapping distributions of flexor and extensor premotor neurons. Altogether, these experiments conducted in different laboratories and using most of the available methods described in the literature for viral transsynaptic tracing of premotor circuits do not show segregation of flexor and extensor premotor neuron distributions.

### Figure 1

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width= 10 ,fig.height= 8}
fig1 <- readPNG("~/figures/Figure_1.png")
Fig1 <- ggplot() + 
  background_image(fig1) +
  coord_equal(ratio = 1.07) +
  annotation_custom(rasterGrob(fig1, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0.5, l=0.5, r=0.5, b = 0.5, unit = "cm"))

Fig1
```

**Figure 1**: Schematic diagram of the currently available techniques for labelling premotor interneurons. A: simultaneous injection of RabV and AAV-G into muscles. Rabies transfer pathways that could potentially contaminate the distribution of premotor interneurons are labelled by a question mark and indicated by dashed lines. A: Afferent labelling could lead to anterograde labelling of sensory related interneurons. B: simultaneous muscle injection of RabV and a Cre dependent AAV-G into mice expressing Cre in motor neurons eliminates the risk of anterograde transfer from afferents. C: intraspinal injection of a flexed AAV-G in mice expressing Cre in motor neurons is followed by intramuscular rabies injection. D and E: RabV muscle injection is performed on mice selectively expressing the rabies glycoprotein in cholinergic neurons (D) or neuron expressing the Olig2 transcription factor (E). F: PRV Bartha is injected in muscles and retrogradely spreads through synapses. Restriction to first order interneurons can be achieved by extracting the tissue early (\~48 hours) after injection.

# Results

### Flexor and extensor premotor interneurons in *ChAT::Cre +/-; RΦGT+/-* mice

In order to determine the spatial distribution of premotor interneurons controlling flexion and extension of the ankle, we injected ΔG-RabV/mCherry and ΔG-RabV/eGFP in the TA (tibialis anterior; ankle flexor) and LG (lateral gastrocnemius; ankle extensor) muscles of postnatal day (P) 1-2 *ChAT::Cre +/-; RΦGT+/-* mice. Analysis of lumbar level (L) 2 and L5 sections 8-9 days after injection revealed two main clusters of premotor interneurons located in the dorsal ipsilateral and ventral contralateral spinal cord (Figure 2A and 2B). Next, we obtained coordinates for the labelled cells in each section of the lumbar spinal cord and mapped premotor interneuron positions in three dimensions. The projection of x-y coordinates along the rostro-caudal axis of the spinal cord showed no difference in medio-lateral and dorso-ventral positions of flexor and extensor premotor interneurons (Figure 2C, left panel). Convolved distributions fully overlapped for the two groups in all four quadrants. Similarly, projection along the coronal plane (Figure 2C, middle panel) or sagittal plane (Figure 2C, right panel) revealed no obvious differences in the rostro-caudal, dorso-ventral and medio-lateral distribution of flexor and extensor premotor interneurons.


### Figure 2
```{r, fig.align='left', fig.width= 12 , fig.height= 5}
## -- 2AB -- ##
fig2ab <- readPNG("~/figures/Figure_2AB.png")
Fig2AB <- ggplot() + 
  background_image(fig2ab) +
  coord_equal(ratio = 0.380) +
  annotation_custom(rasterGrob(fig2ab, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0.0, l=0, r=0, b = 0, unit = "cm"))
Fig2AB
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}

## -- 2C -- ##
panel <- "C"
concatenate = 0
density <- 0
kernel <- 6
bin <- 5
Size <- c(0.2, 1)
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "high"
Animal <- "170427n2_n_rmb"
Figure_2Ci <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, N_A, N_A, kernel, bin, method_used, titer_used, panel)
Figure_2Cii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, method_used, titer_used)
Figure_2Ciii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_2Ci, NULL, Figure_2Cii, NULL, Figure_2Ciii, ncol=5, nrow = 1, align = "hv",
                                   rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
#NULL are to adapt the space between plots. 
```
```{r, fig.align='left', fig.width= 12 , fig.height= 6}
## -- 2DE -- ##
fig2de <- readPNG("~/figures/Figure_2DE.png")
Fig2DE <- ggplot() + 
  background_image(fig2de) +
  coord_equal(ratio = 0.478) +
  annotation_custom(rasterGrob(fig2de, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))

Fig2DE

```
**Figure 2**: Double injections of flexor and extensor muscles shows no segregation of premotor interneurons. A: Maximum intensity projection of a single 30 μm section taken from the L2 segment of a P10 cord infected with ΔG-Rab-mCherry in the LG and ΔG-Rab-EGFP in the TA in *ChAT-RΦGT* mice. B: Same sample as A, showing a L5 section (scale bars: 200 μm). C: Projections along the transverse (left), sagittal (middle) and coronal (right) axes throughout the lumbar region of the sample showed in A and B (170427n2, UCL). Dots denote individual premotor interneurons, triangles denote infected motor neurons. Convolved density along each axe are shown to the sides of the raw data (top-bottom and left-right distributions in all panels sum to 1). For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. D: Half section of a cord on the side of a double injection of LG and TA in the L4 segment. Some isl1+ motor neurons are labelled in the dorsal nuclei and one (indicated by arrowhead and enlarged in E) is labelled by both fluorescent proteins, indicating a potential transsynaptic jump between antagonist motor neurons (scale bars, 250 μm and 50 μm in D and E respectively).

<br/> <br/> To study in detail the positional organization of premotor interneurons controlling the activity of the ankle joint, we analysed 13 animals in which we had performed simultaneous ΔG-RabV-eGFP and ΔG-RabV-mCherry injections in 3 different pairs of antagonist and synergist muscles in P1-2 *ChAT::Cre +/-; RΦGT+/-* mice: TA and LG, LG and MG (medial gastrocnemius; ankle extensor and LG synergist) and TA and PL (peroneus longus; ankle flexor and TA synergist). We started by analysing the results from the antagonist pairs, LG (n=11) and TA (n=7), pooling experiments from single and double injections (Figures 3A-B). All LG and TA experiments are overlaid, with different shades of blue (LG) and orange (TA) representing different animals, showing the reproducibility of premotor interneurons distributions across single experiments (pooled distributions shown in Figure 3C, individual experiments are shown in Figure 12- figure supplements 1-7). The Hedges' G coefficients of the distributions in the ipsilateral dorsal quadrants for all pairs of experiments had a median of -0.06 (IQR -0.26, 0.14), showing homogeneity between experiments and no differences in the positional organization of flexor and extensor premotor interneurons. The reproducibility of the results is confirmed by analysis of the coordinates across all experiments, showing similar correlation values within or across muscles (Figure 3D; r≥0.78). The values of the medians of individual experiments for LG and TA injections were 329 and 315 μm respectively (Hedges' G=-0.06, Figures 3E-F).

Since it was previously reported that the medio-lateral segregation in the distribution of flexor and extensor premotor interneurons is more pronounced in spinal segments rostral to the infected motor nucleus (Tripodi et al., 2011), we analysed the organization of premotor interneurons at different lumbar levels. Positional coordinates were pooled and divided into 800 μm rostro-caudal bins and distributions were plotted for each bin from L1 to L6 (Figure 3-figure supplement 1). No differences in the medio-lateral distributions of LG and TA premotor interneurons were observed in any segment analysed (median positions on the medio-lateral axis for L1, the segment with the largest visible medio-lateral segregation: LG = 309 μm and TA = 327 μm, hierarchical bootstrapped Hedges'G = -0.17 (IQR -0.12, -0.22)). We further tested whether the normalization procedure might have affected the relative position of LG and TA premotor interneurons by plotting the raw coordinates split across segments (Figure 3-figure supplement 2) and confirmed that even in L1 there was no medio-lateral segregation (non-scaled medians were LG = 275 μm and TA = 285 μm, hierarchical bootstrapped Hedges'G = 0.14 (IQR -0.20,-0.09)). Moreover, the relative density of LG and TA interneurons was similar throughout the lumbar segments (Figure 3- figure supplement 3).

### Figure 3
```{r, fig.align='left' , fig.width= 10 ,fig.height= 8}
## -- Figure 3 -- ##
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "high"
Size <- c(0.1, 0)
kernel <- 6
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
## -- 3A -- ##
density <- 0
concatenate <- 0
panel <- ""
Figure_3A <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext2, transpa, color_lg, transpa, 11, 
                                       0, IN, N_A, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)

Figure_3A <- ggarrange(NULL, Figure_3A, NULL,
                       ncol = 3, nrow = 1,  align = "hv",
                       widths = c(0.2,5,0.2), heights = c(4), common.legend = TRUE, labels = "A", font.label = list(size = 28, color = "black"))
## -- 3B -- ##
density <- 0
concatenate <- 0
panel <- ""
Figure_3B <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex2, transpa, color_ta, transpa, 7, 
                                       0, IN, N_A, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)

Figure_3B <- ggarrange(NULL, Figure_3B, NULL,
                       ncol = 3, nrow = 1,  align = "hv",
                       widths = c(0.2,5,0.2), heights = c(4), common.legend = TRUE, labels = "B", font.label = list(size = 28, color = "black"))
## -- 3C -- ##
density <- 1
concatenate <- 1
panel <- ""
Figure_3C <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 11, 
                                       7, IN, N_A, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
Figure_3C <- ggarrange(NULL, Figure_3C, NULL,
                       ncol = 3, nrow = 1,  align = "hv",
                       widths = c(0.2,5, 0.2), heights = c(4), common.legend = TRUE, labels = "C", font.label = list(size = 28, color = "black"))
## -- 3D -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
panel <- "D"
numbers <- 0
Figure_3D <- Corr_HedgesG(data_all, lg, ta, method_used, titer_used, N, lim, panel, color_lg, color_ta, 11, 7, numbers) 


cowplot::plot_grid(Figure_3A, Figure_3B, Figure_3C, Figure_3D, ncol=2,nrow = 2, align = "hv", rel_widths = c(5.17,5.17), rel_heights = c(3,3))


```

**Figure 3**: Consistent distribution of flexor and extensor premotor interneurons across all individual experiments. A, B: Distribution of premotor interneurons of LG (A) and TA (B) for all the injections. Distributions for each individual experiment are represented with different shades of blue and orange. C: All experiments (single or double ΔG-RabV injections) pooled, showing an overlap of the flexor and extensor related distribution in all quadrants of the spinal cord, with individual dots replaced by contours. For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. D: Correlation (top right) and absolute value of Hedges' G coefficient (bottom left) across all pairs of experiments, indicating a high degree of consistency and small effect sizes across all experiments, independent of the muscle injected.

```{r, fig.align='center', fig.width=8 ,fig.height=2}
## -- 3E -- ##
Figure_3E_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, lg, ta, color_lg, color_ta, ymin_d, ymax_950)

## -- 3F -- ##
dat3F <- df_individual(data_all, method_used, titer_used, lg, ta, LG, TA) 
Figure_3F_interactive <- Boxplot_individual_interactive(dat3F, color_lg, color_ta, ymin_d, ymax_950)

Fig_3EF <- subplot(Figure_3E_interactive, Figure_3F_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_3EF <- Fig_3EF %>% layout(width = 900, height = 450)
Fig_3EF
```

Left: Box and whisker plot of the mediolateral position of dorsal ipsilateral premotor interneurons in each experiment. Right: Values of the dorsal ipsilateral interneuron distribution median for each LG and TA experiment. Pooled distributions split into lumbar segments are shown in Figure 3 figure supplement 1, normalized and Figure 3 figure supplement 2, raw data before normalization.

### Figure 3 figure supplement 1

```{r, fig.align='left',, fig.width= 10, fig.height= 12}

## -- Figure 3 supp 1 -- ##
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "high"
Size <- c(0.1, 0)
kernel <- 6
concatenate = 0
density <- 0
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
L1 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
L2 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
L3 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
L4 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
L5 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
L6 <- data.frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
for (i in 1:length(Animal)) {
  dfl1 <- data_all %>% filter(animal == Animal[i]   &  (z  <= -2400) & (-3200  < z) & identity == IN)
  L1 <- rbind(L1, dfl1)
  dfl2 <- data_all %>% filter(animal == Animal[i]   &  (z  <= -1600) & (-2400  < z) & identity == IN)
  L2 <- rbind(L2, dfl2)
  dfl3 <- data_all %>% filter(animal == Animal[i]   &  (z  <= -800) & (-1600  < z) & identity == IN)
  L3 <- rbind(L3, dfl3)
  dfl4 <- data_all %>% filter(animal == Animal[i]   &  (z  <= 0) & (-800  < z) & identity == IN)
  L4 <- rbind(L4, dfl4)
  dfl5 <- data_all %>% filter(animal == Animal[i]   &  (z  <= 800) & (0  < z) & identity == IN)
  L5 <- rbind(L5, dfl5)
  dfl6 <- data_all %>% filter(animal == Animal[i]   &  (z  <= 1600) & (800  < z) & identity == IN)
  L6 <- rbind(L6, dfl6)
}


xmin <- -850
xmax <- 850
ymin <- -450
ymax <- 450
panel <- c("A", "B", "C", "D", "E", "F")
segment <- c("L1", "L2", "L3", "L4", "L5", "L6")
Figure_3supp1A <- complete_graph_xy_density_fig3_figSupp(spinal, L1, lg, ta, color_lg, color_ta, segment[1], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[1])
Figure_3supp1B <- complete_graph_xy_density_fig3_figSupp(spinal, L2, lg, ta, color_lg, color_ta, segment[2], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[2])
Figure_3supp1C <- complete_graph_xy_density_fig3_figSupp(spinal, L3, lg, ta, color_lg, color_ta, segment[3], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[3])
Figure_3supp1D <- complete_graph_xy_density_fig3_figSupp(spinal, L4, lg, ta, color_lg, color_ta, segment[4], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[4])
Figure_3supp1E <- complete_graph_xy_density_fig3_figSupp(spinal, L5, lg, ta, color_lg, color_ta, segment[5], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[5])
Figure_3supp1F <- complete_graph_xy_density_fig3_figSupp(spinal, L6, lg, ta, color_lg, color_ta, segment[6], kernel, LG, TA, Size, ymin, ymax, xmin, xmax, panel[6])

cowplot::plot_grid(Figure_3supp1A, Figure_3supp1B, Figure_3supp1C, Figure_3supp1D, Figure_3supp1E, Figure_3supp1F,
                   ncol=2,nrow = 3, align = "hv", rel_widths = c(5.17,5.17), rel_heights = c(3,3,3))

```

**Figure 3 figure supplement 1**: The distribution of premotor interneurons is similar throughout the lumbar spinal cord: data pooled from 18 experiments (11 LG and 7 TA injections) show that within each lumbar segment, from L1 to L6, the distributions of LG and TA premotor interneurons are overlapping.

### Figure 3 figure supplement 2
```{r, echo=FALSE, warning=FALSE, fig.align='left' , fig.width= 10 ,fig.height= 12}

## -- Figure 3 supp 2 -- ##
segment_norm <- c(800, 850, 900, 950, 1000, 950)
setwd(str_c(maindir, "", "/data"))
spinal_raw <- read.delim("halfcordt.txt", header = FALSE)

spinal_L1 <- spinal_transform(spinal_raw, segment_norm[1])
spinal_L2 <- spinal_transform(spinal_raw, segment_norm[2])
spinal_L3 <- spinal_transform(spinal_raw, segment_norm[3])
spinal_L4 <- spinal_transform(spinal_raw, segment_norm[4])
spinal_L5 <- spinal_transform(spinal_raw, segment_norm[5])
spinal_L6 <- spinal_transform(spinal_raw, segment_norm[6])

setwd(str_c(maindir, "", "/data_raw"))
titer <- "high"
method <- "G_Rab_ChAT_RGT"
panel <- c("A", "B", "C", "D", "E", "F")
segment <- c("L1", "L2", "L3", "L4", "L5", "L6")
xmin <- -1000
xmax <- 1000
ymin <- -650
ymax <- 800

L1_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L1_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L1_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L1_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L1_lg) <- c("x", "y", "slice")
colnames(L1_ta) <- c("x", "y", "slice")
Figure_3supp2A <- complete_graph_xy_density_fig3_figSupp2(spinal_L1, L1_lg, L1_ta, color_lg, color_ta, segment[1], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[1])

L2_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L2_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L2_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L2_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L2_lg) <- c("x", "y", "slice")
colnames(L2_ta) <- c("x", "y", "slice")
Figure_3supp2B <- complete_graph_xy_density_fig3_figSupp2(spinal_L2, L2_lg, L2_ta, color_lg, color_ta, segment[2], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[2])

L3_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L3_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L3_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L3_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L3_lg) <- c("x", "y", "slice")
colnames(L3_ta) <- c("x", "y", "slice")
Figure_3supp2C <- complete_graph_xy_density_fig3_figSupp2(spinal_L3, L3_lg, L3_ta, color_lg, color_ta, segment[3], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[3])

L4_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L4_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L4_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L4_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L4_lg) <- c("x", "y", "slice")
colnames(L4_ta) <- c("x", "y", "slice")
Figure_3supp2D <- complete_graph_xy_density_fig3_figSupp2(spinal_L4, L4_lg, L4_ta, color_lg, color_ta, segment[4], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[4])

L5_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L5_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L5_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L5_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L5_lg) <- c("x", "y", "slice")
colnames(L5_ta) <- c("x", "y", "slice")
Figure_3supp2E <- complete_graph_xy_density_fig3_figSupp2(spinal_L5, L5_lg, L5_ta, color_lg, color_ta, segment[5], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[5])

L6_lg <- read.xlsx(xlsxFile = "figS3_2_LG-L6_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
L6_ta <- read.xlsx(xlsxFile = "figS3_2_TA-L6_raw.xlsx", sheet = 1, skipEmptyRows = TRUE, colNames=FALSE)
colnames(L6_lg) <- c("x", "y", "slice")
colnames(L6_ta) <- c("x", "y", "slice")
Figure_3supp2F <- complete_graph_xy_density_fig3_figSupp2(spinal_L6, L6_lg, L6_ta, color_lg, color_ta, segment[6], kernel, LG, TA, ymin, ymax, xmin, xmax, panel[6])

cowplot::plot_grid(Figure_3supp2A, Figure_3supp2B, Figure_3supp2C, Figure_3supp2D, Figure_3supp2E, Figure_3supp2F,
                   ncol=2,nrow = 3, align = "hv", rel_widths = c(5.17,5.17), rel_heights = c(3,3,3))

```

**Figure 3 figure supplement 2**: Same data as in Figure 3-figure supplement 1 shown before normalization procedures, with idealized spinal cord section scaled to the average size of each segment. Using the raw coordinates, the distribution of LG and TA premotor interneurons are consistently overlapping throughout the lumbar segments.


### Figure 3 figure supplement 3
```{r, fig.align='left',  fig.width= 12 , fig.height= 4}

## -- coronal -- ##
panel <- ""
density <- 1
concatenate <- 1
kernel <- 6
Figure_3supp3 <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 11, 
                                       7, IN, N_A, Size, LG, TA, kernel, bin, method_used, titer_used, panel)

## -- rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_3supp3ii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 11, 
                                         7, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_3supp3iii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 11, 
                                          7, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_3supp3, NULL, Figure_3supp3ii, NULL, Figure_3supp3iii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```

**Figure 3 figure supplement 3**: distribution of flexor and extensor premotor interneurons pooled across all LG and TA injections shown in the transverse plane (left) and as front (middle) and lateral (right) view along the rostrocaudal axis. <br/> <br/> <br/>

### The identity of infected motor neurons

The identity of starter cells represents a critical element for the interpretation of rabies tracing experiments. For the rabies tracing approaches discussed here (Figure 1), it is difficult to determine unambiguously the number of starter motor neurons because of rabies toxicity, that kills many neurons shortly after infection (Reardon et al., 2016). Nevertheless, we took advantage of the topographic organization of motor neuron to muscle connectivity to evaluate the pool identity and number of infected motor neurons that survived until the end of the experiment (Romanes, 1964; McHanwell and Biscoe, 1981). As predicted by the known position of the TA and LG motor pools in the spinal cord, we found that the majority of infected motor neurons were localized in the dorsal part of the ventral horn (Figure 2C; Sürmeli et al., 2011). Surprisingly, we have also found some putative motor neurons (23 out of 1174, see example in Figure 2C) in positions consistent with medial motor column identity and motor neurons in more ventral, "ectopic" positions (Figure 2C and Figure 12 figure supplement 1-7), where pools that innervate muscles controlling the function of the knee and hip joints reside (Sürmeli et al., 2011). Motor neuron labelling occasionally extended outside the lower lumbar segments where most of the ankle flexors and extensor pools are located (Figure 12-figure supplement 1-7). Moreover, in double TA and LG injections we found instances of motor neurons infected with both viruses (Figures 2D, E). In 5 experiments, a total of 200 LG and 150 TA motor neurons were labelled, of which 13 were infected with both ΔG-RabV-eGFP and ΔG-RabV-mCherry (see Table 1).

The presence of ectopic and double labelled motor neurons could be explained by either secondary labelling due to rabies transsynaptic transfer from starter cells through recurrent connections from these other motor neurons or by unintended primary infection of motor neurons due to non-specific muscle injections. We cannot say whether cholinergic cells in the medial motor column are indeed motor neurons. They could be either large cholinergic interneurons that are presynaptic to motor neurons, or (perhaps more likely) medial motor neurons that send recurrent axon collaterals to lateral motor neurons. In fact, while collaterals from lateral or medial motor neurons have not been traced outside their respective columns, the dendritic arborization from medial and lateral motor neuron columns extends to each other and labelling of lateral motor neurons following rabies injection of medial column innervating muscles has been observed (Goetz et al., 2015; Balaskas et al., 2019; Feng et al., 2022), indicating that some connectivity, at least from lateral to medial columns, is possible. With respect to "ectopic" labelling in other LMC pools, it is important to consider leak of the virus at the injection sites. With our small volume (1 μl) injections, leak between muscles located on opposite side of the fibula (GS and TA) is extremely unlikely. However, the lower part of the biceps femoris muscle has some overlap with the gastrocnemius, potentially leading to leak of virus into the biceps compartment (Sürmeli et al., 2011). Careful post-hoc analysis of hindlimb muscles after ΔG-RabV injection did not reveal any evidence of non-specific muscle infection in the upper and lower aspects of the leg, but we cannot exclude contamination of non-targeted muscles that gave rise to expression of reporter protein that was below the threshold for detection. If this were the case, presumably a very small number of motor neurons belonging to a non-targeted muscle could have been infected. We therefore suggest that the majority of ectopic motor neurons were labelled transsynaptically and therefore represent second-order presynaptic neurons. This is not surprising, as motor neurons have been shown to form synapses with other motor neurons and their connections can extend to neighbouring spinal segments (Bhumbra and Beato, 2018), suggesting that the ectopic motor neurons found in our experiments were most likely due to rabies transsynaptic transfer. Regardless of the underlying reasons for the observed ectopic motor neuron labelling, its presence raises the possibility that what we defined as flexor and extensor premotor networks, might, in fact, originate from a mixed population of starter cells containing not only motor neurons of a single pool identity but also a fraction of, "non-specific", motor neurons belonging to other pools, thereby potentially diluting any observable spatial difference between the premotor networks of flexor and extensor muscles. However, it is important to notice that in our experiments the number of presumed "non-specific" starter cells is low and therefore unlikely to confound the results (Table 1; see below and Discussion).

### The number of infected motor neurons does not affect the distribution of premotor interneurons

Spinal or muscle injection of AAV to complement G expression is likely to result in infection of a subset of motor neurons within the targeted pool, whereas the genetic experiments will result in complementation in all motor neurons. Thus, it is possible that the difference in the results obtained with these two methods could be explained by the absolute number of motor neurons from which rabies synaptic transfer occurs. In order to test the effect of the number of starter cells in our experimental conditions, we reasoned that by reducing the viral titre of the rabies solution used for muscle injection, we would scale down the number of infected motor neurons. Therefore, we performed a series of muscle injections (7 LG and 6 TA, of which 3 double LG-TA) in *ChAT::Cre +/-; RΦGT+/-* mice (Figure 1D) as in the experiments described above, but with diluted rabies virus (titre \<10^9^ I.U./ml) to reduce its infection efficiency (see Table 1). In low titre experiments, we detected an average 4.7 infected motor neurons compared to an average of 35.2 in the high titre experiments (titre \>5×10^9^ I.U./ml). Once again, we did not observe segregation in the medio-lateral distribution of LG and TA premotor interneurons (Figures 4A-C), nor along the rostro-caudal axis (Figure 4-figure supplement 1). The pairwise Hedges' G coefficients for all the experiments had median value of -0.05 (IQR -0.19,0.15, Figure 4D). While there was a higher degree of variability between experiments compared to high titre injections, as shown in the correlation matrix of individual experiments (Figure 4D, r\>0.45 for all comparisons), the median value of the medio-lateral positions in each experiment were very similar (LG = 321 μm and TA = 322 μm, hierarchical bootstrapped Hedges' G=-0.02, IQR -0.09, 0.03, Figures 4E, F, individual experiments shown in Figure 12-figure supplement 8-10). Next, we compared high and low titre experiments for each muscle injected (Figure 4-figure supplement 2). The distribution of premotor interneurons shows remarkable overlap for LG and TA injections (Figure 4-figure supplement 2A, E). The medio-lateral distributions were not different for LG (median Hedges' G = -0.05, IQR -0.21,0.11, with high correlation values across experiments, between 0.77 and 0.98, Figure 4-figure supplement 2B, and medians of 321 μm and 329 μm for high and low titre experiments respectively, Figure 4-figure supplement 2C-D). When comparing the high and low titre injections of TA, we found high correlation values across experiments (across experiment median 0.83, IQR 0.74, 0.90) and low Hedges' G (median -0.10, IQR -0.26, 0.13, Figure 4-figure supplement 2F) and no differences in the lateral direction for low titre injections into TA (medians: high = 315 μm and low = 322 μm, Figure 4-figure supplement 2G-H). Of note, in low titre experiments we never observed ectopic motor neurons outside the expected nucleus (individual experiments are shown in Figure 12- figure supplements 8-10).

Despite the almost 10-fold difference in the estimated number of primary infected motor neurons between high and low titre experiments, the spatial distribution of premotor interneurons was not altered, but their absolute number was. This high variability in the number of primary infected neurons has been observed in the combined analysis of large datasets of rabies tracing experiments in the brain where the relation between primary and secondary infected cells is described by a power law (Tran-Van-Minh et al., 2022). To verify that our dataset followed the same statistical rules, the number of cells was scaled across different experiments to reflect the different sampling of the sections (see Table 1) and the number of interneurons vs. number of putative starter motor neurons was plotted (Figure 4-figure supplement 3). The data points are well fitted by a power law (R^2^ = 0.48), in agreement with published brain datasets (Tran-Van-Minh et al., 2022). The ratio between interneurons and motor neurons numbers had medians of 51 (IQR 33-137) and 37 (IQR 25-67) for high and low titre injections respectively (Hedges'G = 0.84). While high variability in the initial number of starter cells seems inherent to rabies tracing, our dataset reflects a similar dependency between the number of primary and secondary infected neurons as that observed in other published datasets obtained in different parts of the central nervous system from different laboratories. Together, these data indicate that neither the absolute number of starter motor neurons nor the infection of ectopic motor neurons observed in high titre experiments significantly affects the positional organization of premotor interneurons.

### Figure 4
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 10 ,fig.height= 8 }
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "low"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
## -- 4A -- ##
panel <- ""
density <- 0
concatenate <- 0
Figure_4A <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext2, transpa, color_lg, transpa, 7, 0, IN, N_A, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)

Figure_4A <- ggarrange(NULL, Figure_4A, NULL,ncol = 3, nrow = 1,  align = "hv",
                widths = c(0.2,5,0.2), heights = c(4), common.legend = TRUE, labels = "A", 
                font.label = list(size = 28, color = "black"))
## -- 4B -- ##
panel <- ""
density <- 0
concatenate <- 0
Figure_4B <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex2, transpa, color_ta, transpa, 6, 
                                       0, IN, N_A, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)

Figure_4B <- ggarrange(NULL, Figure_4B, NULL,ncol = 3, nrow = 1,  align = "hv",
                widths = c(0.2,5,0.2), heights = c(4), common.legend = TRUE, labels = "B", 
                font.label = list(size = 28, color = "black"))
## -- 4C -- ##
panel <- ""
density <- 1
concatenate <- 1
Figure_4C <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 7, 
                                       6, IN, N_A, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
Figure_4C <- ggarrange(NULL, Figure_4C, NULL,ncol = 3, nrow = 1,  align = "hv",
                widths = c(0.2,5,0.2), heights = c(0.2,4, 0.2), common.legend = TRUE, labels = "C", 
                font.label = list(size = 28, color = "black"))
## -- 4D -- ##
panel <- "D"
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 0
Figure_4D <- Corr_HedgesG(data_all, lg, ta, method_used, titer_used, N, lim, panel, color_lg, color_ta, 7, 6, numbers)

cowplot::plot_grid(Figure_4A, Figure_4B, Figure_4C, Figure_4D, ncol=2,nrow = 2, align = "hv", rel_widths = c(5.17,5.17), rel_heights = c(3,3))

```
**Figure 4**: Injection with diluted RabV do not reveal any segregation between flexor and extensor premotor interneurons. A and B show the distribution of LG and TA premotor interneurons on the transverse plane for individual experiments, represented with different colour shades. For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. C: LG and TA distributions overlap and the premotor interneuron distributions are highly correlated across experiments and absolute value of Hedges' G coefficients are small (D).

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}

## -- 4E -- ##
Figure_4E_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, lg, ta, color_lg, color_ta, ymin_d, ymax_950)

## -- 4F -- ##
dat4F <- df_individual(data_all, method_used, titer_used, lg, ta, LG, TA) 
Figure_4F_interactive <- Boxplot_individual_interactive(dat4F, color_lg, color_ta, ymin_d, ymax_950)

Fig_4EF_interactive <- subplot(Figure_4E_interactive, Figure_4F_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_4EF_interactive <- Fig_4EF_interactive %>% layout(width = 900, height = 450)
Fig_4EF_interactive
```
Left: Box and whisker plot of the mediolateral position of dorsal ipsilateral premotor interneurons in each experiment. Right: Values of the dorsal ipsilateral median for each LG and TA experiment.


### Figure 4 figure supplement 1
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
## -- Figure 4 figure supplement 1 -- ##
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "low"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
panel <- ""
density <- 1
concatenate <- 1
Figure_4supp1i <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 7, 
                                       6, IN, N_A, Size, LG, TA, kernel, bin, method_used, titer_used, panel)


## -- rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_4supp1ii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 7, 
                                             6, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_4supp1iii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 7, 
                                              6, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_4supp1i, NULL, Figure_4supp1ii, NULL, Figure_4supp1iii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)

```
**Figure 4 figure supplement 1**: distribution of flexor and extensor premotor interneurons pooled across all LG and TA for low titre injections shown in the transverse plane (left) and as front (middle) and lateral (right) view along the rostrocaudal axis.

### Figure 4 figure supplement 2
```{r,echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 4}

titer_used <- c("high", "low")
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
method_used <- c("G_Rab_ChAT_RGT")
Animal <- 0

## -- A -- ##
panel <- "A"
name1 <- "LG-high"
name2 <- "LG-low"
density <- 1
concatenate <- 1
Figure_4supp1_Ai <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, color_range_ext2, color_lg, color_mg, 2, 
                                              0, IN, N_A, Size, name1, name2, kernel, bin, method_used, titer_used, panel)

kernel <- 1.5
Figure_4supp1_Aii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, color_range_ext2, color_gs, color_mg, 2, 
                                               0, IN, N_A, Size, kernel, bin, method_used, titer_used)

Figure_4supp1_Aiii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, color_range_ext2, color_gs, color_mg, 2, 
                                                0, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_4supp1_Ai, NULL, Figure_4supp1_Aii, NULL, Figure_4supp1_Aiii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)

```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}

## -- B -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
panel <- "B"
numbers <- 0
Figure_4Supp1B <- Corr_HedgesG(data_all, lg, N_A, method_used, titer_used, N, lim, panel, color_lg, color_mg, 11, 7, numbers)
Figure_4Supp1B
```

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}

## -- C -- ##  
Figure_4supp1C_interactive <- boxplot_per_sample_titer_interactive(data_all, method_used, titer_used, lg, color_lg, color_mg, ymin_d, ymax_950)
## -- D and H -- ## 
method_used <- "G_Rab_ChAT_RGT"
titer_used <- "high"
name1 <- "LG-high"
name2 <- "TA-high"
dat4supp1DH <- df_individual(data_all, method_used, titer_used, lg, ta, name1, name2) 
titer_used <- "low"
name1 <- "LG-low"
name2 <- "TA-low"
df <- df_individual(data_all, method_used, titer_used, lg, ta, name1, name2)
dat4supp1DH <- rbind(dat4supp1DH, df)

dat4supp1D <- dat4supp1DH  %>% filter(group == "LG-low" | group == "LG-high")
dat4supp1D$group <- factor(dat4supp1D$group, levels = c("LG-high", "LG-low"))
Figure_4supp1D_interactive <- Boxplot_individual_interactive(dat4supp1D, color_gs, color_mg, ymin_d, ymax_950)

Figure_4supp1CD_interactive <- subplot(Figure_4supp1C_interactive, Figure_4supp1D_interactive, widths = c(0.7,0.3), heights = c(1)) 
Figure_4supp1CD_interactive <- Figure_4supp1CD_interactive %>% layout(width = 900, height = 450)
Figure_4supp1CD_interactive

```

```{r,echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 4}
## -- E -- ##
kernel <- 6
bin <- 5
titer_used <- c("high", "low")
panel <- "E"
name1 <- "TA-high"
name2 <- "TA-low"
density <- 1
concatenate <- 1
Figure_4supp1_Ei <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, color_range_flex2, color_ta, color_pl, 2, 
                                              0, IN, N_A, Size, name1, name2, kernel, bin, method_used, titer_used, panel)

kernel <- 1.5
Figure_4supp1_Eii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_flex, color_range_flex, color_ta, color_pl, 2, 
                                               0, IN, N_A, Size, kernel, bin, method_used, titer_used)
Figure_4supp1_Eiii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, color_range_flex2, color_ta, color_pl, 2, 
                                                0, IN, N_A, Size, kernel, bin, titer_used, method_used)


cowplot::plot_grid(Figure_4supp1_Ei, NULL, Figure_4supp1_Eii, NULL, Figure_4supp1_Eiii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)

```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}

## -- F -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
panel <- "F"
numbers <- 0
Figure_4Supp1F <- Corr_HedgesG(data_all, ta, N_A, method_used, titer_used, N, lim, panel, color_ta, color_pl, 7, 6, numbers)
Figure_4Supp1F
```

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}


## -- G -- ##
Figure_4supp1G_interactive <- boxplot_per_sample_titer_interactive(data_all, method_used, titer_used, ta, color_ta, color_pl, ymin_d, ymax_950)

## -- H -- ##
dat4supp1H <- dat4supp1DH  %>% filter(group == "TA-low" | group == "TA-high")
dat4supp1H$group <- factor(dat4supp1H$group, levels = c("TA-high", "TA-low"))
Figure_4supp1H_interactive <- Boxplot_individual_interactive(dat4supp1H, color_ta, color_pl, ymin_d, ymax_950)


Figure_4supp1GH_interactive <- subplot(Figure_4supp1G_interactive, Figure_4supp1H_interactive, widths = c(0.7,0.3), heights = c(1)) 
Figure_4supp1GH_interactive <- Figure_4supp1GH_interactive %>% layout(width = 900, height = 450)
Figure_4supp1GH_interactive
```
**Figure 4 figure supplement 2**: High and low efficiency infections give rise to the same premotor interneurons distributions. Comparison of high and low titre injections are shown in A and E for LG and TA respectively. For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. Correlations between high and low titre experiments are high and effect sizes are low for both LG and TA muscles (B and F) The distributions are similar across experiments for both muscles (B and E) and the median values of the distributions in the ipsilateral dorsal quadrant are not different for high and low efficiency of infection (C and F).

### Figure 4 figure supplement 3
```{r, fig.align='center', fig.width= 5 , fig.height= 5}
data <- data_all  %>% filter(lab != "Salk")
animals <- distinct(data.frame(data$animal))
Animal <- as.character(animals$data.animal)
data_fig4supp3 <- data.frame(animal = c(), titer = c(), nb_MN = c(), nb_IN = c(), muscle = c())
for (i in 1:length(Animal)){
  df2 <- data  %>% filter(animal == Animal[i], identity == "MN")
  df3 <- data  %>% filter(animal == Animal[i], identity == "IN")
  muscles <- distinct(data.frame(df3$muscle))
  Muscle <- as.character(muscles$df3.muscle)
  for (j in 1:length(Muscle)){
    df3m <- df3  %>% filter(muscle == Muscle[j])
    df2m <- df2  %>% filter(muscle == Muscle[j])
    df3m <- data.frame(lapply(df3m, as.character), stringsAsFactors=FALSE)
    if (df3m$lab[1] == "UCL"){
      MNs <- nrow(df2m)*2
      INs <- nrow(df3m)*2
      titer_exp <- df3m$titer[1]
    }
    if (df3m$lab[1] == "UoG"){
      MNs <- nrow(df2m)*8
      INs <- nrow(df3m)*8
      titer_exp <- df3m$titer[1]
    }
    if (df3m$lab[1] == "MDC"){
      MNs <- nrow(df2m)
      INs <- nrow(df3m)
    }
    v <- c(Animal[i], titer_exp, as.numeric(MNs), as.numeric(INs), Muscle[j])
    data_fig4supp3 <- rbind(data_fig4supp3, v, stringsAsFactors = FALSE)
  }
}
colnames(data_fig4supp3) <- c("animal", "titer", "nb_MN","nb_IN", "muscle")
data_fig4supp3$nb_MN <- as.numeric(as.character(data_fig4supp3$nb_MN))
data_fig4supp3$nb_IN <- as.numeric(as.character(data_fig4supp3$nb_IN))
data_fig4supp3 <- data_fig4supp3  %>% filter(nb_MN != 0)

#Uncomment the three following lines to get the Coefficients, residuals and R-squared values
#model_log_transformed <- lm(formula = log10(nb_MN) ~ log10(nb_IN), data = data_fig4supp3) 
#fit_log <- summary(model_log_transformed)
#model_linear <- lm(formula = nb_MN ~ nb_IN, data = data_fig4supp3)
#fit_log

graph_fig4supp3 <- ggplot(data_fig4supp3, aes(x = log10(nb_MN), y = log10(nb_IN))) +
  geom_point(aes(shape = titer, color = titer), size = 2) +
  geom_smooth(aes(fill = "#76B947"), linetype = "solid", method = "lm", 
              formula = y ~ x, se = TRUE, color = "#76B947", size = 1, alpha = 0.2) +
  scale_linetype(name = NULL, guide = "legend") +
  scale_fill_identity(name = "fitted models", guide = "legend", labels = c("log-transformed")) +
  scale_color_manual(values = c("#979797", "#3B3B3B")) +
  #coord_trans(x="log10", y="log10") +
  guides(fill = guide_legend(order = 1)) + 
  scale_y_continuous(expand = c(0.0,0.05), limits = c(2, 4.5), breaks=seq(from = 2, to = 4.5, by = 0.5)) +
  scale_x_continuous(expand = c(0.0,0.05), limits = c(0.8,2.5), breaks=seq(from = 1, to = 2.5, by = 0.5)) +
  #stat_regline_equation(label.y = 4.2, aes(label = ..eq.label..)) + #If you want the equation of the model on the figure
  #stat_regline_equation(label.y = 4, aes(label = ..rr.label..)) + #if you want the R^2 on the fig
  ylab("log(Input INs)") +  
  xlab("log(Starter MNs)") +
  theme(axis.text=element_text(size=12, colour = "black")) +
  theme_classic() +
  theme(axis.line = element_line(colour = "black", size = 0.5, linetype = "solid")) +
  theme(axis.text = element_text(colour = 1, size = 12),
        legend.background = element_blank(), legend.spacing.y = unit(0.15, 'cm'),
        legend.margin = margin(2, 2, 2, 2),
        legend.box.background = element_rect(colour = "black"), legend.position = c(0.85, 0.20))

graph_fig4supp3
```

**Figure 4 figure supplement 3**: the relation between the number of primary infected motor neurons and premotor interneurons follow a power law y = axb with a = 295 (155, 561 confidence intervals) and b = 0.53 (0.37, 0.69 confidence intervals), R^2^ = 0.48. Motor neuron and interneuron numbers in each experiment are shown for high (red) and low (black) titre experiments. The fitted line represents a power law.

### The distribution of premotor interneurons is similar across different pairs of ankle flexors or extensors

Since it has been proposed that medio-lateral segregation of premotor interneurons is a general feature of flexor and extensor muscles, we analysed premotor interneurons of two more muscles controlling the movement of the ankle joint, PL and MG using the same viral strategy employed for the LG-TA injections (Figure 1D). The distributions of premotor interneurons of LG (6 LG-MG injections, 8 LG-TA injections and 4 single LG injections) and MG (6 LG-MG injections and 4 MG injections) did not reveal any difference in spatial organization (Figure 5A, individual experiments are shown in Figure 12- figure supplements 3,4,6). The Hedges' G for the mediolateral positions in the ipsilateral dorsal quadrants were computed for each pair of experiment and had a median value of 0.01 (IQR -0.12,0.16). Throughout the cord, the positions of rabies-labelled neurons were highly correlated (Figure 5B, r≥0.74) and reproducible along the medio-lateral axis (Figure 5C), with median values of the medio-lateral position across experiments of 322 μm for LG and 327 μm for MG, (bootstrapped median Hedges' G= -0.06, IQR -0.03, -0.10, Figure 5D). The same result was observed for TA and PL premotor interneurons (Figure 5E; 2 TA-PL injections, 8 TA-LG injections, 3 TA single injections and 3 PL single injections, individual experiments are shown in Figure 12- figure supplements 5,7). The median of the Hedges' G coefficients for all the pairs of experiments was-0.01 (IQR -0.20, 0.18), with high correlation values between experiments (Figure 5F; r≥0.66). The medians of the mediolateral pooled distributions were 315 μm for TA and 330 μm for PL, similar medio-lateral distributions (Figure 5G) and median values were observed (Figure 5H). Hierarchical bootstrap of the data from TA and PL injections resulted in a median Hedges' G of -0.09 (IQR -0.15, -0.03). Together, these data show that premotor interneuron maps obtained using ΔG-RabV muscle injection in *ChAT::Cre +/-; RΦGT+/-* mice (Figure 1D) do not reveal any difference in the positional organization of interneurons controlling the activity of any of the main flexor and extensor muscles of the ankle.

### Figure 5

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 4}
method_used <- "G_Rab_ChAT_RGT"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
titer_used <- "low"
df2 <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
df <- rbind(df, df2)
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
## -- 5A coronal -- ##
panel <- "A"
density <- 1
concatenate <- 1
Figure_5Ai <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 18, 
                                        10, IN, N_A, Size, LG, MG, kernel, bin, method_used, titer_used, panel)

## -- 5A rostro-caudal ipsi-lateral -- ##
kernel <- 1.5
Figure_5Aii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 18, 
                                         10, IN, N_A, Size, kernel, bin, method_used, titer_used)
## -- 5A rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_5Aiii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 18, 
                                          10, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_5Ai, NULL, Figure_5Aii, NULL, Figure_5Aiii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}
## -- 5B -- ##
panel <- "B"
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 0
titer_used <- 0
Figure_5B <- Corr_HedgesG(data_all, lg, mg, method_used, titer_used, N, lim, panel, color_lg, color_mg, 18, 10, numbers)
Figure_5B
```

**Figure 5**: Pairs of flexor or extensor muscles show similar distributions of premotor interneurons. A, comparison of pooled data from extensor muscles LG and MG injections. B: correlation and absolute value of Hedges' G coefficients across all experiments.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}
## -- 5C -- ##
titer_used <- 0
Figure_5C_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, lg, mg, color_lg, color_mg, ymin_d, ymax_950)
## -- 5D -- ##
dat5D <- df_individual(data_all, method_used, titer_used, lg, mg, LG, MG) 
Figure_5D_interactive <- Boxplot_individual_interactive(dat5D, color_lg, color_mg, ymin_d, ymax_950)

Fig_5CD_interactive <- subplot(Figure_5C_interactive, Figure_5D_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_5CD_interactive <- Fig_5CD_interactive %>% layout(width = 900, height = 450)
Fig_5CD_interactive
```

Left: box and whisker plots of the mediolateral position of dorsal ipsilateral premotor interneurons for each experiment and distribution of median values (right).

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 4}

## -- 5E coronal -- ##
panel <- "E"
density <- 1
concatenate <- 1
kernel <- 6
Figure_5Ei <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, pl, color_range_ext, color_range_ext2, color_ta, color_pl, 13, 
                                        5, IN, N_A, Size, TA, PL, kernel, bin, method_used, titer_used, panel)

## -- 5E rostro-caudal ipsi- contra-- ##
kernel <- 1.5
Figure_5Eii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 13, 
                                         5, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- 5E rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_5Eiii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 13, 
                                          5, IN, N_A, Size, kernel, bin, titer_used, method_used)


cowplot::plot_grid(Figure_5Ei, NULL, Figure_5Eii, NULL, Figure_5Eiii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}
## -- 5F -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 0
titer_used <- 0
panel <- "F"
Figure_5F <- Corr_HedgesG(data_all, ta, pl, method_used, titer_used, N, lim, panel, color_ta, color_pl, 13, 5, numbers)
Figure_5F
```
E: similar plot as A, showing the distribution of premotor interneurons following injections of the flexor muscles TA and PL. Correlations and absolute value of Hedges' G coefficients across each experiment are shown in F.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}

## -- 5G -- ##
titer_used <- 0
Figure_5G_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, ta, pl, color_ta, color_pl, ymin_d, ymax_950)
## -- 5H -- ##
dat5H <- df_individual(data_all, method_used, titer_used, ta, pl, TA, PL) 
dat5H$group <- factor(dat5H$group, levels = c("TA", "PL"))
Figure_5H_interactive <- Boxplot_individual_interactive(dat5H, color_ta, color_pl, ymin_d, ymax_950)

Fig_5GH_interactive <- subplot(Figure_5G_interactive, Figure_5H_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_5GH_interactive <- Fig_5GH_interactive %>% layout(width = 900, height = 450)
Fig_5GH_interactive
```
Mediolateral distribution and positions of the median for each experiment, respectively. For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments.

### Spatial distribution of GlyT2^off^ and GlyT2^on^ premotor interneurons

Next, we examined whether there are differences in the spatial organization of GlyT2^off^ vs GlyT2^on^ premotor interneurons, where the GlyT2^off^ population will largely comprise excitatory neurons, as well as some purely GABAergic interneurons. We performed single LG or TA injections of ΔG-RabV/mCherry in *ChAT::Cre +/-; RΦGT+/-* mice carrying an allele expressing GFP under the control of the neuronal glycine transporter (GlyT2; Zeilhofer et al., 2005).

We monitored the GlyT2 status of cells while performing transsynaptic labelling from LG and TA motor neurons. First, we compared the distribution of premotor GlyT2^off^ interneurons (Figures 6A-C) and then examined the distribution of GlyT2^on^ (Figures 6D-F). The distribution of GlyT2^off^ premotor interneurons was the same for LG and TA motor neurons (4 LG and 3 TA single injections, Figure 6A). The medians of the medio-lateral position in the dorsal ipsilateral cord were 306 μm and 326 μm for LG and TA respectively (hierarchical bootstrapped Hedges'G = -0.03, IQR -0.08, 0.01) and -112 μm and 74 μm (hierarchical bootstrapped Hedges'G = -0.14, IQR -0.24, -0.04) in the ventral spinal cord (Figures 6 B-C). Similarly, we did not observe segregation in the distribution of GlyT2^on^ LG and TA premotor interneurons (Figure 6D). The medians of the medio-lateral coordinates of the dorsal GlyT2^on^ interneurons were 303 μm for LG and 346 μm for TA (hierarchical bootstrapped Hedges'G = -0.30, IQR -0.35, -0.24), while for ventral interneurons were 395 μm for LG and 437 μm for TA (hierarchical bootstrapped Hedges'G = -0.33, IQR -0.40, 0.27, Figures 6E-F). High correlation values (r\>0.79). between all individual experiments underscored the conserved positional organization of LG and TA premotor interneurons. These data indicate that there is no significant difference in the distribution of GlyT2^off^ and GlyT2^on^ premotor interneurons controlling the activity of flexor and extensor muscles.

Finally, we compared the distributions of GlyT2^off^ and GlyT2^on^ premotor interneurons separately for each muscle, LG (Figures 6G-I) and TA (Figures 6J-L). No differences were observed for the medio-lateral distribution of inhibitory and excitatory dorsal premotor interneurons (medians for LG: GlyT2^on^ = 303 μm and GlyT2^off^ = 306 μm; hierarchical bootstrapped Hedges'G = -0.06, IQR 0.03, 0.09. Medians for TA: GlyT2^on^ = 346 and GlyT2^off^ = 326 μm; hierarchical bootstrapped Hedges'G = 0.15, IQR 0.08, 0.23). In contrast, ventral ipsilateral GlyT2^on^ were more abundant than GlyT2^off^ for both LG (Figure 6H-I) and TA (Figure 6K-L). Conversely, GlyT2^off^ premotor interneurons dominated the ventral contralateral side (Figure 6H, K; medians for LG: GlyT2^off^ = -112 μm and GlyT2^on^ = 395 μm; hierarchical bootstrapped Hedges'G = 0.72, IQR 0.61, 0.84. Medians for TA: GlyT2^off^ = 74 μm and GlyT2^on^ = 437 μm; hierarchical bootstrapped Hedges'G = 0.91, IQR 0.82, 0.99). Overall, the data show a clear segregation in the distributions of GlyT2^off^ and GlyT2^on^ premotor interneurons in the ventral half of the spinal cord: GlyT2^on^ interneurons are almost exclusively found in the ipsilateral side while GlyT2^off^ interneurons also present a prominent peak in the contralateral side (Figures 6G-L). These observations are reflected in the overall low correlation value in the position of GlyT2^off^ and GlyT2^on^ premotor interneurons for both LG and TA premotor interneurons (r≤0.4).

Taken together, these findings indicate that while we are able to detect significant differences in the positional organization of premotor interneurons with different transmitter phenotype, we found that the organization of flexor and extensor premotor circuits were always intermingled regardless of their neurotransmitter status.

### Figure 6
```{r, echo=FALSE, warning=FALSE}
## -- Figure 6 -- ##
data_fig6 <- data_frame(x = c(), y = c(), z= c(), animal = c(),  muscle = c(), GlyT2 = c(), lab = c(), titer = c(), method = c(),identity = c())
method_used <- "G_Rab_ChAT_RGT"
## -- exc and inh premotor INs -- ##
setwd(str_c(maindir, "", "/data/lg_exc_chat"))
titer <- "NA"
method <- "G_Rab_ChAT_RGT"
data_fig6 <- Data_building(data_fig6, titer, method_used) 
setwd(str_c(maindir, "", "/data/ta_exc_chat"))
data_fig6 <- Data_building(data_fig6, titer, method_used)
setwd(str_c(maindir, "", "/data/lg_inh_chat"))
data_fig6 <- Data_building(data_fig6, titer, method_used)
setwd(str_c(maindir, "", "/data/ta_inh_chat"))
data_fig6 <- Data_building(data_fig6, titer, method_used)
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 24}

## -- 6A -- ##
panel <- ""
kernel <- 6
bin <- 5
nt <- "GlyT2-off"
Figure_6A <- complete_graph_xy_density_fig6(spinal, data_fig6, lg, ta, color_lg, color_ta, kernel, bin, nt, panel)

## -- 6B -- ##
transmitter <- "GlyT2-off"
## -- dorsal -- ##
hemicord <- "dorsal"
Figure_6B_dorsal <- boxplot_per_sample_fig6(data_fig6, lg, ta, color_lg, color_ta, ymin_vd, ymax_850, transmitter, hemicord)
Figure_6B_dorsal <- Figure_6B_dorsal +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"OFF")) +
  annotate("text", x = 6, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"OFF")) 
## -- ventral -- ##
hemicord <- "ventral"
Figure_6B_ventral <- boxplot_per_sample_fig6(data_fig6, lg, ta, color_lg, color_ta, ymin_vd, ymax_850, transmitter, hemicord)
Figure_6B_ventral <- Figure_6B_ventral +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"OFF")) +
  annotate("text", x = 6, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"OFF")) 


 ## -- 6C -- ##
## -- dorsal -- ##
hemicord <- "dorsal"
dat_6C_dorsal <- df_individual_fig6(data_fig6, method_used, transmitter, hemicord, lg, ta, LG, TA)
dat_6C_dorsal$group <- factor(dat_6C_dorsal$group, levels = c("LG", "TA"))
Figure_6C_dorsal <- Boxplot_individual(dat_6C_dorsal, color_lg, color_ta, ymin_vd, ymax_850)
## -- ventral -- ##
hemicord <- "ventral"
dat_6C_ventral <- df_individual_fig6(data_fig6, method_used, transmitter, hemicord, lg, ta, LG, TA)
dat_6C_ventral$group <- factor(dat_6C_ventral$group, levels = c("LG", "TA"))
Figure_6C_ventral <- Boxplot_individual(dat_6C_ventral, color_lg, color_ta, ymin_vd, ymax_850)

Figure_6A <- ggarrange(NULL, Figure_6A, NULL,
                       ncol = 1, nrow = 3,  align = "hv",
                       widths = c(5), heights = c(1,4,1), common.legend = TRUE, 
                       labels ="A", font.label = list(size = 28, color = "black"))

Figure_6B <- ggarrange(NULL, Figure_6B_dorsal, NULL, Figure_6B_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.8,7), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "B", font.label = list(size = 28, 
                       color = "black")) 
Figure_6C <- ggarrange(NULL, Figure_6C_dorsal, NULL, NULL, NULL, Figure_6C_ventral,NULL, NULL,
                       ncol = 2, nrow = 4,  align = "hv",
                       widths = c(0.7,3), heights = c(2.5,0.23,2.5, 0.23),
                       common.legend = TRUE, labels = "C", font.label = list(size = 28,
                       color = "black")) 

## -- 6D -- ##
panel <- ""
kernel <- 6
bin <- 5
nt <- "GlyT2-on"
Figure_6D <- complete_graph_xy_density_fig6(spinal, data_fig6, lg, ta, color_lg, color_ta, kernel, bin, nt, panel)

## -- 6E -- ##
transmitter <- "GlyT2-on"
## -- dorsal -- ##
hemicord <- "dorsal"
Figure_6E_dorsal <- boxplot_per_sample_fig6(data_fig6, lg, ta, color_lg, color_ta, ymin_vd, ymax_850, transmitter, hemicord)
Figure_6E_dorsal <- Figure_6E_dorsal +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"ON")) +
  annotate("text", x = 6, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"ON")) 

## -- ventral -- ##
hemicord <- "ventral"
Figure_6E_ventral <- boxplot_per_sample_fig6(data_fig6, lg, ta, color_lg, color_ta, ymin_vd, ymax_850, transmitter, hemicord)
Figure_6E_ventral <- Figure_6E_ventral +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"ON")) +
  annotate("text", x = 6, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"ON")) 

## -- 6F -- ##
## -- dorsal -- ##
hemicord <- "dorsal"
dat_6F_dorsal <- df_individual_fig6(data_fig6, method_used, transmitter, hemicord, lg, ta, LG, TA)
dat_6F_dorsal$group <- factor(dat_6F_dorsal$group, levels = c("LG", "TA"))
Figure_6F_dorsal <- Boxplot_individual(dat_6F_dorsal, color_lg, color_ta, ymin_vd, ymax_850)
## -- ventral -- ##
hemicord <- "ventral"
dat_6F_ventral <- df_individual_fig6(data_fig6, method_used, transmitter, hemicord, lg, ta, LG, TA)
dat_6F_ventral$group <- factor(dat_6F_ventral$group, levels = c("LG", "TA"))
Figure_6F_ventral <- Boxplot_individual(dat_6F_ventral, color_lg, color_ta, ymin_vd, ymax_850)


Figure_6D <- ggarrange(NULL, Figure_6D, NULL,
                       ncol = 1, nrow = 3,  align = "hv",
                       widths = c(5), heights = c(1,4,1), common.legend = TRUE, labels = "D", font.label = list(size = 28, color = "black"))
Figure_6E <- ggarrange(NULL, Figure_6E_dorsal, NULL, Figure_6E_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.8,7), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "E", font.label = list(size = 28, color = "black")) 
Figure_6F <- ggarrange(NULL, Figure_6F_dorsal, NULL, NULL, NULL, Figure_6F_ventral,NULL, NULL,
                       ncol = 2, nrow = 4,  align = "hv",
                       widths = c(0.5,3), heights = c(2.5,0.23,2.5, 0.23),
                       common.legend = TRUE, labels = "F", font.label = list(size = 28, color = "black")) 


## -- 6G -- ##
panel <- ""
kernel <- 6
bin <- 5
nt <- "GlyT2-on"
nt2 <- "GlyT2-off"
Figure_6G <- complete_graph_xy_density_fig6(spinal, data_fig6, nt, nt2, color_glyt2_on, color_glyt2_off, kernel, bin, lg, panel)


## -- 6H -- ##
## -- dorsal -- ##
hemicord <- "dorsal"
Figure_6H_dorsal <- boxplot_per_sample_fig6_glyT2(data_fig6, lg, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850, hemicord)
Figure_6H_dorsal <- Figure_6H_dorsal +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"ON")) +
  annotate("text", x = 6.5, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"OFF")) 

## -- ventral -- ##
hemicord <- "ventral"
Figure_6H_ventral <- boxplot_per_sample_fig6_glyT2(data_fig6, lg, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850, hemicord)
Figure_6H_ventral <- Figure_6H_ventral +
  annotate("text", x = 2.5, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"ON")) +
  annotate("text", x = 6.5, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"OFF")) 

## -- 6I -- ##
## -- dorsal -- ##
dat_6I_dorsal <- Selection_glyT2(dat_6C_dorsal, dat_6F_dorsal, LG)
dat_6I_dorsal$group <- factor(dat_6I_dorsal$group, levels = c("GlyT2<sup>ON</sup>", "GlyT2<sup>OFF</sup>"))
Figure_6I_dorsal <- Boxplot_individual(dat_6I_dorsal, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850)
Figure_6I_dorsal <- Figure_6I_dorsal + scale_x_discrete(labels=c(expression("GlyT2"^"ON"),expression("GlyT2"^"OFF")))  + theme(axis.text.x = element_text(angle=45, size = 9, hjust = 1))

## -- ventral -- ##
dat_6I_ventral <- Selection_glyT2(dat_6C_ventral, dat_6F_ventral, LG)
dat_6I_ventral$group <- factor(dat_6I_ventral$group, levels = c("GlyT2<sup>ON</sup>", "GlyT2<sup>OFF</sup>"))
Figure_6I_ventral <- Boxplot_individual(dat_6I_ventral, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850)
Figure_6I_ventral <- Figure_6I_ventral + scale_x_discrete(labels=c(expression("GlyT2"^"ON"),expression("GlyT2"^"OFF")))  + theme(axis.text.x = element_text(angle=45, size = 9, hjust = 1))

Figure_6G <- ggarrange(NULL, Figure_6G, NULL,
                       ncol = 1, nrow = 3,  align = "hv",
                       widths = c(5), heights = c(1,4,1), common.legend = TRUE, labels = "G", font.label = list(size = 28, color = "black"))
Figure_6H <- ggarrange(NULL, Figure_6H_dorsal, NULL, Figure_6H_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.8,7), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "H", font.label = list(size = 28, color = "black")) 
Figure_6I <- ggarrange(NULL, Figure_6I_dorsal, NULL, Figure_6I_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.5,3), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "I", font.label = list(size = 28, color = "black"))

## -- 6J -- ##
panel <- ""
kernel <- 6
bin <- 5
nt <- "GlyT2-on"
nt2 <- "GlyT2-off"
Figure_6J <- complete_graph_xy_density_fig6(spinal, data_fig6, nt, nt2, color_glyt2_on, color_glyt2_off, kernel, bin, ta, panel)


## -- 6K -- ##
## -- dorsal -- ##
hemicord <- "dorsal"
Figure_6K_dorsal <- boxplot_per_sample_fig6_glyT2(data_fig6, ta, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850, hemicord)
Figure_6K_dorsal <- Figure_6K_dorsal +
  annotate("text", x = 2, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"ON")) +
  annotate("text", x = 5, y = 780, size = 4, colour = 'black', label = expression("dorsal GlyT2"^"OFF")) 

## -- ventral -- ##
hemicord <- "ventral"
Figure_6K_ventral <- boxplot_per_sample_fig6_glyT2(data_fig6, ta, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850, hemicord)
Figure_6K_ventral <- Figure_6K_ventral +
  annotate("text", x = 2, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"ON")) +
  annotate("text", x = 5, y = 780, size = 4, colour = 'black', label = expression("ventral GlyT2"^"OFF")) 

## -- Figure 6L -- ##
## -- dorsal -- ##
dat_6L_dorsal <- Selection_glyT2(dat_6C_dorsal, dat_6F_dorsal, TA)
dat_6L_dorsal$group <- factor(dat_6L_dorsal$group, levels = c("GlyT2<sup>ON</sup>", "GlyT2<sup>OFF</sup>"))
Figure_6L_dorsal <- Boxplot_individual(dat_6L_dorsal, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850)
Figure_6L_dorsal <- Figure_6L_dorsal + scale_x_discrete(labels=c(expression("GlyT2"^"ON"),expression("GlyT2"^"OFF")))  + theme(axis.text.x = element_text(angle=45, size = 9, hjust = 1))

## -- ventral -- ##
dat_6L_ventral <- Selection_glyT2(dat_6C_ventral, dat_6F_ventral, TA)
dat_6L_ventral$group <- factor(dat_6L_ventral$group, levels = c("GlyT2<sup>ON</sup>", "GlyT2<sup>OFF</sup>"))
Figure_6L_ventral <- Boxplot_individual(dat_6L_ventral, color_glyt2_on, color_glyt2_off, ymin_vd, ymax_850)
Figure_6L_ventral <- Figure_6L_ventral + scale_x_discrete(labels=c(expression("GlyT2"^"ON"),expression("GlyT2"^"OFF"))) + theme(axis.text.x = element_text(angle=45, size = 9, hjust = 1))

Figure_6J <- ggarrange(NULL, Figure_6J, NULL,
                       ncol = 1, nrow = 3,  align = "hv",
                       widths = c(5), heights = c(1,4,1), common.legend = TRUE, labels = "J", font.label = list(size = 28, color = "black"))
Figure_6K <- ggarrange(NULL, Figure_6K_dorsal, NULL, Figure_6K_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.8,7), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "K", font.label = list(size = 28, color = "black")) 
Figure_6L <- ggarrange(NULL, Figure_6L_dorsal, NULL, Figure_6L_ventral,
                       ncol = 2, nrow = 2,  align = "hv",
                       widths = c(0.5,3), heights = c(2.5,2.5),
                       common.legend = TRUE, labels = "L", font.label = list(size = 28, color = "black"))

cowplot::plot_grid(Figure_6A, NULL, Figure_6B, Figure_6C,Figure_6D, NULL, Figure_6E, Figure_6F,
                   Figure_6G, NULL, Figure_6H, Figure_6I,Figure_6J, NULL, Figure_6K, Figure_6L,
                   nrow=4,ncol=4, rel_widths = c(5.25,-0.4,6,2.5),rel_heights = c(5, 5, 5, 5),
                   align = 'hv')
                   
```

**Figure 6**: GlyT2^on^ and GlyT2^off^ premotor interneurons are distributed similarly for TA and LG. A, B: Distribution of GlyT2^off^ (A) and GlyT2^on^ (D) premotor interneurons following LG and TA injections pooled from 4 LG and 3 TA experiments in *GlyT2-EGFP::ChAT-Cre* mice crossed with *RΦGT* mice, indicating that neither class of premotor interneurons is segregated across muscles. Boxplots and violin plots (B for GlyT2^off^ and E for GlyT2^on^) show uniformity of distribution across experiments in both the dorsal (top) and ventral (bottom) halves of the cord. C (GlyT2^off^) and F (GlyT2^on^) show boxplots and individual values for the medians of the mediolateral distributions restricted to dorsal (top) or ventral (bottom) part of the cord. Ventral premotor GlyT2^off^ and GlyT2^on^ interneurons are differentially distributed. Comparison of excitatory and inhibitory premotor interneurons in LG (G) and TA (J) muscles are similar in the dorsal cord, but differ in the ventral cord, where most ipsilateral premotor interneurons are GlyT2^on^, and the majority of contralateral premotor interneurons are GlyT2^off^. Boxplots and violin plots of individual experiments are shown in H for LG and K for TA, highlighting the mediolateral differences in the ventral cord. The medians of the ventral and dorsal distributions are shown in I for LG and L for TA. For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments.

### Flexor and extensor premotor interneurons tracing in *Olig2::Cre +/-; RΦGT+/-* mice

An important consideration concerning the use of our genetic approach for G complementation is the expression specificity of the Cre driver: recombination in multiple neuronal subtypes can potentially result in loss of monosynaptic restriction and rabies transfer across multiple synapses (Figures 1D and 1E). In the spinal cord, *ChAT::Cre* is not only expressed in motor neurons but also cholinergic interneurons, including medial partition cells (V0c neurons) that have prominent projections to motor neurons (Zagoraiou et al., 2009). Therefore, given that under our experimental conditions, V0c neurons express G and are presynaptic to motor neurons, they could permit disynaptic rabies transfer: first from motor neurons to V0c neurons and second from V0c neurons to their presynaptic partners. However, it is important to note that V0c presynaptic partners have been previously characterised using rabies monosynaptic tracing and comprise many interneurons (and/or axonal arborisations) located in the dorsal laminae of the spinal cord (Zampieri et al., 2014), an area that is largely devoid of rabies labelling in our experiments.

In order to test whether disynaptic transfer from premotor interneurons is affecting our analysis, we performed a set of experiments (4 gastrocnemius, GS and 3 TA injections) using the *Olig2::Cre* (Dessaud et al., 2007) instead of the *ChAT::Cre* line (Figure 1E). This line would ensure recombination in motor neurons but not in V0c or other cholinergic interneurons. However, Olig2 is transiently expressed during embryonic development in subsets of p2 and p3 progenitors (Chen et al., 2011). We reasoned that if additional transsynaptic transfer from premotor interneurons is significantly affecting our results, using a different Cre line to drive G expression in a non-overlapping subset of premotor interneurons should result in different labelling patterns. We performed monosynaptic tracing experiments after single injections of ΔG-RabV/mCherry in either the TA or GS muscles of P4 *Olig2::Cre +/-; RΦGT+/-* mice. Six days following injection, we observed interneuron labelling with a pattern similar to that of those performed in *ChAT::Cre* mice (Figure 7A-B). There was no difference in the positional organization of flexor and extensor premotor interneurons in the transverse plane (Figures 7C-E individual experiments are shown in Figure 12- figure supplements 11-12) as well as along the rostro-caudal axis (Figure 7-figure supplement 1), with median values along the mediolateral axis of 295 μm for GS and 310 μm for TA (bootstrapped Hedges' G = -0.02, IQR -0.07,0.03).Comparison of the premotor maps obtained from *ChAT::Cre* and *Olig2::Cre* experiments showed that interneuron distributions were indistinguishable, as shown by the high correlation values across mouse lines and muscles (r\>0.9) and the low values of hierarchical bootstraps of the Hedges' G coefficients across muscles and *Olig2::Cre* and *ChAT::Cre* injections (Figure 7F). In addition, the median interneuron positions along the medio-lateral axis for each experiment were similar between *ChAT::Cre* (322 μm for GS and 315 μm for TA, including high and low efficiency experiments) and *Olig2::Cre* (295 μm for GS and 310 μm for TA) animals injected in the same muscle (Figure 7G, Hedges' G =0.14 (IQR 0.09, 0.18) for GS pairs and 0.13 (IQR 0.08, 0.18) for TA pairs). Thus, these results indicate that under our experimental conditions the results of tracing experiments done in *ChAT::Cre +/-; RΦGT+/-* and *Olig2::Cre +/-; RΦGT+/-* mice are unlikely to be influenced by disynaptic rabies transfer from spinal premotor interneurons.

### Figure 7
```{r, fig.align='left', fig.width= 12 , fig.height= 4}
## -- 7AB -- ##
fig7ab <- readPNG("~/figures/Figure_7AB.png")
Fig7AB <- ggplot() + 
  background_image(fig7ab) +
  coord_equal(ratio = 0.320) +
  annotation_custom(rasterGrob(fig7ab, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig7AB
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 15 , fig.height= 4}
method_used <- "G_Rab_Olig2_RGT"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)

## -- 7C -- ##
panel <- "C"
density <- 0
concatenate <- 0
Figure_7C <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext2, transpa, color_gs, transpa, 4, 
                                       0, IN, N_A, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)

## -- 7E -- ##
panel <- "E"
density <- 1
concatenate <- 1
Figure_7E <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                       3, IN, N_A, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
## -- 7D -- ##
panel <- "D"
density <- 0
concatenate <- 0
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used), muscle == "ta")
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
Figure_7D <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex2, transpa, color_ta, transpa, 3, 
                                       0, IN, N_A, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)

cowplot::plot_grid(Figure_7C, NULL, Figure_7D, NULL, Figure_7E,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5.17,-0.15, 5.17,-0.15,5.17), rel_heights = 3)
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}

## -- 7F -- ##
data <- data_all %>% filter(method == "G_Rab_Olig2_RGT" | method == "G_Rab_ChAT_RGT")
data <- data %>% filter(muscle == lg | muscle == gs | muscle == ta)
animals <- distinct(data.frame(data$muscle))
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
titer_used <- 0

df_Hedges <- get_Hedges_boot(titer_used, data, R)
df_corr <- get_Corr_concatenate(data, titer_used, N, lim)
if (identical(df_corr$method_1, df_Hedges$method_1) & identical(df_corr$method_1,df_Hedges$method_1)) {
  df_corr_Hedges <- cbind(df_corr, df_Hedges$HedgesGcoef, stringsAsFactors = FALSE)
} else { print("error in correlation or Hedges G oefficient order")}
colnames(df_corr_Hedges) <- c("sample_1", "sample_2", "CorrCoef", "HedgesG") # Function "organise_Corr_hedges" is built to use dataframe with these exact colonnes' name. 
remove <- c("_RGT","G_Rab_")
df_corr_Hedges$sample_1 <- str_remove_all(df_corr_Hedges$sample_1, paste(remove, collapse = "|"))
df_corr_Hedges$sample_2 <- str_remove_all(df_corr_Hedges$sample_2, paste(remove, collapse = "|"))
df_corr_Hedges <- organise_Corr_hedges(df_corr_Hedges)
numbers <- 1
color1 <- c(color_lg, color_ta)
color2 <- c(color_gs, color_ta)
panel <- "F"
Figure_7F <- Corr_HedgesG_methods(df_corr_Hedges, panel, color1, color2, 2, 2, numbers)
Figure_7F

```

**Figure 7**: The distribution of flexor and extensor premotor INs is similar in *Olig2::Cre; RΦGT*. A, B: Single lumbar sections form animals injected in the GS (A) or TA (B) muscles (scale bars 300 μm). C - E: Overlay of individual GS (C) and TA (D) experiments and pooled experiments (E). For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. F: Correlation coefficients and absolute value of hierarchical bootstrapped Hedges' G effect sizes between injections of different muscles and using a different driver for Cre expression.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=5 ,fig.height=2}
## -- 7G -- ##
colors_vector <- c("#006eb8", "#006eb8", "#eda911", "#eda911")
titer_used <- 0
method_used <- "G_Rab_Olig2_RGT"
dat7G <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA) 
dat7G <- dat7G %>% mutate(group = ifelse(group == "GS", "gs_Olig2::cre", "ta_Olig2::cre"))
method_used <- "G_Rab_ChAT_RGT"
df <- df_individual(data_all, method_used, titer_used, lg, ta , LG, TA)
df <- df %>% mutate(group = ifelse(group == "LG", "lg_ChAT::cre", "ta_ChAT::cre"))
dat7G <- rbind(dat7G, df)
dat7G$group <- factor(dat7G$group, levels = c("lg_ChAT::cre","gs_Olig2::cre","ta_ChAT::cre", "ta_Olig2::cre"))
Figure_7G_interactive <- Boxplot_methods_interactive(dat7G, colors_vector, ymin_d, ymax_950)

Fig_7G_interactive <- subplot(Figure_7G_interactive, widths = c(1), heights = c(1)) 
Fig_7G_interactive <- Fig_7G_interactive %>% layout(width = 500, height = 450)
Fig_7G_interactive
```

Box and whisker plots of median values of all the medio-lateral distributions in the dorsal ipsilateral quadrant.

### Figure 7 figure supplement 1

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
method_used <- "G_Rab_Olig2_RGT"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)
panel <- ""
density <- 1
concatenate <- 1
Figure_7supp1i <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                       3, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)

## -- rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_7supp1ii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4,
                  3, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_7supp1iii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 3, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_7supp1i, NULL, Figure_7supp1ii, NULL, Figure_7supp1iii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```

**Figure 7 figure supplement 1**: distribution of flexor and extensor premotor interneurons pooled across GS and TA injections performed in *Olig2::Cre; RΦGT* mice. Distributions are shown in the transverse plane (left) and as front (middle) and lateral (right) view along the rostrocaudal axis.

### Flexor and extensor premotor interneurons tracing with AAV complementation methods

In contrast to previous findings using AAV-G (Tripodi et al., 2011), we found extensive intermingling of flexor and extensor premotor interneurons when using Cre-based methods to genetically-express G-protein in order to complement rabies replication (Figure 1D, E). Therefore, we sought to replicate the previous findings using AAV-G complementation strategies (Figure 1A and B). We first performed injections of 1 μl of a 3:1 AAV-G: ΔG-RabV mixture in GS and TA muscles of P2 wild-type mice (Figure 1A;(Tripodi et al., 2011)). Two experiments were performed on GS only, with similar protocol, and GS single injections were pooled with the data resulting from the double injections. Ten days following injection, consistent labelling with both viruses was observed throughout the lumbar region (L1: Figure 8A, and L5: Figure 8B). Analysis of rabies-labelled interneurons did not reveal any apparent difference in the distributions of premotor circuits controlling the activity of antagonist muscles in the transverse plane as well as along the rostro-caudal axis (Figure 8C, individual experiments are shown in Figure 12- figure supplements 13). The Hedges' G coefficients of premotor interneuron distributions in the ipsilateral dorsal quadrant for all the experiments had a median of -0.02 (IQR -0.29, 0.28), indicating high uniformity across experiments, as also shown by the high correlation values of positional coordinates of flexor and extensor interneurons (Figure 8D). When the analysis was restricted to the medio-lateral positions in the ipsilateral dorsal quadrant, the median distances from the midline were 276 μm for GS and 321 μm for TA (Figure 8E, F), a negligible shift compared to the typical interneuron soma diameter. These findings obtained using AAV-G complementation are therefore consistent with the pattern of premotor interneuron mixing identified using genetic-delivery of G-protein.

### Figure 8
```{r, fig.align='left', fig.width= 12 , fig.height= 6}
## -- 8AB -- ##
fig8ab <- readPNG("~/figures/Figure_8AB.png")
Fig8AB <- ggplot() + 
  background_image(fig8ab) +
  coord_equal(ratio = 0.490) +
  annotation_custom(rasterGrob(fig8ab, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig8AB
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', , fig.width= 12 , fig.height= 4}
## -- Figure 8 -- ##
method_used <- "AAV-G_WT"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)

## -- 8C coronal -- ##
panel <- "C"
density <- 1
concatenate <- 1
Figure_8Ci <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                        2, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)

## -- 8C rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_8Cii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                         2, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- 8C rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_8Ciii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                          2, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_8Ci, NULL, Figure_8Cii, NULL, Figure_8Ciii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)

```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}

## -- 8D -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 1
panel <- "D"
Figure_8D <- Corr_HedgesG(data_all, gs, ta, method_used, titer_used, N, lim, panel, color_gs, color_ta, 4, 2, numbers)
Figure_8D
```

**Figure 8**: Injection of AAV-G and RabV into GS and TA muscles does not reveal segregation of premotor interneurons. Example of double infected upper (A and lower (B) lumbar sections. Transverse and longitudinal pooled distributions of premotor interneurons from two experiments are overlapping in all quadrants (C). For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. Individual experiments are highly correlated and have low absolute value of Hedges' G effect sizes (D). Scale bars 300 μm.


```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}

## -- 8E -- ##
Figure_8E_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, gs, ta, color_gs, color_ta, ymin_d, ymax_950)
## -- 8F -- ##
dat8F <- df_individual(data_all, method_used, titer_used, gs, ta, GS, TA) 
Figure_8F_interactive <- Boxplot_individual_interactive(dat8F, color_gs, color_ta, ymin_d, ymax_950)

Fig_8EF_interactive <- subplot(Figure_8E_interactive, Figure_8F_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_8EF_interactive <- Fig_8EF_interactive %>% layout(width = 900, height = 450)
Fig_8EF_interactive

```


Box and whisker plots of median values of all the medio-lateral distributions.

<br><br> In these AAV-G complementation experiments, we observed extensive labelling of superficial dorsal interneurons, a feature that was absent when using genetic complementation methods (compare Figures 8C with 3C, 4C and 7E). We therefore reasoned that contribution of transsynaptic transfer from sensory afferents, precluded by our genetic complementation approaches, may result in tracing of dorsal and medial interneurons and thus affect analysis of premotor circuits (Zampieri et al., 2014). To test this idea, we restricted expression of G to motor neurons by injecting AAV-flex-optimizedG (oG, Kim et al., 2016) in GS and TA muscles of *ChAT::Cre +/-* mice (1 μl of a 3:1 AAV-flex-oG: ΔG-RabV mix), a strategy that was previously used to show segregation of flexor and extensor premotor circuits (Figure 1B; Wang et al., 2017). In these experiments (4 GS and 3 TA injections), the distributions of GS and TA premotor interneurons were similar (Figure 9A-C, individual experiments are shown in Figure 12- figure supplements 14-15), with high correlation values across experiments (Figure 9D). Analysis of the medio-lateral positioning in the ipsilateral dorsal quadrant showed that the median position of TA related interneurons was shifted by only 33 μm with respect to that of the GS related interneurons, and that this was in a medial direction (383 μm for GS and 350 μm for TA; Hedges' G from hierarchical bootstrap = -0.15, IQR -0.24, -0.05, Figure 9E,F).

### Figure 9
```{r, fig.align='left', fig.width= 12 , fig.height= 6}
## -- 9AB -- ##
fig9ab <- readPNG("~/figures/Figure_9AB.png")
Fig9AB <- ggplot() + 
  background_image(fig9ab) +
  coord_equal(ratio = 0.434) +
  annotation_custom(rasterGrob(fig9ab, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig9AB
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', , fig.width= 12 , fig.height= 4}

## -- Figure 9 -- ##
method_used <- "AAV-G-FLEX_ChAT-cre"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)

## -- 9C coronal -- ##
panel <- "C"
density <- 1
concatenate <- 1
Figure_9Ci <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                        2, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)

## -- 9C rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_9Cii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                         2, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- 9C rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_9Ciii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                          2, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_9Ci, NULL, Figure_9Cii, NULL, Figure_9Ciii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}

## -- 9D -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 1
panel <- "D"
Figure_9D <- Corr_HedgesG(data_all, gs, ta, method_used, titer_used, N, lim, panel, color_gs, color_ta, 4, 3, numbers)
Figure_9D

```

**Figure 9**: Injection of AAV-FLEX-G in *ChAT::Cre* mice gives rise to overlapping distributions of flexor and extensor related premotor interneurons. Extensive labelling is observed in two lower lumbar sections of two GS (A) and TA (B) injected mice. The pooled distributions of flexor and extensor related infected neurons are similar on the transverse and longitudinal planes (C). For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. There is strong correlation and low effect sizes across individual experiments (D). Scale bars 400 μm.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}
## -- 9E -- ##
Figure_9E_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, gs, ta, color_gs, color_ta, ymin_d, ymax_950)
## -- 9F -- ##
dat9F <- df_individual(data_all, method_used, titer_used, gs, ta, GS, TA) 
Figure_9F_interactive <- Boxplot_individual_interactive(dat9F, color_gs, color_ta, ymin_d, ymax_950)

Fig_9EF_interactive <- subplot(Figure_9E_interactive, Figure_9F_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_9EF_interactive <- Fig_9EF_interactive %>% layout(width = 900, height = 450)
Fig_9EF_interactive
```
Box and whisker plots of median values of all the medio-lateral distributions with identical values for flexor and extensor injections.

### Anterograde transsynaptic jumps alter the distribution of premotor interneurons

Regardless of whether the expression of G was restricted to motor neurons (Figure 9) or not (Figure 8), we observed no differences in the mediolateral position of dorsal ipsilateral premotor interneurons related to flexor or extensor muscles. However, the use of AAV-oG-Flex that, contrarily to AAV-G, prevents anterograde jumps by restricting G expression to motor neurons (Figure 1A-B), led to important differences. Indeed, despite being performed and analysed by the same lab at similar timepoints for injection (P1-P2) and harvesting (7-8 days post-injection), experiments using AAV-G result in an extensive labelling of superficial dorsal horn neurons for both tested muscles (Figure 10A).

The median values along the dorsoventral axis were 249 and 375 μm for GS and 295 and 344 μm for TA following AAV-G or AAV-Flex-oG injections respectively. The Hedges' G bootstrapped coefficients were -0.55 (IQR -0.64, -0.46) for the GS pairs and -0.14 (IQR -0.23, -0.08) for the TA pairs. In addition, there was an excess of medially located infected neurons, in both GS and TA experiments, when the expression of G was not restricted to motor neurons. The dorsoventral and mediolateral differences in the distributions observed in AAV-G and AAV-oG-Flex experiments were substantial for both GS and TA, thus indicating that when G could be expressed by sensory neurons (AAV-G muscle injection), the average position of labelled cells extended more dorsally and medially compared to experiments in which G could only be expressed in motor neurons (AAV-oG-Flex). The observed medio dorsal shift is compatible with a contribution of anterograde tracing from sensory neurons (Figure 10A-C), which is absent when G is restricted to motor neurons, either by using AAV-oG-Flex, or by genetic restrictions. Nonetheless, regardless of the possible effects derived from anterograde tracing, the distributions of labelled cells observed in flexor and extensor experiments did not differ.

### Figure 10
```{r, echo=FALSE, warning=FALSE, fig.align='left', , fig.width= 12 , fig.height= 6}

method_used <- c("AAV-G_WT", "AAV-G-FLEX_ChAT-cre")
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
Animal <- 0


## -- 10Ai -- ##
panel <- ""
density <- 1
concatenate <- 1
Figure_10Ai <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, color_range_ext2, color_gs, color_mg, 2, 
                                         0, IN, N_A, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)

## -- 10Aii -- ##
panel <- ""
density <- 1
concatenate <- 1
Figure_10Aii <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, color_range_flex2, color_ta, color_pl, 2, 
                                          0, IN, N_A, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)

im1 <- readPNG("~/figures/Legend_Fig10A.png")
Legend_fig <- ggplot() + 
  background_image(im1) +
  theme(plot.margin = margin(t = 1, l= 1, r=0.5, b = 0.5, unit = "cm"))

Figure_10A <- cowplot::plot_grid(Figure_10Ai,NULL, Figure_10Aii, ncol=3,nrow = 1, align = "hv", rel_widths = c(4,2,4), rel_heights = 3)

ggarrange(Legend_fig, Figure_10A, nrow = 2, ncol = 1, heights = c(1.5, 4), labels = "A", font.label = list(size = 28, color = "black"))

```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 4.8}
## -- 10B -- ##
data <- data_all %>% filter(method == "AAV-G_WT" | method == "AAV-G-FLEX_ChAT-cre")
data <- data %>% filter(muscle == gs | muscle == ta)
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
titer_used <- 0

df_Hedges <- get_Hedges_boot(titer_used, data, R)
df_corr <- get_Corr_concatenate(data, titer_used, N, lim)
if (identical(df_corr$method_1, df_Hedges$method_1) & identical(df_corr$method_1,df_Hedges$method_1)) {
  df_corr_Hedges <- cbind(df_corr, df_Hedges$HedgesGcoef, stringsAsFactors = FALSE)
} else { print("error in correlation or Hedges G oefficient order")}
colnames(df_corr_Hedges) <- c("sample_1", "sample_2", "CorrCoef", "HedgesG") # Function "organise_Corr_hedges" is built to use dataframe with these exact colones' name. 
remove <- c("-cre")
df_corr_Hedges$sample_1 <- str_remove_all(df_corr_Hedges$sample_1, paste(remove, collapse = "|"))
df_corr_Hedges$sample_2 <- str_remove_all(df_corr_Hedges$sample_2, paste(remove, collapse = "|"))
df_corr_Hedges <- organise_Corr_hedges(df_corr_Hedges)
numbers <- 1
color1 <- c(color_mg, color_pl)
color2 <- c(color_gs, color_ta)
panel <- "B"
Figure_10B <- Corr_HedgesG_methods(df_corr_Hedges, panel, color1, color2, 2, 2, numbers)
Figure_10B
```
**Figure 10**: Injection of AAV-FLEX-G prevents anterograde spreading of RabV from afferent terminals. A: side by side comparison of pooled transverse distribution of infected interneurons from GS (left) and TA (right) with AAV-G or AAV-FLEX-G and RabV. The distributions obtained with AAV-G show a marked mediolateral shift and extensive labelling of superficial dorsal horn neurons, a feature that is absent when anterograde transfer is prevented by restricting the expression of the G protein to motor neurons only. The distributions obtained are very similar across the different muscles, but the mediolateral shift is reflected in the lower correlation values and high hierarchical bootstrapped effect size (B) between AAV-G and AAV-FLEX-G injections.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=5 ,fig.height=12}

## -- 10C -- ##
colors_vector <- c("#006eb8", "#003e68", "#eda911", "#be6100")
titer_used <- 0
method_used <- "AAV-G_WT"
dat10C <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA) 
dat10C <- dat10C %>% mutate(group = ifelse(group == "GS", "gs_AAV-G_WT", "ta_AAV-G_WT"))
method_used <- "AAV-G-FLEX_ChAT-cre"
df <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA)
df <- df %>% mutate(group = ifelse(group == "GS", "gs_AAV-G-FLEX_ChAT", "ta_AAV-G-FLEX_ChAT"))
dat10C <- rbind(dat10C, df)
dat10C$group <- factor(dat10C$group, levels = c("gs_AAV-G_WT","gs_AAV-G-FLEX_ChAT", "ta_AAV-G_WT", "ta_AAV-G-FLEX_ChAT"))
Figure_10C_interactive <- Boxplot_methods_interactive(dat10C, colors_vector, ymin_d, ymax_950)



Fig_10C_interactive <- subplot(Figure_10C_interactive, widths = c(1), heights = c(1)) 
Fig_10C_interactive <- Fig_10C_interactive %>% layout(width = 500, height = 500)
Fig_10C_interactive
```

<br>

Medians evidence the mediolateral shift obtained the different methods, AAV-G and AAV-FLEX-G injections.

### Flexor and extensor premotor interneurons organization using pseudorabies virus timed tracing experiments

Finally, we studied organization of premotor circuits using a different viral tracing method. Pseudorabies virus (PRV) is a neurotropic virus that travels transsynaptically in the retrograde direction. It has previously been used to resolve the connectivity order in polysynaptic circuits (Gu et al., 2017; Ugolini, 2020) using timed infections protocols. In particular, the Bartha strain has been shown to trace spinal premotor circuits around 40 hours (h) after injection (Jovanovic et al., 2010). We simultaneously injected P11 GS and TA muscles with 0.5 µl of PRV-Bartha (PRV-152 and PRV-614). Analysis of the spinal cords two days after injection showed extensive labelling of interneurons throughout the lumbar segments (Figures 11A-B). There were no differences in the distributions or median mediolateral positioning of flexor and extensor premotor interneurons (Figure 11C, individual experiments are shown in Figure 12-figure supplements 16), and correlation analysis showed high correlation coefficients between experiments and across muscles (Figure 11D) and low values of the Hedges' G for pairs of experiments (median 0.09, IQR 0.04, 0.22). Accordingly, the medians of mediolateral positions of flexor and extensor premotor interneurons were similar (353 μm for GS and 366 μm for TA; Figure 11E,F).

### Figure 11
```{r, fig.align='left', fig.width= 12 , fig.height= 5}
## -- 11AB -- ##
fig11ab <- readPNG("~/figures/Figure_11AB.png")
Fig11AB <- ggplot() + 
  background_image(fig11ab) +
  coord_equal(ratio = 0.426) +
  annotation_custom(rasterGrob(fig11ab, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig11AB
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', , fig.width= 12 , fig.height= 4}

## -- Figure 11 -- ##
method_used <- "PRV_WT"
Size <- c(0.1, 0)
kernel <- 6
bin <- 5
titer_used <- "high"
df <- data_all %>% filter(method == as.character(method_used), titer == as.character(titer_used))
animals <- distinct(data.frame(df$animal))
Animal <- as.character(animals$df.animal)

## -- 11C coronal -- ##
panel <- "C"
density <- 1
concatenate <- 1
Figure_11Ci <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                         2, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)

## -- 11C rostro-caudal ipsi-contra-- ##
kernel <- 1.5
Figure_11Cii <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                          2, IN, N_A, Size, kernel, bin, method_used, titer_used)

## -- 11C rostro-caudal ventro-dorsal-- ##
kernel <- 1.5
Figure_11Ciii <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 4, 
                                           2, IN, N_A, Size, kernel, bin, titer_used, method_used)

cowplot::plot_grid(Figure_11Ci, NULL, Figure_11Cii,NULL, Figure_11Ciii,
                   ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 6 ,fig.height= 5}
## -- 11D -- ##
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
numbers <- 1
panel <- "D"
Figure_11D <- Corr_HedgesG(data_all, gs, ta, method_used, titer_used, N, lim, panel, color_gs, color_ta, 4, 4, numbers)
Figure_11D
```
**Figure 11**: Injections of PRV-Bartha in GS and TA muscles give rise to similar distribution of premotor interneurons. Example of a lower lumbar section of an animal injected in GS(A) and TA (B). The distribution of premotor interneurons are similar along all axis (C). For each section the data are scaled to the reference points indicated in the methods in order to account for size differences along the segments. Individual experiments are highly correlated, with small effect size (D). Scale bars 300 μm.

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=2}
## -- 11E -- ##
Figure_11E_interactive <- boxplot_per_sample_interactive(data_all, method_used, titer_used, gs, ta, color_gs, color_ta, ymin_d, ymax_950)
## -- 11F -- ##
dat11F <- df_individual(data_all, method_used, titer_used, gs, ta, GS, TA) 
Figure_11F_interactive <- Boxplot_individual_interactive(dat11F, color_gs, color_ta, ymin_d, ymax_950)


Fig_11EF_interactive <- subplot(Figure_11E_interactive, Figure_11F_interactive, widths = c(0.8,0.2), heights = c(1)) 
Fig_11EF_interactive <- Fig_11EF_interactive %>% layout(width = 900, height = 450)
Fig_11EF_interactive
```

Box and whisker plots of median values of all the medio-lateral distributions with identical values for flexor and extensor injections. <br><br><br>
We finally pooled together all the results obtained with different methods. In all cases, the correlations between flexor and extensor muscles labelled with the same methods were very high (Figure 12A). The lowest correlation coefficients (which were still high) were typically observed between experiments with AAV-G injection in wild type mice and all the other methods, regardless of the muscle injected, consistent with a substantial contamination from interneurons anterogradely labelled through sensory afferents. Overall, the medians of the mediolateral positions (Figure 12B) did not differ between muscles for any of the methods used and the range of bootstrapped Hedges' G coefficient was between -0.23 and 0.17 across pairs of antagonist muscles throughout five different experimental paradigms tested. The whole set of experiments is shown in Figure 12- figure supplements 1-16. The labelling of experiments keeps the original lab conventions and is also reported in Table 1. Taken together, our findings indicate that premotor interneurons innervating flexor and extensor motor neurons are not spatially segregated.

### Figure 12
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 ,fig.height= 10.4}
data <- data_all %>% filter(muscle == lg |muscle == gs | muscle == ta)
N <- c(21, 21)
lim <- c(-850, 850, -450, 450)
titer_used <- 1

df_Hedges <- get_Hedges_boot(titer_used, data, R)
df_corr <- get_Corr_concatenate(data, titer_used, N, lim)
if (identical(df_corr$method_1, df_Hedges$method_1) & identical(df_corr$method_1,df_Hedges$method_1)) {
  df_corr_Hedges <- cbind(df_corr, df_Hedges$HedgesGcoef, stringsAsFactors = FALSE)
} else { print("error in correlation or Hedges G oefficient order")}
colnames(df_corr_Hedges) <- c("sample_1", "sample_2", "CorrCoef", "HedgesG") # Function "organise_Corr_hedges" is built to use dataframe with these exact colonnes' name. 
remove <- c("-cre", "_RGT", "G_Rab_")
df_corr_Hedges$sample_1 <- str_remove_all(df_corr_Hedges$sample_1, paste(remove, collapse = "|"))
df_corr_Hedges$sample_2 <- str_remove_all(df_corr_Hedges$sample_2, paste(remove, collapse = "|"))
df_corr_Hedges <- organise_Corr_hedges(df_corr_Hedges)
numbers <- 1
color1 <- c(color_gs, color_ta,color_gs, color_ta,color_gs, color_ta)
color2 <- c(color_gs, color_ta,color_gs, color_ta,color_gs, color_ta)
panel <- "A"
Figure_12A <- Corr_HedgesG_methods(df_corr_Hedges, panel, color1, color2, 6, 6, numbers)
Figure_12A
```
```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width=8 ,fig.height=9}

## -- 12B -- ##

colors_vector <- rep(c("#006eb8", "#eda911"), times= 6)
titer_used <- "high"
method_used <- "G_Rab_ChAT_RGT"
dat12B <- df_individual(data_all, method_used, titer_used, lg, ta , LG, TA) 
dat12B <- dat12B %>% mutate(group = ifelse(group == "LG", "LG_ChAT_high", "TA_ChAT_high"))
titer_used <- "low"
method_used <- "G_Rab_ChAT_RGT"
df <- df_individual(data_all, method_used, titer_used, lg, ta , LG, TA) 
df <- df %>% mutate(group = ifelse(group == "LG", "LG_ChAT_low", "TA_ChAT_low"))
dat12B <- rbind(dat12B, df)
titer_used <- 0
method_used <- "G_Rab_Olig2_RGT"
df <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA) 
df <- df %>% mutate(group = ifelse(group == "GS", "GS_Olig2", "TA_Olig2"))
dat12B<- rbind(dat12B, df)
method_used <- "AAV-G_WT"
df <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA)
df <- df %>% mutate(group = ifelse(group == "GS", "GS_AAV-G_wt", "TA_AAV-G_wt"))
dat12B<- rbind(dat12B, df)
method_used <- "AAV-G-FLEX_ChAT-cre"
df <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA)
df <- df %>% mutate(group = ifelse(group == "GS", "GS_AAV-Flex-G_ChAT", "TA_AAV-Flex-G_ChAT"))
dat12B<- rbind(dat12B, df)
method_used <- "PRV_WT"
df <- df_individual(data_all, method_used, titer_used, gs, ta , GS, TA)
df <- df %>% mutate(group = ifelse(group == "GS", "GS_PRV", "TA_PRV"))
dat12B<- rbind(dat12B, df)
dat12B$group <- factor(dat12B$group, levels = c("LG_ChAT_high", "TA_ChAT_high","LG_ChAT_low", "TA_ChAT_low","GS_Olig2", "TA_Olig2", "GS_AAV-G_wt", "TA_AAV-G_wt", "GS_AAV-Flex-G_ChAT", "TA_AAV-Flex-G_ChAT", "GS_PRV", "TA_PRV"))

Figure_12B_interactive <- Boxplot_methods_interactive(dat12B, colors_vector, ymin_d, ymax_950)
Figure_12B_interactive <- Figure_12B_interactive %>% layout(width = 900, height = 500)
Figure_12B_interactive
```

<br> <br> **Figure 12**: Regardless of the tracing methods, flexor and extensor related premotor interneurons distributions are always overlapping. The correlation and absolute value of hierarchical bootstrapped Hedges' G coefficient matrix across pooled experiments obtained with different injection strategies exhibits high values of correlation and low effect sizes across different techniques and across the two different muscles (A). The median values of the mediolateral position of premotor interneurons in the dorsal ipsilateral quadrant are similar for all conditions (B).

# Discussion

Spinal circuits are responsible for integrating descending commands and sensory information to ensure precise control and coordination of movement. In order to understand how these circuits organise movement, it is necessary to first identify and then study the roles and contributions of spinal interneurons that control the activity of different muscles. Previous work (Tripodi et al., 2011) exploited rabies monosynaptic tracing to examine the organization of premotor circuits controlling the activity of selected muscles. These studies, using intramuscular injection of an AAV expressing the rabies glycoprotein G, identified clear segregation in the spatial organization of premotor interneurons directing the activity of flexor and extensor muscles (Tripodi et al., 2011; Wang et al., 2017; Takeoka and Arber, 2019). In contrast, our study using either genetic and AAV complementation of G expression, as well as PRV timed infections, demonstrates complete spatial overlap amongst flexor and extensor premotor interneurons.

### Mouse genetic-based strategies for rabies monosynaptic tracing of premotor circuits

We opted for a mouse genetic strategy that was previously used to trace premotor circuits of vibrissal, orofacial, and forelimb muscles (Takatoh et al., 2013; Stanek et al., 2014; Skarlatou et al., 2020). Combining a conditional allele expressing G from the rosa locus (*RΦGT* mice; Takatoh et al., 2013) with either the *ChAT::Cre* or *Olig2::Cre* lines (Figure 1D-E) is predicted to result in high levels of G expression in all motor neurons at the time of rabies muscle injection and therefore in robust transsynaptic transfer. Indeed, under these conditions, several hundred premotor neurons can be reproducibly traced in each experiment (Table 1; Skarlatou et al., 2020; Ronzano et al., 2021). On the other hand, this strategy suffers from the undesirable consequences of lineage tracing, namely G complementation in all Cre expressing cells in the spinal cord, including those that transiently activate the targeted promoter during development. This problem is in part shared with the AAV-based experiments using intraspinal injection of AAV-flex-G in *ChAT::Cre* mice (Figure 1C; Takeoka and Arber, 2019). Thus, it is unlikely that the differences in the results obtained using these two strategies were caused by disynaptic transfer through cholinergic interneurons. Indeed, work using rabies monosynaptic tracing to identify spinal neurons presynaptic to the most prominent population of premotor cholinergic interneurons, V0c neurons, found a high density of pre-V0c neurons located in superficial laminae of the dorsal horn (Zampieri et al., 2014), an area where no labelling was observed in our mouse genetic-based experiments.

Nonetheless, to further explore the possibility of disynaptic pathways via cholinergic spinal interneurons, we examined the organization of flexor and extensor premotor circuits in experiments conducted with the *Olig2::Cre +/-; RΦGT+/-* line. Olig2 is expressed in motor neurons and in subsets of p2 and p3 progenitors, thus potentially generating opportunities for disynaptic transfer through V2 and V3 premotor interneurons (Chen et al., 2011). We did not find any difference in the distribution of premotor interneurons obtained in these mice compared to those in *ChAT::Cre+/-; RΦGT+/-* mice. Therefore, it appears that disynaptic transfer is not a frequent event in our experimental conditions; otherwise, neuronal labelling in *ChAT::Cre* experiments would reflect the contribution of cholinergic interneurons, and *Olig2::Cre* experiments would reflect jumps through V2 and V3 interneurons. It is also important to consider the timing of rabies transsynaptic transfer (Ugolini, 2011). The earliest expression of rabies in primary infected motor neurons is first observed 3-4 days after injection and monosynaptic transfer not earlier than 5 days after injection, with strong labelling observed around 7-8 days. Since in our experiments, mice were sacrificed between 6 to 9 days following RabV injections (8-9 days in *ChAT::Cre* and 6 days in *Olig2::Cre* mice), it is unlikely that many, if any, double jumps would have occurred in this time window. We cannot exclude that at least some of the labelled interneurons were generated by second order transfers, but arguably these are rare events and unlikely to be the source of the different results obtained in AAV vs mouse genetic experiments.

### AAV- based strategies for rabies monosynaptic tracing of premotor circuits

In previous studies in which segregation of flexor and extensor premotor interneurons have been observed, AAV was used to express G in motor neurons. In the first report, AAV-G and ΔG-RabV were co-injected intramuscularly in wild type mice (Tripodi et al., 2011). This approach has the advantage of complementing G only in motor neurons projecting to the targeted muscle, thus avoiding the problem of G expression in spinal interneurons that could lead to loss of monosynaptic restriction. However, since sensory neurons in the dorsal root ganglia also innervate muscles, such strategy could lead to anterograde transsynaptic spread to the spinal cord through the sensory route (Figure 1A, Zampieri et al., 2014). In order to avoid this problem, intramuscular co-injection of a conditional AAV vector (AAV-flex-G) with ΔG-RabV in *ChAT::Cre* mice was used (Figure 1B, Wang et al., 2017). In this more stringent condition, G would only be expressed in motor neurons. A more recent study used intraspinal injection of AAV-flex-G in *ChAT::Cre* mice (Figure 1C, Takeoka and Arber, 2019), which would also avoid transfer from sensory neurons. However, despite the fact that these AAV-based strategies have distinct advantages and disadvantages, they all resulted in labelling of flexor and extensor premotor interneurons with distributions that were medio-laterally segregated in the dorsal ipsilateral quadrant of the spinal cord in experiments performed on neonatal (Tripodi et al., 2011) and adult (Takeoka and Arber, 2019) mouse hindlimbs, as well as neonatal forelimbs (Wang et al., 2017).

In order to resolve the discrepancy with the results obtained with genetic complementation, we tried to replicate the previous findings by directly testing two of the AAV complementation strategies, namely AAV-G and ΔG-RabV co-injection in wild type mice (Figure 1A;Tripodi et al., 2011), and AAV-flex-G and ΔG-RabV co-injection in *ChAT::Cre* mice (Figure 1B; Wang et al., 2017). Given the small size of the injected muscles, we limited our injection volumes to 1 µl, as opposed to the 5 µl used in the original study (Tripodi et al., 2011) routinely checking for injection specificity by careful examination of the muscles. Surprisingly, despite our attempts at replication, we did not observe segregation of flexor and extensor premotor circuits. However, the contribution of anterograde transsynaptic spread to the spinal cord through the sensory route was clearly detected in AAV-G experiments (but, as expected, not in AAV-flex-oG experiments), reflecting the contribution of muscle-innervating sensory afferents that have post-synaptic targets predominantly located in the dorsal and medial aspects of the spinal cord (Zampieri et al., 2014; Pimpinella et al, 2022). Notably, this observation does not explain why we failed to replicate segregation of flexor and extensor premotor circuits, as we detected overlapping distributions both in the presence (AAV-G) or absence (AAV-flex-oG) of sensory contributions. It is nonetheless interesting to notice that in the original report using AAV-G and ΔG-RabV co-injection in wild type mice, the only condition where flexor-extensor segregation was not reported by the authors is upon elimination of the sensory route by ablation of proprioceptors with diphtheria toxin (Tripodi et al., 2011).

It is worth noting that while the original paper used a cytomegalovirus (CMV) promoter to drive G expression (Tripodi et al., 2011), we used either a Human elongation factor-1 alpha promoter (EF-1a) or a CMV early enhancer/chicken β actin promoter (CAG). In adddition, we used an AAV to RabV ratio of 3:1 as opposed to 1:1. Although these differences may account for higher expression of G and increased efficacy in jumps, they cannot explain the observed lack of segregation. Finally, the major difference in the experiments is the total volume of viral suspension injected intramuscularly. We limited our injections to 1 µl as we measured the volumes of the GS and TA muscles to be around 2 μl in the early post-natal period. Thus, it is possible that the larger volume (5 μl) injected in the other studies might have affected the results, for example by differential infection of proprioceptive afferents innervating the flexor vs extensor muscles, or by infection of cutaneous afferents supplying the overlying skin (Li et al., 2011).

### The issue of starter motor neurons

The identity and number of starter cells are the main determinants of reproducibility in rabies tracing experiments and thus represent key parameters for comparing different approaches. For experiments using ΔG-RabV, starter cells are those that are both primarily infected with RabV and express G. In general, for both the AAV and mouse genetics methods discussed here, it is difficult to precisely determine these factors, as neither approach employs expression of a reporter gene to mark G-expressing cells. Moreover, RabV is known to be toxic to neurons and some primary infected motor neurons are likely to die before analysis (Reardon et al., 2016). Because of the well-known topographic organization of neuromuscular maps, muscle identity of infected motor neurons can be inferred by their stereotyped position in the spinal cord (Romanes, 1964; McHanwell and Biscoe, 1981). Thus, for all the methodologies discussed here, it is only possible to approximate the identity and number of starter motor neurons by surveying the position of RabV-infected motor neurons present at the end of the experiment.

Restriction of starter "status" to motor neurons connected to a single muscle is determined by two aspects: the specificity of rabies virus injection and the availability of sufficient levels of G protein in the same cells (Callaway and Luo, 2015). All the approaches discussed here used intramuscular injections of G-deleted rabies virus (SAD-B19) to selectively infect a motor pool. In this step, sources of variability are represented by 1) specificity of muscle injection and 2) the titre of the rabies virus injected. Muscle injection specificity was routinely checked following injections of adjacent synergist muscles and for all the co-injections of AAV-G and RabV. Rabies leak from antagonist muscles (LG and TA) located in different anatomical compartments on opposite sides of the tibia and fibula would be very unlikely. The titre of the injected rabies virus can affect the efficiency of primary infection: the data presented here show that the RabV titre, while affecting the number of motor neurons and secondary neurons labelled, does not affect the overall distribution of premotor interneurons. The same data indicate that the presence of a small number of "ectopic" motor neurons (seen only following high titre injections) does not significantly contribute to the tracing results, as the premotor distributions in high and low titre experiments are not different. Furthermore, these "ectopic" motor neurons likely represent recurrently connected presynaptic motor neurons (Bhumbra and Beato, 2018); therefore any labelling originating from them would represent a much less frequent disynaptic transfer event.

In the short term, the introduction of a reporter system to label G-expressing neurons, as routinely done in many rabies experiments, combined with the use of non-toxic rabies variants that would prevent motor neuron death (Reardon et al., 2016; Ciabatti et al., 2017; Chatterjee et al., 2018) will help resolve potential confusion about the identity and number of starter cells. Such tools could be used in both the AAV and the mouse genetic approaches. In addition, the ability to precisely restrict the selection of starter motor neurons either by the introduction of more specific Cre lines (e.g. Koronfel et al., 2021) or through the use of novel intersectional strategies could improve premotor tracing experiments. Finally, tracing from single motor neurons using delivery of DNA for G and TVA expression via patch clamp is a precise way to generate specific starter cells (Marshel et al., 2010; Rancz et al., 2011). This approach, followed by intraspinal injection of EnvA-pseudotyped ΔG-RabV, would ensure infection and pre-synaptic tracing from only selected neurons. This method would have the added value of directly showing whether functionally distinct motor neurons within a pool receive differentially distributed presynaptic input, but it would rely on being able to perform patch clamp recordings from motor neurons in vivo, followed by a sufficient survival time to allow for sufficient viral expression, a feat that to our knowledge has not been attempted and might not even be possible.

### Functional implications

While it has been proposed that spatial segregation of premotor interneurons provides an anatomical substrate for labelled line inputs from proprioceptive afferents to motor neurons (Tripodi et al., 2011), our data do not support such a model. Proprioceptive afferents projecting to the intermediate and dorsal spinal cord relay many types of information (e.g., changes in muscle length, muscle length itself, force, joint position), and form synapses with both excitatory and inhibitory interneurons that process and convey these data to flexor and extensor motor neurons to precisely regulate patterns of contraction. It is clear that, at least in the case of motor neurons, position plays an important role in the specificity of afferent inputs received (Sürmeli et al., 2011), and it is reasonable to think that it might also be the case for interneurons. Indeed, it has been shown for V1 inhibitory interneurons that cell body positioning constrains wiring from proprioceptive afferents (Bikoff et al., 2016). However, the location of V1 subtypes does not seem to influence their output connectivity to motor neurons, as indicated by the case of Sp8+ V1 interneurons that are located in a medial position in the intermediate spinal cord, in about the same location described for premotor interneuron serving extensor motor pools (Tripodi et al 2011) but send uniform synaptic output to both flexor and extensor motor pools (Bikoff et al., 2016). Our data are consistent with the idea that interneuron position plays an important role in controlling sensory input connectivity, but not their output connectivity to motor neurons. That is, positional organization has been shown to represent a developmental strategy that facilitates wiring onto spatially segregated populations of neurons, rather than determining output connectivity, where neuronal position does not seem to have a major role. It is important to stress that both the original study showing segregation of premotor interneurons (Tripodi et al., 2011) and ours, showing lack of it, are performed on neonatal animals, because the variant of rabies used in both studies cannot cross the mature neuromuscular junction and infect adult motor neurons (Stepien et al., 2010, but see Takeoka and Arber, 2019). We have no evidence that the output connectivity of premotor interneurons that we describe here is conserved throughout development, because the initial wiring of circuits is later refined as the mice develop, adapt to their environment and learn new motor tasks. Nonetheless, we cannot attribute the observed lack of segregation to developmental factors, since we have used animals in the same age range as in the original study.

# Conclusion

In conclusion, despite using five different methods, we have not been able to find evidence that spinal premotor interneurons innervating flexor vs extensor motor neurons are segregated. It is important to stress that none of the methods discussed here is completely exempt from potential problems (Table 2). However, full appreciation of the strengths and weaknesses of each approach can guide both the choice of method for mapping premotor circuits and the interpretation of the results obtained.

### Figure 12 figure supplement 1

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}

panel <- ""
concatenate = 0
density <- 0
kernel <- 6
bin <- 5
Size <- c(0.2, 2)

method_used <- "G_Rab_ChAT_RGT"
titer_used <- "high"
Animal <- "170427n2_n_rmb"

plot_xy_170427n2_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_170427n2_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170427n2_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170427n2", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170427n2_n_rmb <- cowplot::plot_grid(plot_xy_170427n2_n_rmb, NULL, plot_xz_170427n2_n_rmb, NULL, plot_yz_170427n2_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid( title, Fig_170427n2_n_rmb,ncol = 1, rel_heights = c(0.1, 1))

```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "170427n3_n_rmb"
plot_xy_170427n3_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_170427n3_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170427n3_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170427n3", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170427n3_n_rmb <- cowplot::plot_grid(plot_xy_170427n3_n_rmb, NULL, plot_xz_170427n3_n_rmb, NULL, plot_yz_170427n3_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid( title, Fig_170427n3_n_rmb,ncol = 1, rel_heights = c(0.1, 1))

```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}

Animal <- "170503n6_n_rmb"
plot_xy_170503n6_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_170503n6_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170503n6_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 
                                                    1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170503n6", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170503n6_n_rmb <- cowplot::plot_grid(plot_xy_170503n6_n_rmb, NULL, plot_xz_170503n6_n_rmb, NULL, plot_yz_170503n6_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid( title, Fig_170503n6_n_rmb,ncol = 1, rel_heights = c(0.1, 1))
```

**Figure 12 figure supplement 1**: Distribution of motor neurons (triangles) and premotor interneurons (dots) of 3 LG (blue) and TA (yellow) double injections (UCL) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 2
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1577_n_djm"
plot_xy_1577_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1577_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)

plot_yz_1577_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1,1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1577", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1577_n_djm <- cowplot::plot_grid(plot_xy_1577_n_djm, NULL, plot_xz_1577_n_djm, NULL, plot_yz_1577_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid( title, Fig_1577_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1578_n_djm"
plot_xy_1578_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1578_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)

plot_yz_1578_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1,1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1578", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1578_n_djm <- cowplot::plot_grid(plot_xy_1578_n_djm, NULL, plot_xz_1578_n_djm, NULL, plot_yz_1578_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1578_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 2** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 2 LG (blue) and TA (yellow) double injections (UoG) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 3
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "170125n3_n_rmb"
plot_xy_170125n3_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
plot_xz_170125n3_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170125n3_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170125n3", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170125n3_n_rmb <- cowplot::plot_grid(plot_xy_170125n3_n_rmb, NULL, plot_xz_170125n3_n_rmb, NULL, plot_yz_170125n3_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_170125n3_n_rmb,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}

Animal <- "170508n7_n_rmb"
plot_xy_170508n7_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
plot_xz_170508n7_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170508n7_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170508n7", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170508n7_n_rmb <- cowplot::plot_grid(plot_xy_170508n7_n_rmb, NULL, plot_xz_170508n7_n_rmb, NULL, plot_yz_170508n7_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_170508n7_n_rmb,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 3** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 2 LG (blue) and MG (dark blue) double injections (UCL) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 4
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1579_n_djm"
plot_xy_1579_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1579_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1579_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1579", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1579_n_djm <- cowplot::plot_grid(plot_xy_1579_n_djm, NULL, plot_xz_1579_n_djm, NULL, plot_yz_1579_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1579_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1580_n_djm"
plot_xy_1580_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1580_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1580_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1580", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1580_n_djm <- cowplot::plot_grid(plot_xy_1580_n_djm, NULL, plot_xz_1580_n_djm, NULL, plot_yz_1580_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1580_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1701_n_djm"
plot_xy_1701_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1701_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1701_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)
title <- ggdraw() + draw_label("1701", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1701_n_djm <- cowplot::plot_grid(plot_xy_1701_n_djm, NULL, plot_xz_1701_n_djm, NULL, plot_yz_1701_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1701_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1702_n_djm"
plot_xy_1702_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, LG, MG, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1702_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1702_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, mg, color_range_ext, color_range_ext2, color_lg, color_mg, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)
title <- ggdraw() + draw_label("1702", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1702_n_djm <- cowplot::plot_grid(plot_xy_1702_n_djm, NULL, plot_xz_1702_n_djm, NULL, plot_yz_1702_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1702_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 4** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 4 LG (blue) and MG (yellow) double injections (UoG) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 5
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "170125n7_n_rmb"
plot_xy_170125n7_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1,1, IN, MN, Size, TA, PL, kernel, bin, method_used, titer_used, panel)
plot_xz_170125n7_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170125n7_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1,1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170125n7", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170125n7_n_rmb <- cowplot::plot_grid(plot_xy_170125n7_n_rmb, NULL, plot_xz_170125n7_n_rmb, NULL, plot_yz_170125n7_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_170125n7_n_rmb,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "170125n8_n_rmb"
plot_xy_170125n8_n_rmb <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1,1, IN, MN, Size, TA, PL, kernel, bin, method_used, titer_used, panel)
plot_xz_170125n8_n_rmb <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_170125n8_n_rmb <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, pl, color_range_flex, color_range_flex2, color_ta, color_pl, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("170125n8", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_170125n8_n_rmb <- cowplot::plot_grid(plot_xy_170125n8_n_rmb, NULL, plot_xz_170125n8_n_rmb, NULL, plot_yz_170125n8_n_rmb, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_170125n8_n_rmb,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 5** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 2 TA (yellow) and PL (dark orange) double injections (UCL) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 6
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1605_n_djm"
plot_xy_1605_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1, 0, IN, MN, Size, MG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1605_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1605_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1605", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1605_n_djm <- cowplot::plot_grid(plot_xy_1605_n_djm, NULL, plot_xz_1605_n_djm, NULL, plot_yz_1605_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1605_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1660_n_djm"
plot_xy_1660_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, MG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1660_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1660_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1660", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1660_n_djm <- cowplot::plot_grid(plot_xy_1660_n_djm, NULL, plot_xz_1660_n_djm, NULL, plot_yz_1660_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1660_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1661_n_djm"
plot_xy_1661_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, MG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1661_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1661_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1661", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1661_n_djm <- cowplot::plot_grid(plot_xy_1661_n_djm, NULL, plot_xz_1661_n_djm, NULL, plot_yz_1661_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1661_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1662_n_djm"
plot_xy_1662_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1, 0, IN, MN, Size, MG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1662_n_djm <-complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1662_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, mg, N_A, color_range_ext2, transpa, color_mg, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1662", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1662_n_djm <- cowplot::plot_grid(plot_xy_1662_n_djm, NULL, plot_xz_1662_n_djm, NULL, plot_yz_1662_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1662_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 6** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 4 MG (dark blue) single injections (UoG) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 7
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1611_n_djm"
plot_xy_1611_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,0, IN, MN, Size, PL, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1611_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,  0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1611_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,  0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1611", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1611_n_djm <- cowplot::plot_grid(plot_xy_1611_n_djm, NULL, plot_xz_1611_n_djm, NULL, plot_yz_1611_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1611_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1613_n_djm"
plot_xy_1613_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1, 0, IN, MN, Size, PL, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1613_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1613_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1613", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1613_n_djm <- cowplot::plot_grid(plot_xy_1613_n_djm, NULL, plot_xz_1613_n_djm, NULL, plot_yz_1613_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1613_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1640_n_djm"
plot_xy_1640_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,0, IN, MN, Size, PL, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1640_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1640_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, pl, N_A, color_range_flex2, transpa, color_pl, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1640", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1640_n_djm <- cowplot::plot_grid(plot_xy_1640_n_djm, NULL, plot_xz_1640_n_djm, NULL, plot_yz_1640_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1640_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 7** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 3 PL (dark orange) single injections (UoG) of rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice.

### Figure 12 figure supplement 8
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
titer_used <- "low"
Animal <- "1644_n_djm"
plot_xy_1644_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1644_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1,  1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1644_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)


title <- ggdraw() + draw_label("1644", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1644_n_djm <- cowplot::plot_grid(plot_xy_1644_n_djm, NULL, plot_xz_1644_n_djm, NULL, plot_yz_1644_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1644_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1646_n_djm"
plot_xy_1646_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1646_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1646_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1646", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1646_n_djm <- cowplot::plot_grid(plot_xy_1646_n_djm, NULL, plot_xz_1646_n_djm, NULL, plot_yz_1646_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1646_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1653_n_djm"
plot_xy_1653_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1,1, IN, MN, Size, LG, TA, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1653_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1653_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, ta, color_range_ext, color_range_flex, color_lg, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1653", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1653_n_djm <- cowplot::plot_grid(plot_xy_1653_n_djm, NULL, plot_xz_1653_n_djm, NULL, plot_yz_1653_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1653_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 8** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 3 LG (blue) and TA (yellow) double injections with low titre rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice (UoG).

### Figure 12 figure supplement 9
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1656_n_djm"
plot_xy_1656_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1656_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1656_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1656", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1656_n_djm <- cowplot::plot_grid(plot_xy_1656_n_djm, NULL, plot_xz_1656_n_djm, NULL, plot_yz_1656_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1656_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1657_n_djm"
plot_xy_1657_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1657_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,  0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1657_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1657", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1657_n_djm <- cowplot::plot_grid(plot_xy_1657_n_djm, NULL, plot_xz_1657_n_djm, NULL, plot_yz_1657_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1657_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1570_n_djm"
plot_xy_1570_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1570_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1570_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1570", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1570_n_djm <- cowplot::plot_grid(plot_xy_1570_n_djm, NULL, plot_xz_1570_n_djm, NULL, plot_yz_1570_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1570_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1571_n_djm"
plot_xy_1571_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, LG, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1571_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1571_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, lg, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1571", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1571_n_djm <- cowplot::plot_grid(plot_xy_1571_n_djm, NULL, plot_xz_1571_n_djm, NULL, plot_yz_1571_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1571_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 9** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 4 LG (blue) single injections with low titre rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice (UoG).

### Figure 12 figure supplement 10
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1573_n_djm"
plot_xy_1573_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1573_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1573_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1573", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1573_n_djm <- cowplot::plot_grid(plot_xy_1573_n_djm, NULL, plot_xz_1573_n_djm, NULL, plot_yz_1573_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1573_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1574_n_djm"
plot_xy_1574_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1574_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1574_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1574", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1574_n_djm <- cowplot::plot_grid(plot_xy_1574_n_djm, NULL, plot_xz_1574_n_djm, NULL, plot_yz_1574_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1574_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "1639_n_djm"
plot_xy_1639_n_djm <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,  0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_1639_n_djm <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,  0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1639_n_djm <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1639", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1639_n_djm <- cowplot::plot_grid(plot_xy_1639_n_djm, NULL, plot_xz_1639_n_djm, NULL, plot_yz_1639_n_djm, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1639_n_djm,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 10** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 3 TA (yellow) single injections with low titre rabies virus in *ChAT::Cre+/-; RΦGT+/-* mice (UoG).

### Figure 12 figure supplement 11
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
titer_used <- "high"
method_used <- "G_Rab_Olig2_RGT"
Animal <- "1332_n_nz"
plot_xy_1332_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_1332_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1332_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1332", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1332_n_nz <- cowplot::plot_grid(plot_xy_1332_n_nz, NULL, plot_xz_1332_n_nz, NULL, plot_yz_1332_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1332_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "1349_n_nz"
plot_xy_1349_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_1349_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1349_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1349", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1349_n_nz <- cowplot::plot_grid(plot_xy_1349_n_nz, NULL, plot_xz_1349_n_nz, NULL, plot_yz_1349_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1349_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "353_n_nz"
plot_xy_353_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_353_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_353_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("353", x = 0.55,hjust = 0, fontface = 'bold',size = 28)
Fig_353_n_nz <- cowplot::plot_grid(plot_xy_353_n_nz, NULL, plot_xz_353_n_nz, NULL, plot_yz_353_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_353_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "399_n_nz"
plot_xy_399_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_lg, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_399_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,  0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_399_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("399", x = 0.55,hjust = 0, fontface = 'bold',size = 28)
Fig_399_n_nz <- cowplot::plot_grid(plot_xy_399_n_nz, NULL, plot_xz_399_n_nz, NULL, plot_yz_399_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_399_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 11** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 4 GS (blue) single injections with rabies virus in *Olig2::Cre +/-; RΦGT+/-* mice (MDC).

### Figure 12 figure supplement 12
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "700_n_nz"
plot_xy_700_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_700_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_700_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("700", x = 0.55,hjust = 0, fontface = 'bold',size = 28)
Fig_700_n_nz <- cowplot::plot_grid(plot_xy_700_n_nz, NULL, plot_xz_700_n_nz, NULL, plot_yz_700_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_700_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "721_n_nz"
plot_xy_721_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_721_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_721_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("721", x = 0.55,hjust = 0, fontface = 'bold',size = 28)
Fig_721_n_nz <- cowplot::plot_grid(plot_xy_721_n_nz, NULL, plot_xz_721_n_nz, NULL, plot_yz_721_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_721_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "1324_n_nz"
plot_xy_1324_n_nz <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_1324_n_nz <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1324_n_nz <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("1324", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_1324_n_nz <- cowplot::plot_grid(plot_xy_1324_n_nz, NULL, plot_xz_1324_n_nz, NULL, plot_yz_1324_n_nz, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1324_n_nz,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 12** Distribution of motor neurons (triangles) and premotor interneurons (dots) of 3 TA (yellow) single injections with rabies virus in *Olig2::Cre+/-; RΦGT+/-* mice (MDC).

### Figure 12 figure supplement 13
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
method_used <- "AAV-G_WT"
Animal <- "1_n_slp"
plot_xy_1_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1,1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_1_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1,1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)


title <- ggdraw() + draw_label("GS-TA_1", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_1_n_slp <- cowplot::plot_grid(plot_xy_1_n_slp, NULL, plot_xz_1_n_slp, NULL, plot_yz_1_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "2_n_slp"
plot_xy_2_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_2_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_2_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS-TA_2", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_2_n_slp <- cowplot::plot_grid(plot_xy_2_n_slp, NULL, plot_xz_2_n_slp, NULL, plot_yz_2_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_2_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "a_n_slp"
plot_xy_a_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_a_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_a_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_a", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_a_n_slp <- cowplot::plot_grid(plot_xy_a_n_slp, NULL, plot_xz_a_n_slp, NULL, plot_yz_a_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_a_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "b_n_slp"
plot_xy_b_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
kernel <- 4
plot_xz_b_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_b_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_b", x = 0.54,hjust = 0, fontface = 'bold',size = 28)
Fig_b_n_slp <- cowplot::plot_grid(plot_xy_b_n_slp, NULL, plot_xz_b_n_slp, NULL, plot_yz_b_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_b_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 13** Distribution of infected neurons (primary infected motor neurons or secondary infected interneurons are not distinguished) of 2 GS (blue) and TA (yellow) double injections and 2 GS single injections of rabies and AAV-Ef1a-B19G in wild type mice (Salk).

### Figure 12 figure supplement 14
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
method_used <- "AAV-G-FLEX_ChAT-cre"
Animal <- "22a_4_n_slp"
plot_xy_22a_4_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_22a_4_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_22a_4_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_22a_4", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_22a_4_n_slp <- cowplot::plot_grid(plot_xy_22a_4_n_slp, NULL, plot_xz_22a_4_n_slp, NULL, plot_yz_22a_4_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_22a_4_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "26a_1_n_slp"
plot_xy_26a_1_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26a_1_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26a_1_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_26a_1", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_26a_1_n_slp <- cowplot::plot_grid(plot_xy_26a_1_n_slp, NULL, plot_xz_26a_1_n_slp, NULL, plot_yz_26a_1_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26a_1_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "26a_2_n_slp"
plot_xy_26a_2_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26a_2_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26a_2_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_26a_2", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_26a_2_n_slp <- cowplot::plot_grid(plot_xy_26a_2_n_slp, NULL, plot_xz_26a_2_n_slp, NULL, plot_yz_26a_2_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26a_2_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "26a_4_n_slp"
plot_xy_26a_4_n_slp <-  complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1,0, IN, MN, Size, GS, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26a_4_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26a_4_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, N_A, color_range_ext, transpa, color_gs, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_26a_4", x = 0.5,hjust = 0, fontface = 'bold',size = 28)
Fig_26a_4_n_slp <- cowplot::plot_grid(plot_xy_26a_4_n_slp, NULL, plot_xz_26a_4_n_slp, NULL, plot_yz_26a_4_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26a_4_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 14** Distribution of infected neurons (primary infected motor neurons or secondary infected interneurons are not distinguished) of 4 GS (blue) single injections of rabies and AAV-CAG-Flex-oG in *Chat::Cre+/-* mice (Salk).

### Figure 12 figure supplement 15
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
Animal <- "26_1_n_slp"
plot_xy_26_1_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,  0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26_1_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26_1_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("TA_26_1", x = 0.505,hjust = 0, fontface = 'bold',size = 28)
Fig_26_1_n_slp <- cowplot::plot_grid(plot_xy_26_1_n_slp, NULL, plot_xz_26_1_n_slp, NULL, plot_yz_26_1_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26_1_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "26_3_n_slp"
plot_xy_26_3_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26_3_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26_3_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("TA_26_3", x = 0.505,hjust = 0, fontface = 'bold',size = 28)
Fig_26_3_n_slp <- cowplot::plot_grid(plot_xy_26_3_n_slp, NULL, plot_xz_26_3_n_slp, NULL, plot_yz_26_3_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26_3_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "26_4_n_slp"
plot_xy_26_4_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1,  0, IN, MN, Size, TA, N_A, kernel, bin, method_used, titer_used, panel)
plot_xz_26_4_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_26_4_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, ta, N_A, color_range_flex, transpa, color_ta, transpa, 1, 0, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("TA_26_4", x = 0.505,hjust = 0, fontface = 'bold',size = 28)
Fig_26_4_n_slp <- cowplot::plot_grid(plot_xy_26_4_n_slp, NULL, plot_xz_26_4_n_slp, NULL, plot_yz_26_4_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_26_4_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 15** Distribution of infected neurons (primary infected motor neurons or secondary infected interneurons are not distinguished) of 3 TA (yellow) single injections of rabies and AAV-CAG-Flex-oG in *Chat::Cre+/-* mice (Salk).

### Figure 12 figure supplement 16
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
kernel <- 6
method_used <- "PRV_WT"
Animal <- "1_1_n_slp"
plot_xy_1_1_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_1_1_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1_1_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_TA_1_1", x = 0.49,hjust = 0, fontface = 'bold',size = 28)
Fig_1_1_n_slp <- cowplot::plot_grid(plot_xy_1_1_n_slp, NULL, plot_xz_1_1_n_slp, NULL, plot_yz_1_1_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1_1_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "1_4_n_slp"
plot_xy_1_4_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_1_4_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_1_4_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_TA_1_4", x = 0.49,hjust = 0, fontface = 'bold',size = 28)
Fig_1_4_n_slp <- cowplot::plot_grid(plot_xy_1_4_n_slp, NULL, plot_xz_1_4_n_slp, NULL, plot_yz_1_4_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_1_4_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "2_2_n_slp"
plot_xy_2_2_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_2_2_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_2_2_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_TA_2_2", x = 0.49,hjust = 0, fontface = 'bold',size = 28)
Fig_2_2_n_slp <- cowplot::plot_grid(plot_xy_2_2_n_slp, NULL, plot_xz_2_2_n_slp, NULL, plot_yz_2_2_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_2_2_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```

```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 12 , fig.height= 4}
Animal <- "3_3_n_slp"
plot_xy_3_3_n_slp <- complete_graph_xy_density(spinal, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1,1, IN, N_A, Size, GS, TA, kernel, bin, method_used, titer_used, panel)
plot_xz_3_3_n_slp <- complete_graph_xz_density(spinal_rc, spinal_rc_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1,  1, IN, MN, Size, kernel, bin, method_used, titer_used)
plot_yz_3_3_n_slp <- complete_graph_yz_density(spinal_rc_y, spinal_rc_y_contra, density, concatenate, data_all, Animal, gs, ta, color_range_ext, color_range_flex, color_gs, color_ta, 1, 1, IN, MN, Size, kernel, bin, titer_used, method_used)

title <- ggdraw() + draw_label("GS_TA_3_3", x = 0.49,hjust = 0, fontface = 'bold',size = 28)
Fig_3_3_n_slp <- cowplot::plot_grid(plot_xy_3_3_n_slp, NULL, plot_xz_3_3_n_slp, NULL, plot_yz_3_3_n_slp, ncol=5,nrow = 1, align = "hv", rel_widths = c(5,-0.3, 4.3,-0.3,3.7), rel_heights = 3)
plot_grid(title, Fig_3_3_n_slp,ncol = 1, rel_heights = c(0.1, 1))
```
**Figure 12 figure supplement 16** Distribution of infected neurons (primary infected motor neurons or secondary infected interneurons are not distinguished) of 4 GS (blue) and TA (yellow) double injections of PRV-152 and PRV-614 in wild type mice (Salk).

### Table 1

|   Code   | Lab  | Injection | Perfusion | Muscle | Titre (I.U.) | MNs | Double labelled MNs | Ipsi dorsal median (μm) | Ipsi premotor INs | Contra premotor INs | Total premotor INs | premotor INs/MNs ratio | Section sampling |
|:--------:|:----:|:---------:|:---------:|:------:|:------------:|:---:|:-------------------:|:-----------------------:|:-----------------:|:-------------------:|:------------------:|:----------------------:|:----------------:|
|          |      |           |           |        |              |     |                     |                         |                   |                     |                    |                        |                  |
| 170427n2 | UCL  |    P2     |    P10    |   LG   |  1×10¨^10^   | 46  |          3          |           285           |        741        |         116         |        857         |           19           |   1/2 (30 μm)    |
|          |      |           |           |   TA   |   5×10^9^    | 41  |                     |           298           |        912        |         88          |        1000        |           24           |                  |
| 170427n3 | UCL  |    P2     |    P10    |   LG   |   1×10^10^   | 32  |          2          |           267           |        620        |         87          |        707         |           22           |   1/2 (30 μm)    |
|          |      |           |           |   TA   |   5×10^9^    |  6  |                     |           276           |        386        |         34          |        420         |           70           |                  |
| 170503n6 | UCL  |    P2     |    P10    |   LG   |   1×10^10^   | 83  |          1          |           294           |       1935        |         639         |        2574        |           31           |   1/2 (30 μm)    |
|          |      |           |           |   TA   |   5×10^9^    | 55  |                     |           315           |       1887        |         315         |        2202        |           40           |                  |
| 170125n3 | UCL  |    P2     |    P10    |   LG   |   5×10^9^    | 39  |          0          |           365           |        670        |         107         |        777         |           20           |   1/2 (30 μm)    |
|          |      |           |           |   MG   |   5×10^9^    | 39  |                     |           353           |        819        |         307         |        1126        |           29           |                  |
| 170508n7 | UCL  |    P2     |    P10    |   LG   |   1×10^10^   | 110 |          3          |           352           |       1955        |         382         |        2337        |           21           |   1/2 (30 μm)    |
|          |      |           |           |   MG   |   5×10^9^    | 67  |                     |           322           |       1497        |         429         |        1926        |           29           |                  |
| 170125n7 | UCL  |    P2     |    P10    |   TA   |   5×10^9^    | 47  |          0          |           363           |        907        |         308         |        1215        |           26           |   1/2 (30 μm)    |
|          |      |           |           |   PL   |   5×10^9^    | 39  |                     |           380           |       1044        |         195         |        1239        |           32           |                  |
| 170125n8 | UCL  |    P2     |    P10    |   TA   |   5×10^9^    | 22  |          2          |           343           |        920        |         157         |        1077        |           49           |   1/2 (30 μm)    |
|          |      |           |           |   PL   |   5×10^9^    | 22  |                     |           330           |        741        |         83          |        824         |           37           |                  |
|   1570   | UoG  |    P1     |    P10    |   LG   |   2×10^8^    | 11  |         \-          |           322           |       1111        |         404         |        1515        |          138           |   1/8 (60 μm)    |
|   1571   | UoG  |    P1     |    P10    |   LG   |   2×10^8^    | 12  |         \-          |           340           |        760        |         196         |        956         |           80           |   1/8 (60 μm)    |
|   1573   | UoG  |    P1     |    P10    |   TA   |   5×10^8^    | 10  |         \-          |           332           |        447        |         68          |        515         |           52           |   1/8 (60 μm)    |
|   1574   | UoG  |    P1     |    P10    |   TA   |   5×10^8^    | 14  |         \-          |           365           |        297        |         26          |        323         |           23           |   1/8 (60 μm)    |
|   1577   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    | 18  |          2          |           329           |        313        |         43          |        356         |           20           |   1/8 (60 μm)    |
|          |      |           |           |   TA   |   5×10^9^    | 26  |                     |           312           |        688        |         105         |        793         |           31           |                  |
|   1578   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    | 21  |          5          |           330           |        292        |         34          |        326         |           16           |   1/8 (60 μm)    |
|          |      |           |           |   TA   |   5×10^9^    | 22  |                     |           346           |        790        |         130         |        920         |           42           |                  |
|   1579   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    | 30  |          1          |           322           |       1023        |         194         |        1217        |           41           |   1/8 (60 μm)    |
|          |      |           |           |   MG   |   5×10^8^    |  7  |                     |           306           |        169        |         19          |        188         |           27           |                  |
|   1580   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    | 14  |          0          |           316           |        414        |         48          |        462         |           33           |   1/8 (60 μm)    |
|          |      |           |           |   MG   |   5×10^8^    |  8  |                     |           348           |        470        |         87          |        557         |           70           |                  |
|   1605   | UoG  |    P1     |    P10    |   MG   |   1×10^8^    |  6  |         \-          |           340           |        412        |         110         |        522         |           87           |   1/8 (60 μm)    |
|   1611   | UoG  |    P1     |    P10    |   PL   |   1×10^8^    |  2  |         \-          |           328           |        167        |         24          |        191         |           96           |   1/8 (60 μm)    |
|   1613   | UoG  |    P2     |    P10    |   PL   |   1×10^8^    |  1  |         \-          |           340           |        164        |         16          |        180         |          180           |   1/8 (60 μm)    |
|   1639   | UoG  |    P2     |    P10    |   TA   |   2×10^8^    | 15  |         \-          |           341           |        591        |         94          |        685         |           46           |   1/8 (60 μm)    |
|   1640   | UoG  |    P2     |    P10    |   PL   |   2×10^8^    | 20  |         \-          |           322           |        629        |         122         |        751         |           38           |   1/8 (60 μm)    |
|   1644   | UoG  |    P2     |    P10    |   LG   |   1×10^8^    |  1  |         \-          |           344           |        142        |         32          |        174         |          174           |   1/8 (60 μm)    |
|          |      |           |           |   TA   |   2×10^8^    | \-  |                     |           296           |        57         |         11          |         68         |           \-           |                  |
|   1646   | UoG  |    P2     |    P10    |   LG   |   1×10^8^    |  1  |         \-          |           261           |        90         |         16          |        106         |          106           |   1/8 (60 μm)    |
|          |      |           |           |   TA   |   2×10^8^    |  3  |                     |           305           |        76         |         13          |         89         |           30           |                  |
|   1653   | UoG  |    P2     |    P10    |   LG   |   1×10^8^    |  2  |         \-          |           307           |        60         |          6          |         66         |           33           |   1/8 (60 μm)    |
|          |      |           |           |   TA   |   2×10^8^    |  2  |                     |           312           |        58         |          8          |         66         |           33           |                  |
|   1656   | UoG  |    P2     |    P10    |   LG   |   1×10^8^    | \-  |         \-          |           311           |        563        |         145         |        708         |           \-           |   1/8 (60 μm)    |
|   1657   | UoG  |    P2     |    P10    |   LG   |   1×10^8^    |  1  |         \-          |           321           |        323        |         51          |        374         |          374           |   1/8 (60 μm)    |
|   1660   | UoG  |    P2     |    P10    |   MG   |   2×10^8^    |  7  |         \-          |           324           |        509        |          3          |        512         |           73           |   1/8 (60 μm)    |
|   1661   | UoG  |    P2     |    P10    |   MG   |   2×10^8^    | 10  |         \-          |           338           |        175        |         63          |        238         |           24           |   1/8 (60 μm)    |
|   1662   | UoG  |    P2     |    P10    |   MG   |   2×10^8^    | 10  |         \-          |           313           |        375        |         230         |        605         |           61           |   1/8 (60 μm)    |
|   1701   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    |  8  |          2          |           351           |        169        |         26          |        195         |           24           |   1/8 (60 μm)    |
|          |      |           |           |   MG   |   5×10^9^    | 34  |                     |           329           |        594        |         190         |        784         |           23           |                  |
|   1702   | UoG  |    P2     |    P10    |   LG   |   2×10^9^    | 14  |          2          |           331           |        561        |         107         |        668         |           48           |   1/8 (60 μm)    |
|          |      |           |           |   MG   |   5×10^9^    |  2  |                     |           322           |        76         |         11          |         87         |           44           |                  |
|   353    | MDC  |    P4     |    P10    |   GS   |   1×10^9^    | 31  |         \-          |           283           |       1542        |         431         |        1973        |           64           |   All (40 μm)    |
|   399    | MDC  |    P4     |    P10    |   GS   |   1×10^9^    | 41  |         \-          |           286           |        569        |         77          |        646         |           16           |   All (40 μm)    |
|   1332   | MDC  |    P4     |    P10    |   GS   |   1×10^9^    | 18  |         \-          |           317           |       1605        |         323         |        1928        |          107           |   All (40 μm)    |
|   1349   | MDC  |    P4     |    P10    |   GS   |   1×10^9^    | 18  |         \-          |           305           |       1416        |         459         |        1875        |          104           |   All (40 μm)    |
|   700    | MDC  |    P4     |    P10    |   TA   |   1×10^9^    | 47  |         \-          |           318           |       1723        |         122         |        1845        |           39           |   All (40 μm)    |
|   721    | MDC  |    P4     |    P10    |   TA   |   1×10^9^    | 22  |         \-          |           310           |       1934        |         465         |        2399        |          109           |   All (40 μm)    |
|   1324   | MDC  |    P4     |    P10    |   TA   |   1×10^9^    | 17  |         \-          |           292           |       2041        |         301         |        2342        |          138           |   All (40 μm)    |
|    1     | Salk |    P2     |    P10    |   GS   |   1×10^11^   | N/A |         N/A         |           328           |       9185        |        2735         |       11920        |          N/A           |   All (60 μm)    |
|          |      |           |           |   TA   |   1×10^11^   | N/A |         N/A         |           349           |       3330        |         731         |        4061        |          N/A           |                  |
|    2     | Salk |    P2     |    P10    |   GS   |   1×10^11^   | N/A |         N/A         |           303           |       8827        |        3867         |       12694        |          N/A           |   All (60 μm)    |
|          |      |           |           |   TA   |   1×10^11^   | N/A |         N/A         |           294           |       3198        |        1132         |        4330        |          N/A           |                  |
|    a     | Salk |    P1     |    P8     |   GS   |   1×10^10^   | N/A |         N/A         |           248           |        334        |         42          |        376         |          N/A           |   1/9 (30 μm)    |
|    b     | Salk |    P1     |    P8     |   GS   |   1×10^10^   | N/A |         N/A         |           237           |        275        |         30          |        305         |          N/A           |   1/9 (30 μm)    |
|  22a_4   | Salk |    P2     |    P9     |   GS   |   3×10^11^   | N/A |         N/A         |           403           |        464        |         58          |        522         |          N/A           |   All (60 μm)    |
|  26a_1   | Salk |    P2     |    P9     |   GS   |   3×10^11^   | N/A |         N/A         |           383           |        941        |         91          |        1032        |          N/A           |   All (60 μm)    |
|  26a_2   | Salk |    P2     |    P9     |   GS   |   3×10^11^   | N/A |         N/A         |           351           |       1910        |         401         |        2311        |          N/A           |   All (60 μm)    |
|  26a_4   | Salk |    P2     |    P9     |   GS   |   3×10^11^   | N/A |         N/A         |           382           |       1923        |         392         |        2315        |          N/A           |   All (60 μm)    |
|   26_1   | Salk |    P2     |    P9     |   TA   |   3×10^11^   | N/A |         N/A         |           348           |       3236        |         263         |        3499        |          N/A           |   All (60 μm)    |
|   26_3   | Salk |    P2     |    P9     |   TA   |   3×10^11^   | N/A |         N/A         |           367           |       2078        |         465         |        2543        |          N/A           |   All (60 μm)    |
|   26_4   | Salk |    P2     |    P9     |   TA   |   3×10^11^   | N/A |         N/A         |           350           |       2494        |         597         |        3091        |          N/A           |   All (60 μm)    |
| 1_1 PRV  | Salk |    P11    |    P13    |   GS   |   1×10^9^    | N/A |         N/A         |           318           |        430        |         54          |        484         |          N/A           |   1/4 (60 μm)    |
|          |      |           |           |   TA   |   1×10^9^    | N/A |         N/A         |                         |                   |                     |                    |          N/A           |                  |
| 1_4 PRV  | Salk |    P11    |    P13    |   GS   |   1×10^9^    | N/A |         N/A         |           349           |        238        |         23          |        261         |          N/A           |   1/4 (60 μm)    |
|          |      |           |           |   TA   |   1×10^9^    | N/A |         N/A         |                         |                   |                     |                    |          N/A           |                  |
| 2_2 PRV  | Salk |    P11    |    P13    |   GS   |   1×10^9^    | N/A |         N/A         |           357           |        515        |         82          |        597         |          N/A           |   1/4 (60 μm)    |
|          |      |           |           |   TA   |   1×10^9^    | N/A |         N/A         |                         |                   |                     |                    |          N/A           |                  |
| 3_3 PRV  | Salk |    P11    |    P13    |   GS   |   1×10^9^    | N/A |         N/A         |           377           |       1005        |         53          |        1058        |          N/A           |   1/4 (60 μm)    |
|          |      |           |           |   TA   |   1×10^9^    | N/A |         N/A         |                         |                   |                     |                    |          N/A           |                  |

**Table 1**: details of individual experiments performed in the four different laboratories (University College London, UCL, University of Glasgow, UoG, Max Delbrück Center for Molecular Medicine, MDC, Salk Institute), with individual cell count and virus concentration. Same experimental code as in Figure 3D-E, 4D-E, 5B-C-F-G, 6B-E-H-K, 8E,9E,11E and figure supplements. Experiments performed at the Salk Institute (except the PRV-Bartha experiments) were co-injections of either AAV6-B19G (3×10^12^ titre) or of AAV6 - CAG-Flex (1×10^12^ titre). AAV and RV were injected in a 3:1 ratio. For all labs and muscles the total volume injected was 1 μl, with the exception of PRV-Bartha experiments, where 0.5 μl were injected. In the Salk Institute experiments, no attempt was made to identify primarily infected motor neurons, therefore the cell count refers to both motor neurons and interneurons.

### Table 2

+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| Method                                                                                                                         | Pros                                                                                                                                                    | Cons                                                                                                                                                             | Outcome                             | Reference                |
+================================================================================================================================+=========================================================================================================================================================+==================================================================================================================================================================+=====================================+==========================+
| Muscle injection of AAV-G (serotype 2.6) + RabV (Figure 1A)                                                                    | Avoids the possibility of retrograde disynaptic transfer from second order motor neurons due to restriction of G expression to targeted motor neurons   | The labelled premotor population could be contaminated by anterogradely labelled neurons from primary sensory neurons.                                           | Flexor- extensor segregation        | (Tripodi et al., 2011)   |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
|                                                                                                                                | Avoids the possibility of retrograde disynaptic transfer from premotor spinal interneurons.                                                             |                                                                                                                                                                  | No flexor- extensor segregation      |  Present study         |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| Muscle injection of AAV-flex-G (serotype 2.6) + RabV in *ChAT::Cre* mice (Figure 1B)                                           |  Avoids the possibility of retrograde disynaptic transfer from second order motor neurons due to restriction of G expression to targeted motor neurons  | Conditional expression of G may be inefficient                                                                                                                    | Flexor-extensor segregation         | (Wang et al., 2017)         |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
|                                                                                                                                | Avoids the possibility of retrograde disynaptic transfer from premotor spinal interneurons.                                                             |                                                                                                                                                                  | No flexor- extensor segregation      |  Present study         |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
|                                                                                                                                | Avoids potential anterograde sensory contamination.                                                                                                     |                                                                                                                                                                  |                                     |                          |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| Central injection of AAV-flex-G (serotype 2.9) in *ChAT::Cre* mice followed by muscle injection of RabV, in adults (Figure 1C) | Limits the issue of potential disynaptic transfer from cholinergic interneurons                                                                          | Potential for disynaptic transfer from cholinergic premotor interneurons, transsynaptically labelled motor neurons and mis-targeted primary motor neurons.      | Flexor-extensor segregation         | (Takeoka and Arber, 2019) |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
|                                                                                                                                | Avoids potential anterograde tracing from sensory neurons                                                                                               |                                                                                                                                                                  |                                     |                          |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| Genetically driven expression of G in *ChAT::Cre* or *Olig2::Cre* mice + muscle RabV injection in neonates                     | Avoids potential anterograde tracing from sensory neurons                                                                                               | Potential for disynaptic transfer from premotor spinal interneurons, transsynaptically labelled motor neurons and mis-targeted primary motor neurons             | No flexor-extensor segregation      |Present study                    |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| (Figure 1D and E)                                                                                                              | Ensures homogenous expression of G in all motor neurons                                                                                                 |                                                                                                                                                                  |                                     |                          |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+
| Muscle injection of PRV-Bartha with strictly timed fixation of tissue (Figure 1F)                                              | High efficiency in transsynaptic transmission. Not reliant on viral recombination.                                                                      | Timed fixation does not guarantee that transsynaptic jumps occur only up to the second order                                                                     | No flexor-extensor segregation      | Present study             |
+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+--------------------------+

**Table 2**: Summary of pros and cons of each described method

# Methods

### Key Resources Table

+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| **Reagent type (species) or resource**                   | **Designation**                                                               | **Source or reference**                                 | **Identifiers**                             | **Additional information**                                                                  |
+==========================================================+===============================================================================+=========================================================+=============================================+=============================================================================================+
| strain, strain background (include species and sex here) | ΔG-Rab-eGFP                                                                   | Gift from M. Tripodi lab, LMCB Cambridge                |                                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| strain, strain background (include species and sex here) | ΔG-Rab-mCherry                                                                | Gift from M. Tripodi lab, LMCB Cambridge                |                                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| strain, strain background (include species and sex here) | AAV6-Ef1a-B19G                                                                | Produced by Applied Viromics (USA)                      |                                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| strain, strain background (include species and sex here) | AAV6-CAG-Flex-oG                                                              | Produced at the Salk GT3 virus core facility            |                                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| genetic reagent (M. musculus)                            | ChAT-IRES-Cre                                                                 | Jackson laboratory                                      | IMSR Cat\# JAX:006410; RRID:IMSR_JAX:006410 | allele symbol: Chattm2(cre)Lowl; maintained on a C57BL6/J background                        |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| genetic reagent (M. musculus)                            | Olig2-Cre                                                                     | Jackson laboratory                                      | IMSR Cat\# JAX:025567; RRID:IMSR_JAX:025567 | allele symbol: B6.129-Olig2tm1.1(cre)Wdr/J maintained on a C57BL6/J background              |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| genetic reagent (M. musculus)                            | RΦGT                                                                          | Jackson laboratory                                      | IMSR Cat\# JAX:024708; RRID:IMSR_JAX:024708 | allele symbol: Gt(ROSA)26Sortm1(CAG-RABVgp4,-TVA)Arenk; maintained on a C57BL6/J background |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| genetic reagent (M. musculus)                            | GlyT2-eGFP                                                                    | Gift from H. Zeilhofer lab, University of Zurich        | IMSR Cat\# RBRC04708; RRID:IMSR_RBRC04708   | allele symbol: Tg(Slc6a5-EGFP)1Uze; maintained on a C57BL6/J background                     |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| cell line (Homo-sapiens, female)                         | HEK293t/17                                                                    | Gift from M. Tripodi lab, LMCB Cambridge                | RRID:CVCL_1926                              | ATCC, cat. no. CRL-1126                                                                     |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| cell line (Mesocricetus auratus, male)                   | BHK-21                                                                        | Gift from M. Tripodi lab, LMCB Cambridge                | RRID: CVCL_1915                             | ATCC \# CCL-10                                                                              |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| cell line (Mesocricetus auratus, male)                   | BHK-G                                                                         | Gift from M. Tripodi lab, LMCB Cambridge                | RRID:CVCL_1915                              | Derived from ATCC \# CCL-10                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-ChAT (Goat polyclonal)                                                   | Millipore                                               | Cat\# AB144P; RRID:AB_2079751               | IF (1:100)                                                                                  |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-mCherry (Chicken polyclonal)                                             | Abcam                                                   | Cat\# ab205402; RRID:AB_2722769             | IF (1:2500)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-GFP (Rabbit polyclonal)                                                  | Abcam                                                   | Cat\# ab290; RRID:AB_303395                 | IF (1:2500)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-vGluT2 (Guinea pig polyclonal)                                           | Millipore                                               | Cat\# AB2251-I; RRID:AB_2665454             | IF (1:2500)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-Isl1 (Guinea pig polyclonal)                                             | Gift from T. Jessell lab, Columbia University, New York |                                             | IF (1:7500)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-guinea pig IgG H&L Alexa Fluor® 647 (Donkey polyclonal)                  | Millipore                                               | Cat\# AP193SA6; RRID:AB_2340477             | IF (1:700)                                                                                  |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-Goat IgG H&L Alexa Fluor® 405 (Donkey polyclonal preadsorbed)            | Abcam                                                   | Abcam Cat\# AB175665; RRID:AB_2636888       | IF (1:200)                                                                                  |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-Rabbit IgG H&L Alexa Fluor®488 (Donkey polyclonal Highly Cross-Adsorbed) | Thermo Fisher Scientific                                | Cat\# A-21206; RRID:AB_2535792              | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| antibody (UCL)                                           | anti-Chicken IgY (IgG) H&L Cy3-AffiniPure (Donkey polyclonal)                 | Jackson ImmunoResearch Labs                             | Cat\# 703-165-155; RRID:AB_2340363          | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Glasgow University)                            | anti-GFP (chicken polyclonal)                                                 | Abcam                                                   | Cat\# Ab13970; RRID:AB_300798               | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Glasgow University)                            | anti-mCherry (rabbit polyclonal)                                              | Abcam                                                   | Cat\# Ab167453; RRID:AB_2571870             | IF (1:2000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Glasgow University)                            | anti-chicken IgY H&L Alexa Fluor®488 (Donkey polyclonal)                      | Jackson ImmunoResearch Labs                             | Cat\# 703-545-155; RRID:AB_2340363          | IF (1:500)                                                                                  |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (MDC)                                           | anti-ChAT (rabbit polyclonal)                                                 | Abcam                                                   | Cat\# Ab2750952; RRID:AB_2750952            | IF (1:16000)                                                                                |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (MDC)                                           | anti-Rabbit IgG H&L Alexa Fluor®488 (Donkey polyclonal)                       | Thermo Fisher Scientific                                | Cat\# A-21206; RRID:AB_2535792              | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Salk)                                          | anti-GFP (goat polyclonal)                                                    | Rockland                                                | Cat\#600-101-215; RRID:AB_218182            | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Salk)                                          | anti-RFP (rabbit polyclonal)                                                  | Rockland                                                | Cat\#600-401-379; RRID:AB_2209751           | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Salk)                                          | anti-goat IgY H&L Alexa Fluor®488 (Donkey polyclonal)                         | Invitrogen                                              | Cat\#A11055; RRID:AB_2534102                | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| Antibody (Salk)                                          | anti-rabbit IgY H&L Alexa Fluor®555 (Donkey polyclonal)                       | Invitrogen                                              | Cat\#A32794; RRID:AB_2762834                | IF (1:1000)                                                                                 |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| chemical compound, drug                                  | Mowiol® 4-88                                                                  | Sigma Aldrich                                           | Cat\# 81381-250G                            |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| software, algorithm                                      | ZEN Digital Imaging for Light Microscopy: Zen Blue 2.3                        | Carl Zeiss light microscopy imaging systems             | RRID:SCR_013672                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| software, algorithm                                      | Imaris 9.1                                                                    | Bitplane                                                | RRID:SCR_007370                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| software, algorithm                                      | Adobe illustrator version CC 2019                                             | Adobe                                                   | RRID:SCR_010279                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+
| software, algorithm                                      | Matlab version 2021b                                                          | Mathworks                                               | RRID:SCR_001622                             |                                                                                             |
+----------------------------------------------------------+-------------------------------------------------------------------------------+---------------------------------------------------------+---------------------------------------------+---------------------------------------------------------------------------------------------+

### Experimental settings

The experiments were performed and analysed across 4 different laboratories. The injections labelled as UoG (University of Glasgow) were performed in the Beato lab at UCL (three different operators) and processed in Glasgow (Maxwell and Todd labs) using RabV produced in the Beato lab. The injections labelled as UCL were performed in the Brownstone lab (two different operators) using RabV produced in the Brownstone lab and the tissue was processed at UCL (Brownstone lab). The injections in *Olig2::Cre* mice were performed and analysed at the Max Delbrück Center (MDC), with locally produced RabV. The injections of AAV-G and AAV-G-Flex in wild type and *ChAT::Cre* mice respectively were performed and analysed at the Salk Institute. The AAV-G construct was produced from Applied Viromics, while the AAV-G-Flex was produced at the Salk GT3 Core and the RabV was obtained from Janelia Farm. Experiments with PRV-Bartha rabies strain were performed and analysed at the Salk Institute. PRV-Bartha was obtained from NIH CNNV.

### Animal Experimentation Ethical Approval

All experiments at UCL were carried out according to the Animals (Scientific Procedures) Act UK (1986) and were approved by the UCL AWERB committee under project licence number 70/9098. All experiments performed at the MDC were carried out in compliance with the German Animal Welfare Act and approved by the Regional Office for Health and Social Affairs Berlin (LAGeSo). All experiments performed at the Salk Institute were conducted in accordance with IACUC and AAALAC guidelines of the Salk Institute for Biological Studies. Experimental design followed the ARRIVE guidelines.

### Mouse strains

Homozygous *ChAT::Cre* mice (Rossi et al., 2011, Jackson lab, stock \#006410) or heterozygous *Olig2::Cre* mice (Dessaud et al., 2007) were crossed with homozygous *RΦGT* mice (Jackson Lab, stock \#024708), to generate *ChAT::Cre +/-; RΦGT +/-* or *Olig2::Cre +/-; RΦGT +/-* mice (Takatoh et al., 2013; Skarlatou et al., 2020) that were used for rabies tracing experiments. For experiments aimed at distinguishing excitatory and inhibitory populations of premotor interneurons, we crossed homozygous *ChAT::Cre* mice with heterozygous *GlyT2-eGFP* mice (a gift from Prof. Zeilhofer, University of Zurich, Zeilhofer et al., 2005) and their double-positive offspring were mated with the homozygous *RΦGT* mice. AAV-G-Flex constructs were injected in heterozygous *ChAT::Cre* mice.

To quantify possible "leak" of Cre expression in the spinal cord, we crossed *ChAT::Cre* and *RCL-tdTom* (Ai9, Jax stock \#007909) mice (13 sections from 3 mice), and found tdTom expression in cholinergic motor neurons and interneurons as expected, as well as in some non-cholinergic neurons distributed in intermediate (10%, 70/690) and dorsal (9%, 62/690) laminae, with the remaining 35 located ventrally (Figure 13) indicating that ectopic expression of Cre in ChAT negative neurons is minimal but not nil, and mostly confined to superficial dorsal laminae, an area that is devoid of premotor interneurons, indicating that while double jumps from "leaky" Cre expressing interneurons remains a possibility, its extent would not be sufficient to alter the distribution of labelled interneurons. The possible "leak" of G and the avian receptor protein (TVA) expression in *RΦGT* mice was then tested by injecting EnvA-ΔG-Rab-eGFP, produced according to standard protocols (Osakada and Callaway, 2013) to a titre of 1×10^9^ IU/ml. Lack of contamination from non-pseudotyped virus was confirmed by infecting HEK cells at high (up to 20) multiplicity of infection. Three RΦGT heterozygous mice were injected in the lateral gastrocnemius muscle at P1 and fixed 9 days post injections. The tissue was cut as described below, but along the sagittal plane in 60 μm sections, in order to isolate the dorsal motor column. Following immunoreaction for eGFP, in each of the 3 cords, we found a maximum of 3 labelled motor neurons (1, 1 and 3 motor neurons in n=3 animals) but no interneurons labelled, indicating some leakage in the expression of the TVA-IRES-glycoprotein cassette from the *RΦGT* mice, but insufficient G expression to support transsynaptic jumps (Figure 13, figure supplement 1).

### Virus production

Rabies virus used in experiments performed at UCL, was obtained from in house stocks of a variant of the SAD-B19 rabies strain where the sequence coding for the glycoprotein was replaced by the sequence for either eGFP or mCherry (Wickersham et al., 2007). Virus was produced at high concentration using the protocol described in (Osakada and Callaway, 2013): baby hamster kidney cells stably expressing the rabies glycoprotein (BHK-G, kindly provided by Dr. Tripodi (LMCB, Cambridge) were thawed and plated in standard Dulbecco modified medium, supplemented with 10% fetal bovine serum (FBS), incubated at 37°C with 5% CO2 and split until \~70% confluence was obtained in 5 dishes (10 ml medium each). The cells were then inoculated at 0.2-0.3 multiplicity of infection with either the ΔG-RabV-eGFP or the ΔG-RabV-mCherry (initial samples kindly provided by Prof. Arber and Dr. Tripodi). Cells were incubated for 6 hours at 35°C and 3% CO2 and then split 1 to 4 into 20 dishes (10 ml) with 10% FBS medium and kept at 37°C and 5% CO2 for 12-24 hours, until \~70% confluent and medium was then replaced with 2% FBS medium and cells incubated at 35°C and 3% CO2 for virus production. The supernatant was collected after \~3 days, new medium was added for another round of production and supernatant was filtered (0.45 μm filter) and centrifuged for 2 hours at 19400 rpm (SW28 Beckman rotor). The pellets were suspended in phosphate buffer saline (PBS), dispersed and collected in a single tube and further centrifuged for 4 hours at 21000 rpm in a 20% sucrose gradient (SW55 Beckman rotor). The resulting pellet was suspended in 100 μl PBS and the virus was aliquoted (5-10 μl) and stored in a -80° freezer. The viral titre of each round of production was measured by serial 10-fold dilution of three different aliquots using standard protocols (Osakada and Callaway, 2013). For each injection, the virus titre is reported in Table 1. In a subset of experiments, we diluted the virus 10-fold in order to limit the number of starter cells. Rabies virus used for experiments performed at MDC was produced as previously described (Skarlatou et al., 2020). For the premotor labelling experiments mediated through co-injection of AAV-Flex-G and delta-G-RabV at the Salk Institute, pSADΔG-mCherry and pSADΔG-eGFP were obtained from Janelia Viral Tools facility. AAV6-Ef1a-B19G (1x10^13^ GC/mL) was generated at Applied Viriomics, while AAV6-CAG-Flex-oG was generated at the GT3 core at the Salk Institute. For retrograde PRV-Bartha experiments, PRV-152 (GFP) and PRV-614 (mRFP) were obtained from the Center for Neuroanatomy with Neurotropic Viruses (CNNV).

### Intramuscular injection

Neonatal pups (P1-P2) were anaesthetized using isoflurane inhalation and an incision was made on the skin to expose the belly of the targeted muscle, either lateral or medial gastrocnemius (LG, MG, whose primary role is that of ankle extensors, but also contribute to knee flexion), tibialis anterior (TA) or peroneus longus (PL, ankle flexors). The virus was injected intramuscularly using a 5 μl Hamilton syringe (model 7652-01) fixed to a manual Narishige micromanipulator (M-3333) and loaded with a bevelled glass pipette of inner diameter 50-70 μm. The volume injected was 1 μl, compatible with the estimated volume of the muscles at this age (\~2 μl), in order to minimize the risk of leaks to adjacent muscles. Viral batches of similar titres were slowly (\> 1 minute) injected, the skin incision was sutured with Vicryl 8-0 (Ethicon, USA) and the pups received a subcutaneous injection of carprofen (10%) for pain management. Mice were closely monitored for the next 24 hours for signs of movement impairment and were perfused under terminal general anaesthesia 8-9 days after injection.

In order to compare directly the distributions of flexor and extensor associated premotor interneurons and avoid confounding factors in the coordinate representations of these interneurons across different spinal cords, we performed double injections of ΔG-RabV in the same animal, using ΔG-Rab-eGFP and ΔG-Rab-mCherry injected in pairs of antagonist muscles, lateral gastrocnemius and tibialis anterior. For comparison, we also performed double injections in pairs of synergist muscles: LG and MG or TA and PL. Due to the close proximity of these pairs of muscles, before cutting the spinal tissue for immunohistochemistry, we dissected the injected leg and confirmed that there was no contamination of virus across the injected muscles or in adjacent muscles below or above the knee. To exclude confounding factors in our observed premotor interneuron distributions due to systematic viral interference (Ohara et al., 2009b, 2009a), in a subset of experiments single injections of ΔG-Rab-mCherry (4 LG, 4 MG, 2 TA and 4 PL) were performed in the progeny of either *ChAT:: Cre;RΦGT* or *GlyT2-eGFP;ChAT::Cre;RΦGT* mice.

For experiments performed at MDC, intramuscular injections were done as previously described (Skarlatou et al., 2020). Briefly, P4 animals were anesthetized with isoflurane and a small incision in the skin was made to reveal either the gastrocnemius (GS, 4 experiments, no attempts were made at selective targeting of the two heads of the GS muscle) or the tibialis anterior (3 experiments) muscles. A volume of 1.5 μl of ΔG-RabV-mCherry was injected in *Olig2::Cre +/-; RΦGT +/-* mice using a glass capillary. Animals were perfused at P10, 6 days after injection in order to minimize the chance of disynaptic transfer.

For wild type ΔG-RabV premotor labelling experiments performed at the Salk Institute, P2 pups were anesthetised on ice and 1 μl of a 3:1 mixture of AAV-Ef1a-B19G and ΔG-Rab-GFP or ΔG-Rab-mCherry was injected through the skin into the GS or TA muscles of the same animal, using a 10 μl Hamilton syringe with 30-gauge metal needle. Animals were fixed by perfusion under terminal general anaesthesia, as described below, 10 days post injection. For *ChAT::Cre* experiments, a 3:1 mixture of AAV-CAG-Flex-oG (Addgene \#74292) and ΔG-Rab-GFP was injected into either the GS or TA muscles of P1-P2 pups. Anaesthesia, injection volume, and syringe used were the same as above. Animals were fixed by perfusion under terminal general anaesthesia, as described below, 7 days post-injection. Hindlimbs were visualized under a stereoscope to confirm correct targeting of muscles.

For PRV experiments, intramuscular injections of PRV were performed in P11 pups under 2-3% isoflurane anaesthesia. Briefly, a small incision was made in the skin to expose the TA and GS and 0.5 μl of PRV-152 or PRV-614 was delivered to either muscle, in the same animal, using a glass needle and picospritzer. Animals were perfused 48 hours post injection. Hindlimbs were visualized under a stereoscope and sectioned to confirmed correct targeting of muscles.

### Tissue collection and immunohistochemistry

Under ketamine/xylazine terminal anaesthesia (i.p. 80 mg/kg and 10 mg/kg respectively), mice were intracardially perfused with phosphate buffer solution (0.1 M PBS), followed by 4% paraformaldehyde in PBS. The spinal cords were dissected and post-fixed for 2 hours at 4ºC, cryoprotected overnight at 4ºC in 30% PBS sucrose solution and embedded in OCT (optimal cutting temperature, Tissue-Tek, \#4583) compound.

For UCL experiments, injections were all conducted at UCL, whereas the immunohistochemistry and imaging were conducted on different animals independently in two different laboratories (Maxwell at Glasgow University and Beato/Brownstone at UCL). For tissue processing performed at UCL, lumbar spinal cords were cut (30 μm thickness) in series in the transverse plane with a cryostat (Bright Instruments, UK) mounted onto charged glass slides (VWR, \#631-0108), and stored at -20ºC. Sections were incubated for 36 hours at 4˚C with primary antibodies and overnight at 4˚C with secondary antibodies in PBS containing 0.3M NaCl, 0.2% Triton 100-X (Sigma, T9284-500ml), 7% donkey normal serum (Sigma, D9663-10ml). The primary antibodies used were: guinea pig anti-Isl1 (1:7500, from Dr. T Jessell, Columbia University, New York), goat anti choline acetyl transferase (ChAT, 1:100, Millipore, AB144P), rabbit anti-GFP (1:2500, Abcam. Ab290), chicken anti-mCherry (1:2500, Abcam, Ab205402). The secondary antibodies were: donkey anti-guinea pig Alexa 647 (1:700, Millipore, AP193SA6), donkey anti-goat preabsorbed Alexa 405 (1:200, Abcam, ab175665), donkey anti-rabbit Alexa 488 (1:1000, Thermofisher, A21206), and donkey anti-chicken Cy3 (1:1000, Jackson Immunoresearch, \#703-165-155). The slides were mounted in Mowiol (Sigma, 81381-250G) and cover-slipped (VWR, \#631-0147) for imaging.

At Glasgow University, the spinal cords were sectioned using a Leica VT1000 vibratome (thickness 60 μm) and incubated in 50% ethanol for 30 minutes. Primary antibodies used were: chicken anti-GFP (1:1000, Abcam, Ab13970), rabbit anti-mCherry (1:2000, Abcam, Ab167453) and goat anti-ChAT (1:100, Abcam, AB254118). The secondary antibodies were: donkey anti-chicken A488 (1:500, Jackson Immunoresearch, 703-545-155), and donkey anti-rabbit Rhodamine red (1:100, Jackson Immunoresearch, 711-295-152). Sections were mounted in Vectashield (Vector Laboratories, Peterborough, UK) and coverslipped. For the experiments performed at MDC, spinal cords were processed as previously described (Skarlatou et al., 2020). Briefly, animals were anesthetized by intraperitoneal injection of ketamine /xylazine mix and transcardially perfused with ice-cold PBS (until the liver was cleared of blood), followed by freshly made ice-cold 4% PFA. Spinal cords were dissected and post-fixed for 90 minutes with 4% PFA on ice. Consecutive 40 μm spinal cord cryosections including the caudal thoracic and lumbar spinal regions were obtained using a Leica cryostat and incubated overnight at 4ºC with rabbit anti-ChAT 1:16000 (Sürmeli et al., 2011 RRID:AB_2750952) followed by 1 hour incubation at room temperature with secondary antibody (Alexa-Fluor 488, 1:1000). Slides were mounted in Vectashield.

For PRV and ΔG-RabV experiments performed at the Salk Institute, animals were transcardially perfused with ice cold 1x PBS followed by ice cold 4% PFA (Electron Microscopy Sciences, 100504-858). Spinal cords were dissected and post-fixed in 4% PFA for 2 hours at 4ºC. After the post-fixation, samples were washed with 1x PBS for 10 minutes followed by a 1-3 day incubation in 30% sucrose for cryoprotection (4ºC). The lumbar spinal cord was then embedded in Tissue-Tek OCT (Sakura 4583) for cryosectioning onto glass slides (Fisherbrand Superfrost slides 12-550-15) in the transverse plane. The section thickness was 60 μm for delta-G-RabV experiments and 30 μm for PRV experiments. Immunohistochemistry was performed by incubating tissue with primary antibodies overnight at 4ºC and secondary antibodies for 2 hours at room temperature, in a 0.3% Triton-X and 20% donkey serum blocking buffer. After staining slides were mounted with a glass coverslip and Fluoromount-G (00-49-58-02). The antibodies used for this portion of the study were: goat anti-GFP (1:1000, Rockland 600-101-215), rabbit anti RFP (1:1000, Rockland 600-401-379), donkey anti-goat Alexa 488 (1:1000, Invitrogen A11055), donkey anti-rabbit-Alexa 555 (1:1000, Invitrogen A32794).

### Confocal imaging and analysis

For UCL experiments, confocal images were acquired using a Zeiss LSM800 confocal microscope with a 20x (0.8 NA) air objective and tile advanced set up function of ZEN Blue 2.3 software for imaging of the entire slice. The tiles were stitched using ZEN Blue software and cell detection was performed using Imaris (version up to 9.1, Bitplane) software. Cell counts were manually performed on every other section, in order to minimize the risk of counting the same cell twice in two consecutive sections.

For experiments performed at Glasgow University, the images were acquired using a Zeiss LSM710, with a 20x (0.8 NA) air objective and cells were counted manually using Neurolucida. Only a subset of sections was analysed (1 in every 8 consecutive sections), thus accounting for approximately 2 sections for every spinal segment.

For experiments performed at the MDC, confocal images were acquired using a Zeiss LSM800 confocal microscope. Regions of interest corresponding to each section and consisting of 8 tiles were imaged with a 10x (0.3 NA) air objective. The tiles were subsequently stitched using ZEN 2.3 Software. Acquisition and processing were performed immediately after immunohistochemistry where possible, to obtain the best possible signal. The resulting images were used for three-dimensional positional analysis as previously described (Skarlatou et al., 2020).

For experiments performed at the Salk Institute, fluorescent images were acquire using an Olympus VS-120 virtual slide scanner microscope with a 10x objective (0.3 NA). For ΔG-RabV experiments cells counts were performed on every section. For PRV experiments cell counts were performed on every 8th section. Both samples were analyzed in MATLAB using a custom script for cell detection.

A consistent system of coordinates was established using the central canal as origin of the x-y plane. The y-axis was defined as parallel to the dorso-ventral axis, with positive values towards the dorsal side and the x-axis was determined by the mediolateral direction, with positive values on the side of injection. For UCL and University of Glasgow experiments the L4-L5 border in the z direction was determined during the slicing procedure and its location was confirmed post-processing by identifying the slices with the widest mediolateral width. For MDC and Salk Institute experiments, the border between T13 and L1 was chosen as the starting point for slicing. The z coordinates obtained in different labs were subsequently aligned using the widest section as a point of reference for the border between L4 and L5 segments. For both Neurolucida and Imaris data files, in order to account for the different shapes of sections throughout the lumbar cord and deformation of individual sections, normalization of coordinates was performed independently for each quadrant using as reference points those indicated in Figure 13, figure supplement 2: the x dimension was normalized to the outer edge of the white matter at the level of the central canal, while the y dimension was normalized for each quadrant using the outermost points of the white matter for both dorsal and ventral horns. The resulting cylindrical reconstruction of the spinal cord was then scaled to the idealized spinal cord size (1700 μm in the mediolateral direction and 900 μm in the dorsoventral direction) for illustrational purposes. All coordinate transformations were performed using a custom script in MATLAB, adapted to read both Neurolucida and Imaris file formats. For experiments performed at UCL and MDC, infected motor neurons were identified by co-localization of either Isl1 or ChAT, and the presence of the reporter fluorescent protein (eGFP or mCherry) expressed after rabies infection. For experiments performed at Glasgow University, sections were not reacted for ChAT, but infected motor neurons were identified by size and location. For experiments analysed at the Salk Institute, no attempt was made to identify infected motor neurons, therefore, the resulting spatial distributions do not distinguish between primary and secondary infected cells. Since infected motor neurons are located in the ventral horn, the inclusion of motor neurons in the overall distributions does not affect our analysis of the mediolateral segregation in the dorsal horn. Distributions of infected interneurons were calculated using a Gaussian convolution with a kernel size evaluated from the standard deviation of the original data (Bhumbra and Dyball, 2010).

Gaussian convolutions were calculated splitting the transverse, normalized, spinal cord profile into ipsi- and contra-lateral, and dorsal and ventral halves, with the corresponding distributions shown surrounding the transverse spinal cord maps. Areas under the top-bottom or left-right distributions of each label sum to 1. Correlations across individual experiments were calculated from the x-y coordinates projected along the rostrocaudal axis by computing a density matrix ρn(xi ,yi) for each experiment n and evaluating the correlation coefficient rnm between experiments n and m using the formula:

```{r, echo=FALSE, warning=FALSE, fig.align='center', fig.width= 10 ,fig.height= 2}
im2 <- readPNG("~/figures/Equation_1.png")
Equation <- ggplot() + 
  background_image(im2) +
  coord_equal(ratio = 0.138) +
  annotation_custom(rasterGrob(im2, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0.5, l=0.5, r=0.5, b = 0.5, unit = "cm"))

Equation
```

Correlations across groups of experiments were calculated using the same method, after pooling all the experiments corresponding to one muscle and one injection technique. We decided not to base our analysis on p-values standard hypothesis testing for two reasons: first, the p-value obtained for any statistical test, be it parametric or not, depends strongly on the sample size (Gómez-de-Mariscal et al., 2021). Given our large sample size (coordinates for several thousands of neurons), this would lead to an unacceptable rate of false positive. Indeed, within our dataset, we obtain very low (\<10^-4^) p-values for distributions whose medians where only a few μm apart. Since the spatial resolution of our cell location procedure is no higher than the diameter of the infected cells, such results cannot be considered meaningful. Second, our data have an intrinsically nested structure, therefore, hypothesis testing on pooled experiments would lead to an error caused by ignoring the hierarchical nature of our data. Therefore, when comparing different methods of injections where distributions from multiple animals were analysed, we first performed a hierarchical bootstrap procedure (described in detail in Saravanan et al., 2020): briefly if n animals were injected with a given technique, we resampled (with replacement) the animals (first level bootstrap) and then within each animal, we resampled (with replacement) the coordinates distribution (second level bootstrap). This procedure was repeated 5000 times, giving rise to 5000 resampled experiments. In order to compare such experiments, we calculated the Hedges' G coefficient (Hedges, 1981)across pairs randomly selected within the 5000 replicas and obtained a distribution of Hedges' G. The medians of such distribution and inter quartile range (IQR) are reported for each pair of techniques or muscles we compared. When individual experiments are compared, the Hedges' G is reported for each pair of individual experiments. Rather than providing a dichotomous decision, absolute values of effect sizes can be classified as no effect (0-0.19), small (0.2-0.49), medium (0.5-0.79) and large (0.8 and above, Hedges, 1981). Sample sizes and median values are reported individually for each experiment in Table 1. All data processing was performed in MATLAB, using custom written software. The paper can be downloaded in executable format as a MATLAB live script from <https://github.com/marcobeato/Spinal_premotor_interneurons_controlling_antagonistic_muscles_are_spatially_intermingled>, where all the data are available An R version of the executable paper is available at <https://mybinder.org/v2/gh/rronzano/Spinal_premotor_interneurons_controlling_antagonistic_muscles_are_spatially_intermingled.git/HEAD?urlpath=rstudio>.

### Figure 13
```{r, fig.align='left', fig.width= 5 , fig.height= 4}
## -- Figure 13 -- ##
## -- 13A -- ##
fig13a <- readPNG("~/figures/Figure_13A.png")
Fig13A <- ggplot() + 
  background_image(fig13a) +
  coord_equal(ratio = 0.738) +
  annotation_custom(rasterGrob(fig13a, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig13A
```
```{r, echo=FALSE, warning=FALSE, fig.align='left', fig.width= 5 ,fig.height= 8}


setwd(str_c(maindir, "", "/data"))
df13_BC <- read.xlsx(xlsxFile = "fig13B-C.xlsx", sheet = 2, skipEmptyRows = TRUE, colNames=TRUE)

df13_BC$Section <- str_c(df13_BC$ID, "_", df13_BC$Section)
sectionID <- distinct(data.frame(df13_BC$Section))
sectionID <- unlist(sectionID, use.names = FALSE)
for (i in 1:length(sectionID)){
  cc <- df13_BC %>% filter(Section == as.character(sectionID[i]), Label == 0)
  for (j in 1:nrow(df13_BC)) {
    if (df13_BC$Section[j] == sectionID[i]){
      df13_BC$Xpos[j] <- df13_BC$Xpos[j] - cc$Xpos 
      df13_BC$Ypos[j] <- df13_BC$Ypos[j] - cc$Ypos
    }
  }
}

spinal_13 <- read.delim("halfcordt.txt", header = FALSE)
spinal_13$V1 <- spinal_13$V1 * 1900
spinal_13 <- spinal_13 %>% mutate(V2 = ifelse(V2 > 0, V2 * 1300, V2 * 1200))
spinal_13_contra <- spinal_13[order(nrow(spinal_13):1),]
spinal_13_contra$V1[sapply(spinal_13_contra$V1 , is.numeric)] <- spinal_13_contra$V1 [sapply(spinal_13_contra$V1, is.numeric)] * -1
spinal_13 <- rbind(spinal_13, spinal_13_contra)

## -- 13B -- ##
df13_BC <- df13_BC %>%  mutate(Label = replace(Label, Label == 4, 3))
df13_BC$Label <- as.character(as.character(df13_BC$Label))
df13_BC_plot <- df13_BC %>%  filter(Label != 0)

Figure_13B <- ggplot() +
  annotate("point", x = 0, y = 0, size = 4, colour = "black") +
  geom_path(data = spinal_13, aes(x = V1, y = V2), size = 1) + 
  geom_point(data = df13_BC_plot, aes(x = Xpos, y = - Ypos, fill = Label), shape = 21, colour = "black", size = 2, alpha = 0.8) + #Here plotting -Ypos because ventral is positive in data frame
  scale_y_continuous(limits=c(-600, 650), breaks=c(-600,0,650), expand = c(0, 0)) +
  scale_x_continuous(limits=c(-955, 955), breaks=seq(from = -850, to = 850, by = 850), expand = c(0, 0)) +
  scale_fill_manual(values = c("#7fff01","#ff6868","#ffcc33"), name = NULL, guide = "legend", labels = c("ChAT only", "TdTom only", "Dual")) +
  ylab("V-D (μm)") +  
  xlab("M-L (μm)") +
  theme(axis.line = element_line(size = 0.5, colour = "black", linetype=1),
        panel.background = element_rect(fill = "transparent", colour = "transparent"),
        plot.background = element_rect(fill = "transparent", colour = NA_character_),
        legend.background = element_rect(fill = "transparent", linetype = 'blank'), legend.position="top",
        legend.text = element_text(face="bold"),
        legend.key = element_rect(fill = "transparent")) +
  guides(fill = guide_legend(override.aes = list(size=3))) 
  
Figure_13B  <- ggarrange(NULL, Figure_13B, nrow = 1, widths = c(0.2, 4), labels = "B", font.label = list(size = 28, color = "black"))


## -- 13C -- ##

Figure_13C <- plot(euler(c("ChAT"= sum(df13_BC_plot$Label == "1"),"TdTom"=sum(df13_BC_plot$Label == "2"),
                           "ChAT&TdTom"=sum(df13_BC_plot$Label == "3"))), quantities = list(type = "counts", fontsize = 24), # if values wanted as percent -> quantities = list(type = "percent")
                   fills=c("#7fff01","#ff6868","#ffcc33"), legend = NULL)
Figure_13C  <- ggarrange(NULL, Figure_13C, nrow = 1, widths = c(0.2, 4), labels = "C", font.label = list(size = 28, color = "black"))

cowplot::plot_grid(Figure_13B, Figure_13C, ncol=1,nrow = 2, align = "hv", rel_widths = c(5), rel_heights = c(4,4))
```

**Figure13**: Analysis of ectopic Cre expression in *ChAT-IRES-Cre* and *RCL-tdTom* mice. A: representative lumbar section stained with antibodies against ChAT (green) and tdTom (red) and (B) map of neurons labelled with both or one of the two antibodies in all the 13 analysed sections from 3 mice, showing that some of the tdTom positive neurons do not express ChAT, indicating either a developmental downregulation of ChAT expression or a modest leak in the Cre expression. C: Venn diagram showing the overall number of mapped neurons.

### Figure 13 Supplement 1

```{r, fig.align='left', fig.width= 12 , fig.height= 6}
## -- 13Supp1 -- ##
fig13S1 <- readPNG("~/figures/Figure_13supp1.png")
Fig13S1 <- ggplot() + 
  background_image(fig13S1) +
  coord_equal(ratio = 0.511) +
  annotation_custom(rasterGrob(fig13S1, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig13S1
```

**Figure 13 figure supplement 1**: Two examples of a longitudinal section of two different spinal cords from a heterozygous RΦGT mouse injected in the LG with EnvA-ΔG-Rab-EGFP, showing a small number of infected motor neurons, but no evidence of transsynaptic jumps, indicating ectopic expression of the TVA receptor, but not of the rabies glycoprotein.

### Figure 13 Supplement 2

```{r, fig.align='left', fig.width= 12 , fig.height= 6}
## -- 13Supp2 -- ##
fig13S2 <- readPNG("~/figures/Figure_13supp2.png")
Fig13S2 <- ggplot() + 
  background_image(fig13S2) +
  coord_equal(ratio = 0.689) +
  annotation_custom(rasterGrob(fig13S2, 
                               width = unit(1,"npc"), 
                               height = unit(1,"npc")), 
                    -Inf, Inf, -Inf, Inf) +
  theme(plot.margin = margin(t = 0, l=0, r=0, b = 0, unit = "cm"))
Fig13S2
```

**Figure 13 figure supplement 2**: Schematic of a section of the spinal cord, indicating the reference points used for normalization. Each section was translated to have the origin of a Cartesian set of axes centered on the central canal (CC). A line passing through the central canal and perpendicular to the dorso-ventral axis was used to identify the edge of the white matter in the mediolateral direction (ml) and the ml-CC distance was used to normalize the x-coordinates. Along the dorso-ventral axes, the 4 edges of the white matter (ne, nw, se, sw) were identified and their distance from the horizontal line passing through the central canal were used to normalize the y-coordinates independently in each of the 4 quadrants. .

# Acknowledgements

This work was supported by BBSRC grants BB/L001454 to MB, AJT and DJM and BB/S005943/1 to MB, a Leverhulme Trust grant to MB (grant number RPG-2013-176) and a Wellcome Trust Investigator Award to RMB (110193). JDM was supported by a fellowship from the Jane Coffins Child Memorial Fund for Medical Research and 5K99HD096512 from NICHD. BKB was supported by UCSD Cell and Molecular Genetics Training Program (T32 GM007240), Timken-Sturgis Foundation, Salk Women in Science and Marguerite Vogt Award. RMB is supported by Brain Research UK. SS and NZ were supported by the DFG (ZA 885/1-1 and EXC 257 NeuroCure). SLP is supported as the Benjamin H. Lewis chair in neuroscience and Sol Goldman Charitable Trust, and National Institutes of Health (1 U19 NS112959-01 and 1 R01 NS123160-01). We thank Dr. Ariel Levine for participation in some early experiments and helpful discussions on the data. We are grateful to Drs. Silvia Arber and Marco Tripodi for constructive discussions and helpful comments during the preparation of this manuscript. We thank Roser Montañana-Rosell for critical reading of the manuscript.

# References

Bacskai T, Rusznak Z, Paxinos G, Watson C (2013) Musculotopic organization of the motor neurons supplying the mouse hindlimb muscles: a quantitative study using Fluoro-Gold retrograde tracing. Brain Struct Funct Available at: <http://www.ncbi.nlm.nih.gov/pubmed/23288256>.

Balaskas N, Abbott LF, Jessell TM, Ng D (2019) Positional Strategies for Connection Specificity and Synaptic Organization in Spinal Sensory-Motor Circuits. Neuron 102:1143-1156.e4.

Bhumbra GS, Beato M (2018) Recurrent excitation between motoneurones propagates across segments and is purely glutamatergic. PLoS Biology 16.

Bhumbra GS, Dyball RE (2010) Reading between the spikes of the hypothalamic neural code. J Neuroendocrinol 22:1239-1250. Bikoff JB, Gabitto MI, Rivard AF, Drobac E, MacHado TA, Miri A, Brenner-Morton S, Famojure E, Diaz C, Alvarez FJ, Mentis GZ, Jessell TM (2016) Spinal Inhibitory Interneuron Diversity Delineates Variant Motor Microcircuits. Cell 165.

Callaway EM, Luo L (2015) Monosynaptic Circuit Tracing with Glycoprotein-Deleted Rabies Viruses. J Neurosci 35:8979-8985.

Chatterjee S et al. (2018) Nontoxic, double-deletion-mutant rabies viral vectors for retrograde targeting of projection neurons. Nat Neurosci 21:638-646.

Chen J-A, Huang Y-P, Mazzoni EO, Tan GC, Zavadil J, Wichterle H (2011) Mir-17-3p Controls Spinal Neural Progenitor Patterning by Regulating Olig2/Irx3 Cross-Repressive Loop. Neuron 69:721-735.

Ciabatti E, González-Rueda A, Mariotti L, Morgese F, Tripodi M (2017) Life-Long Genetic and Functional Access to Neural Circuits Using Self-Inactivating Rabies Virus. Cell 170:382-392.e14.

Dessaud E, Yang LL, Hill K, Cox B, Ulloa F, Ribeiro A, Mynett A, Novitch BG, Briscoe J (2007) Interpretation of the sonic hedgehog morphogen gradient by a temporal adaptation mechanism. Nature 450:717-720.

Feng R, Xie J, Lu J, Hu H, Chen Y, Wang D, Gao L (2022) Decoding the mouse spinal cord locomotor neural network using tissue clearing, tissue expansion and tiling light sheet microscopy techniques. :2022.07.04.498760 Available at: <https://www.biorxiv.org/content/10.1101/2022.07.04.498760v1> [Accessed October 10, 2022].

Goetz C, Pivetta C, Arber S (2015) Distinct Limb and Trunk Premotor Circuits Establish Laterality in the Spinal Cord. Neuron 85:131-144.

Gómez-de-Mariscal E, Guerrero V, Sneider A, Jayatilaka H, Phillip JM, Wirtz D, Muñoz-Barrutia A (2021) Use of the p-values as a size-dependent function to address practical differences when analyzing large datasets. Sci Rep 11:20942.

Gu Z, Serradj N, Ueno M, Liang M, Li J, Baccei ML, Martin JH, Yoshida Y (2017) Skilled Movements Require Non-apoptotic Bax/Bak Pathway-Mediated Corticospinal Circuit Reorganization. Neuron.

Hedges LV (1981) Distribution Theory for Glass's Estimator of Effect Size and Related Estimators. Journal of Educational Statistics 6:107-128.

Hultborn H, Jankowska E, Lindstrom S (1971) Recurrent inhibition of interneurones monosynaptically activated from group Ia afferents. The Journal of Physiology Online 215:613-636.

Jankowska E (2001) Spinal interneuronal systems: identification, multifunctional character and reconfigurations in mammals. J Physiol 533:31-40.

Jovanovic K, Pastor AM, O'Donovan MJ (2010) The use of PRV-Bartha to define premotor inputs to lumbar motoneurons in the neonatal spinal cord of the mouse. PLoS ONE 5:e11743-e11743.

Kim EJ, Jacobs MW, Ito-Cole T, Callaway EM (2016) Improved Monosynaptic Neural Circuit Tracing Using Engineered Rabies Virus Glycoproteins. Cell Reports 15:692-699.

Koronfel LM, Kanning KC, Alcos A, Henderson CE, Brownstone RM (2021) Elimination of glutamatergic transmission from Hb9 interneurons does not impact treadmill locomotion. Sci Rep 11:16008.

Li L, Rutlin M, Abraira VE, Cassidy C, Kus L, Gong S, Jankowski MP, Luo W, Heintz N, Koerber HR, Woodbury CJ, Ginty DD (2011) The Functional Organization of Cutaneous Low-Threshold Mechanosensory Neurons. Cell 147:1615-1627.

Marshel JH, Mori T, Nielsen KJ, Callaway EM (2010) Targeting single neuronal networks for gene expression and cell labeling in vivo. Neuron 67:562-574.

McHanwell S, Biscoe TJ (1981) The localization of motoneurons supplying the hindlimb muscles of the mouse. Philos Trans R Soc Lond B Biol Sci 293:477-508.

Ohara S, Inoue K, Witter MP, Iijima T (2009a) Untangling neural networks with dual retrograde transsynaptic viral infection. Front Neurosci 3 Available at: <https://www.frontiersin.org/articles/10.3389/neuro.01.032.2009/full> [Accessed November 18, 2020].

Ohara S, Inoue K, Yamada M, Yamawaki T, Koganezawa N, Tsutsui K-I, Witter MP, Iijima T (2009b) Dual transneuronal tracing in the rat entorhinal-hippocampal circuit by intracerebral injection of recombinant rabies virus vectors. Front Neuroanat 3 Available at: <https://www.frontiersin.org/articles/10.3389/neuro.05.001.2009/full> [Accessed November 18, 2020].

Osakada F, Callaway EM (2013) Design and generation of recombinant rabies virus vectors. Nature Protocols 8:1583-1601.

Pimpinella S, Zampieri N (2021) Rabies anterograde monosynaptic tracing reveals organization of spinal sensory circuits. Available at: <https://www.biorxiv.org/content/10.1101/2021.02.11.429920v1> [Accessed September 3, 2021].

Rancz EA, Franks KM, Schwarz MK, Pichler B, Schaefer AT, Margrie TW (2011) Transfection via whole-cell recording in vivo: bridging single-cell physiology, genetics and connectomics. Nat Neurosci 14:527-532.

Rathelot J-A, Strick PL (2009) Subdivisions of primary motor cortex based on cortico-motoneuronal cells. Proceedings of the National Academy of Sciences 106:918-923.

Reardon TR, Murray AJ, Turi GF, Wirblich C, Croce KR, Schnell MJ, Jessell TM, Losonczy A (2016) Rabies Virus CVS-N2c Strain Enhances Retrograde Synaptic Transfer and Neuronal Viability. Neuron Available at: <http://www.ncbi.nlm.nih.gov/pubmed/26804990>.

Romanes GJ (1964) THE MOTOR POOLS OF THE SPINAL CORD. Prog Brain Res 11:93-119.

Ronzano R, Lancelin C, Bhumbra GS, Brownstone RM, Beato M (2021) Proximal and distal spinal neurons innervating multiple synergist and antagonist motor pools Thoby-Brisson M, Calabrese RL, Gosgnach S, eds. eLife 10:e70858.

Rossi J, Balthasar N, Olson D, Scott M, Berglund E, Lee CE, Choi MJ, Lauzon D, Lowell BB, Elmquist JK (2011) Melanocortin-4 Receptors Expressed by Cholinergic Neurons Regulate Energy Balance and Glucose Homeostasis. Cell Metabolism 13:195-204.

Saravanan V, Berman GJ, Sober SJ (2020) Application of the hierarchical bootstrap to multi-level data in neuroscience. Neuron Behav Data Anal Theory 3:<https://nbdt.scholasticahq.com/article/13927-application-of-the-hierarchical-bootstrap-to-multi-level-data-in-neuroscience>.

Skarlatou S, Hérent C, Toscano E, Mendes CS, Bouvier J, Zampieri N (2020) Afadin Signaling at the Spinal Neuroepithelium Regulates Central Canal Formation and Gait Selection. Cell Rep 31:107741.

Stanek E IV, Cheng S, Takatoh J, Han B-X, Wang F (2014) Monosynaptic premotor circuit tracing reveals neural substrates for oro-motor coordination Mason P, ed. eLife 3:e02511.

Stepien AE, Tripodi M, Arber S (2010) Monosynaptic rabies virus reveals premotor network organization and synaptic specificity of cholinergic partition cells. Neuron 68:456-472.

Strack AM, Loewy AD (1990) Pseudorabies virus: a highly specific transneuronal cell body marker in the sympathetic nervous system. J Neurosci 10:2139-2147.

Sürmeli G, Akay T, Ippolito GC, Tucker PW, Jessell TM (2011) Patterns of Spinal Sensory-Motor Connectivity Prescribed by a Dorsoventral Positional Template. Cell 147:653-665.

Takatoh J, Nelson A, Zhou X, Bolton MM, Ehlers MD, Arenkiel BR, Mooney R, Wang F (2013) New modules are added to vibrissal premotor circuitry with the emergence of exploratory whisking. Neuron 77:346-360.

Takeoka A, Arber S (2019) Functional Local Proprioceptive Feedback Circuits Initiate and Maintain Locomotor Recovery after Spinal Cord Injury. Cell Reports 27:71-85.e3.

Tran-Van-Minh A, Ye Z, Rancz E (2022) Quantitative analysis of rabies virus-based synaptic connectivity tracing. :2022.06.08.494952 Available at: <https://www.biorxiv.org/content/10.1101/2022.06.08.494952v1> [Accessed September 15, 2022].

Tripodi M, Stepien AE, Arber S (2011) Motor antagonism exposed by spatial segregation and timing of neurogenesis. Nature 479:61-66.

Ugolini G (2011) Rabies Virus as a Transneuronal Tracer of Neuronal Connections.

Ugolini G (2020) Viruses in connectomics: Viral transneuronal tracers and genetically modified recombinants as neuroscience research tools. Journal of Neuroscience Methods 346:108917.

Wall NR, Wickersham IR, Cetin A, De La Parra M, Callaway EM (2010) Monosynaptic circuit tracing in vivo through Cre-dependent targeting and complementation of modified rabies virus. Proc Natl Acad Sci U S A 107:21848-21853.

Wang X, Liu Y, Li X, Zhang Z, Yang H, Zhang Y, Williams PR, Alwahab NSA, Kapur K, Yu B, Zhang Y, Chen M, Ding H, Gerfen CR, Wang KH, He Z (2017) Deconstruction of Corticospinal Circuits for Goal-Directed Motor Skills. Cell 171:440-455.e14.

Wickersham IR, Lyon DC, Barnard RJ, Mori T, Finke S, Conzelmann KK, Young JA, Callaway EM (2007) Monosynaptic restriction of transsynaptic tracing from single, genetically targeted neurons. Neuron 53:639-647.

Zagoraiou L, Akay T, Martin JF, Brownstone RM, Jessell TM, Miles GB (2009) A Cluster of Cholinergic Premotor Interneurons Modulates Mouse Locomotor Activity. Neuron 64:645-662.

Zampieri N, Jessell TM, Murray AJ (2014) Mapping sensory circuits by anterograde transsynaptic transfer of recombinant rabies virus. Neuron 81:766-778.

Zeilhofer HU, Studler B, Arabadzisz D, Schweizer C, Ahmadi S, Layh B, Bosl MR, Fritschy JM (2005) Glycinergic neurons expressing enhanced green fluorescent protein in bacterial artificial chromosome transgenic mice. J Comp Neurol 482:123-141.

